{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Presentación final\n",
    "El objetivo de las presentaciones es que aprendan a buscar y poblar ustedes sus propios datasets con el fin de crear valor a partir de los datos.\n",
    "Serán en parejas, tendrán hasta el lunes 12 de junio a las 23:59 para subirla a Aula y serán el miércoles 14 de junio. Deberán presentar entre 5 y 7\n",
    "minutos más 3 minutos máximo para responder preguntas.\n",
    "Para ello se debe trabajar con los sitios Web of Science y Scopus. Para acceder a los repositorios deben entrar al link Bibliotecas PUCV. En la\n",
    "sección “A un click” deben seleccionar Web of Science o Scopus dentro de la columna Bases de datos multidisciplinarias e ingresar con su correo\n",
    "@mail.pucv.cl.\n",
    "Los temas a trabajar deben tener relación con la carrera (ej: machine learning aplicado en medicina).\n",
    "Para generar el dataset con los resultados de la búsqueda seleccionar el cuadro “Export”.\n",
    "Para escoger las columnas para poblar el dataset seleccionar la opción Excel o CSV, luego asegurarse de que las columnas seleccionadas sean las\n",
    "mismas que en las siguientes imágenes y exportar:\n",
    "Para Web of Science\n",
    "Para Scopus\n",
    "Generar un archivo que contenga los primeros 1000 registros para cada caso y concatenar los archivos resultantes en uno solo.\n",
    "Como primera instancia de preprocesamiento deberán eliminar los registros duplicados según DOI (Digital Object Identifier), que corresponde a\n",
    "un valor único asignado a cada publicación.\n",
    "Lo que se evaluará:\n",
    "1. Introducción al tema que eligieron (contextualización) + review del paper de mayor impacto 1pt\n",
    "2. Preprocesamiento de los datos (registros duplicados, manejo de nulos, etc) 1pt\n",
    "3. Estadísticos básicos del dataset ya preprocesado (resultados con los que puedan obtener conclusiones) 1pt\n",
    "4. Gráficos (al menos 3 distintos y que aporten a la visualización de información de valor) 1pt\n",
    "5. Expresión oral (manejo de lenguaje formal) 1pt\n",
    "6. Presentación (ppt, colab, otros) 1pt\n",
    "7. Comunicación no verbal (postura, acciones) 1p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: xlrd in /home/ignacio/.local/lib/python3.11/site-packages (2.0.1)\n",
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Requirement already satisfied: openpyxl in /home/ignacio/.local/lib/python3.11/site-packages (3.1.2)\n",
      "Requirement already satisfied: et-xmlfile in /home/ignacio/.local/lib/python3.11/site-packages (from openpyxl) (1.1.0)\n"
     ]
    }
   ],
   "source": [
    "!pip install xlrd\n",
    "!pip install openpyxl\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd \n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import os\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['.~lock.savedrecs.xls#',\n",
       " 'scopus.csv',\n",
       " 'savedrecs.xls',\n",
       " 'AstorgaIgnacioPresentacion.ipynb']"
      ]
     },
     "execution_count": 78,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#ver librerias dentro de la carpeta\n",
    "os.listdir()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1000, 14)\n",
      "(873, 14)\n",
      "(873, 14)\n",
      "Index(['Authors', 'Author full names', 'Author(s) ID', 'Title', 'Year',\n",
      "       'Source title', 'Cited by', 'DOI', 'Link', 'Abstract',\n",
      "       'Molecular Sequence Numbers', 'Chemicals/CAS', 'Document Type',\n",
      "       'Source'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "source": [
    "scopus = pd.read_csv('scopus.csv')\n",
    "print(scopus.shape)\n",
    "scopus = scopus.drop_duplicates(subset=['DOI'])\n",
    "print(scopus.shape)\n",
    "scopus = scopus.dropna(axis=1, how='all')\n",
    "print(scopus.shape)\n",
    "print(scopus.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1000, 72)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Publication Type</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Book Authors</th>\n",
       "      <th>Book Editors</th>\n",
       "      <th>Book Group Authors</th>\n",
       "      <th>Author Full Names</th>\n",
       "      <th>Book Author Full Names</th>\n",
       "      <th>Group Authors</th>\n",
       "      <th>Article Title</th>\n",
       "      <th>Source Title</th>\n",
       "      <th>...</th>\n",
       "      <th>Web of Science Index</th>\n",
       "      <th>Research Areas</th>\n",
       "      <th>IDS Number</th>\n",
       "      <th>Pubmed Id</th>\n",
       "      <th>Open Access Designations</th>\n",
       "      <th>Highly Cited Status</th>\n",
       "      <th>Hot Paper Status</th>\n",
       "      <th>Date of Export</th>\n",
       "      <th>UT (Unique WOS ID)</th>\n",
       "      <th>Web of Science Record</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>J</td>\n",
       "      <td>Kallens, PC; Kristensen-McLachlan, RD; Christi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contreras Kallens, Pablo; Kristensen-McLachlan...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Large Language Models Demonstrate the Potentia...</td>\n",
       "      <td>COGNITIVE SCIENCE</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-05-31</td>\n",
       "      <td>WOS:000939418000001</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>J</td>\n",
       "      <td>Kim, KH; Jung, DY; Lee, D; Lee, HJ; Park, SY; ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Kim, Kwang-Ho; Jung, Dae-Young; Lee, Donghyun;...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Implementation of a large-scale language model...</td>\n",
       "      <td>MULTIMEDIA TOOLS AND APPLICATIONS</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-05-31</td>\n",
       "      <td>WOS:000376601700012</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>J</td>\n",
       "      <td>Leippold, M</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Leippold, Markus</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Thus spoke GPT-3: Interviewing a large-languag...</td>\n",
       "      <td>FINANCE RESEARCH LETTERS</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-05-31</td>\n",
       "      <td>WOS:000956118300001</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>J</td>\n",
       "      <td>Veres, C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Veres, Csaba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Large Language Models are Not Models of Natura...</td>\n",
       "      <td>IEEE ACCESS</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-05-31</td>\n",
       "      <td>WOS:000812546700001</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>J</td>\n",
       "      <td>Kasneci, E; Sessler, K; Kuchemann, S; Bannert,...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Kasneci, Enkelejda; Sessler, Kathrin; Kuechema...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ChatGPT for good? On opportunities and challen...</td>\n",
       "      <td>LEARNING AND INDIVIDUAL DIFFERENCES</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2023-05-31</td>\n",
       "      <td>WOS:000961991400001</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 72 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  Publication Type                                            Authors   \n",
       "0                J  Kallens, PC; Kristensen-McLachlan, RD; Christi...  \\\n",
       "1                J  Kim, KH; Jung, DY; Lee, D; Lee, HJ; Park, SY; ...   \n",
       "2                J                                        Leippold, M   \n",
       "3                J                                           Veres, C   \n",
       "4                J  Kasneci, E; Sessler, K; Kuchemann, S; Bannert,...   \n",
       "\n",
       "   Book Authors Book Editors  Book Group Authors   \n",
       "0           NaN          NaN                 NaN  \\\n",
       "1           NaN          NaN                 NaN   \n",
       "2           NaN          NaN                 NaN   \n",
       "3           NaN          NaN                 NaN   \n",
       "4           NaN          NaN                 NaN   \n",
       "\n",
       "                                   Author Full Names  Book Author Full Names   \n",
       "0  Contreras Kallens, Pablo; Kristensen-McLachlan...                     NaN  \\\n",
       "1  Kim, Kwang-Ho; Jung, Dae-Young; Lee, Donghyun;...                     NaN   \n",
       "2                                   Leippold, Markus                     NaN   \n",
       "3                                       Veres, Csaba                     NaN   \n",
       "4  Kasneci, Enkelejda; Sessler, Kathrin; Kuechema...                     NaN   \n",
       "\n",
       "  Group Authors                                      Article Title   \n",
       "0           NaN  Large Language Models Demonstrate the Potentia...  \\\n",
       "1           NaN  Implementation of a large-scale language model...   \n",
       "2           NaN  Thus spoke GPT-3: Interviewing a large-languag...   \n",
       "3           NaN  Large Language Models are Not Models of Natura...   \n",
       "4           NaN  ChatGPT for good? On opportunities and challen...   \n",
       "\n",
       "                          Source Title  ... Web of Science Index   \n",
       "0                    COGNITIVE SCIENCE  ...                  NaN  \\\n",
       "1    MULTIMEDIA TOOLS AND APPLICATIONS  ...                  NaN   \n",
       "2             FINANCE RESEARCH LETTERS  ...                  NaN   \n",
       "3                          IEEE ACCESS  ...                  NaN   \n",
       "4  LEARNING AND INDIVIDUAL DIFFERENCES  ...                  NaN   \n",
       "\n",
       "   Research Areas  IDS Number Pubmed Id Open Access Designations   \n",
       "0             NaN         NaN       NaN                      NaN  \\\n",
       "1             NaN         NaN       NaN                      NaN   \n",
       "2             NaN         NaN       NaN                      NaN   \n",
       "3             NaN         NaN       NaN                      NaN   \n",
       "4             NaN         NaN       NaN                      NaN   \n",
       "\n",
       "  Highly Cited Status Hot Paper Status  Date of Export   UT (Unique WOS ID)   \n",
       "0                 NaN              NaN      2023-05-31  WOS:000939418000001  \\\n",
       "1                 NaN              NaN      2023-05-31  WOS:000376601700012   \n",
       "2                 NaN              NaN      2023-05-31  WOS:000956118300001   \n",
       "3                 NaN              NaN      2023-05-31  WOS:000812546700001   \n",
       "4                 NaN              NaN      2023-05-31  WOS:000961991400001   \n",
       "\n",
       "   Web of Science Record  \n",
       "0                      0  \n",
       "1                      0  \n",
       "2                      0  \n",
       "3                      0  \n",
       "4                      0  \n",
       "\n",
       "[5 rows x 72 columns]"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# archivo de excel savedrecs.xls \n",
    "webofscience = pd.read_excel('savedrecs.xls', engine='xlrd')\n",
    "print(webofscience.shape)\n",
    "webofscience.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [],
   "source": [
    "webofscience = webofscience.dropna(axis=1, how='all')\n",
    "webofscience = webofscience.drop_duplicates(subset=['DOI'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(940, 37)\n",
      "Index(['Publication Type', 'Authors', 'Book Editors', 'Author Full Names',\n",
      "       'Group Authors', 'Article Title', 'Source Title', 'Book Series Title',\n",
      "       'Document Type', 'Conference Title', 'Conference Date',\n",
      "       'Conference Location', 'Conference Host', 'Abstract', 'Addresses',\n",
      "       'Reprint Addresses', 'Email Addresses', 'Times Cited, WoS Core',\n",
      "       'Times Cited, All Databases', 'Publication Date', 'Publication Year',\n",
      "       'Volume', 'Issue', 'Part Number', 'Supplement', 'Special Issue',\n",
      "       'Start Page', 'End Page', 'Article Number', 'DOI', 'DOI Link',\n",
      "       'Early Access Date', 'Highly Cited Status', 'Hot Paper Status',\n",
      "       'Date of Export', 'UT (Unique WOS ID)', 'Web of Science Record'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "source": [
    "print(webofscience.shape)\n",
    "print(webofscience.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index(['Authors', 'Author Full Names', 'Author(s) ID', 'Article Title', 'Year',\n",
      "       'Source Title', 'Times Cited, All Databases', 'DOI', 'DOI Link',\n",
      "       'Abstract', 'Molecular Sequence Numbers', 'Chemicals/CAS',\n",
      "       'Document Type', 'Source'],\n",
      "      dtype='object')\n",
      "Index(['Publication Type', 'Authors', 'Book Editors', 'Author Full Names',\n",
      "       'Group Authors', 'Article Title', 'Source Title', 'Book Series Title',\n",
      "       'Document Type', 'Conference Title', 'Conference Date',\n",
      "       'Conference Location', 'Conference Host', 'Abstract', 'Addresses',\n",
      "       'Reprint Addresses', 'Email Addresses', 'Times Cited, WoS Core',\n",
      "       'Times Cited, All Databases', 'Publication Date', 'Publication Year',\n",
      "       'Volume', 'Issue', 'Part Number', 'Supplement', 'Special Issue',\n",
      "       'Start Page', 'End Page', 'Article Number', 'DOI', 'DOI Link',\n",
      "       'Early Access Date', 'Highly Cited Status', 'Hot Paper Status',\n",
      "       'Date of Export', 'UT (Unique WOS ID)', 'Web of Science Record'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "source": [
    "scopus = scopus.rename(columns={\n",
    "    'Author full names': 'Author Full Names',\n",
    "    'Title': 'Article Title',\n",
    "    'Source title': 'Source Title',\n",
    "    'Cited by': 'Times Cited, All Databases',\n",
    "    'DOI': 'DOI',\n",
    "    'Link': 'DOI Link',\n",
    "    'Abstract': 'Abstract',\n",
    "    'Document Type': 'Document Type'\n",
    "})\n",
    "\n",
    "webofscience = webofscience.rename(columns={\n",
    "    'Authors': 'Authors',\n",
    "    'Author Full Names': 'Author Full Names',\n",
    "    'Article Title': 'Article Title',\n",
    "    'Source Title': 'Source Title',\n",
    "    'Times Cited, All Databases': 'Times Cited, All Databases',\n",
    "    'DOI': 'DOI',\n",
    "    'DOI Link': 'DOI Link',\n",
    "    'Abstract': 'Abstract',\n",
    "    'Document Type': 'Document Type'\n",
    "})\n",
    "\n",
    "print(scopus.columns)\n",
    "print(webofscience.columns)\n",
    "\n",
    "\n",
    "dataframe = pd.concat([scopus, webofscience], axis=0, ignore_index=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1813, 42)\n",
      "Index(['Authors', 'Author Full Names', 'Author(s) ID', 'Article Title', 'Year',\n",
      "       'Source Title', 'Times Cited, All Databases', 'DOI', 'DOI Link',\n",
      "       'Abstract', 'Molecular Sequence Numbers', 'Chemicals/CAS',\n",
      "       'Document Type', 'Source', 'Publication Type', 'Book Editors',\n",
      "       'Group Authors', 'Book Series Title', 'Conference Title',\n",
      "       'Conference Date', 'Conference Location', 'Conference Host',\n",
      "       'Addresses', 'Reprint Addresses', 'Email Addresses',\n",
      "       'Times Cited, WoS Core', 'Publication Date', 'Publication Year',\n",
      "       'Volume', 'Issue', 'Part Number', 'Supplement', 'Special Issue',\n",
      "       'Start Page', 'End Page', 'Article Number', 'Early Access Date',\n",
      "       'Highly Cited Status', 'Hot Paper Status', 'Date of Export',\n",
      "       'UT (Unique WOS ID)', 'Web of Science Record'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "source": [
    "print(dataframe.shape)\n",
    "print(dataframe.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1813, 7)\n",
      "(1813, 7)\n"
     ]
    }
   ],
   "source": [
    "# Paso 1: Eliminación de columnas redundantes\n",
    "dataframe = dataframe.drop(['Publication Year', 'Article Title'], axis=1)\n",
    "\n",
    "# Paso 2: Manejo de valores perdidos\n",
    "# Eliminar columnas con más del 50% de valores NaN\n",
    "half_count = len(dataframe) / 2\n",
    "dataframe = dataframe.dropna(thresh=half_count, axis=1)\n",
    "\n",
    "# O alternativamente, rellenar los valores NaN con un valor por defecto\n",
    "# dataframe = dataframe.fillna(\"Valor por defecto\")\n",
    "\n",
    "# Paso 3: Eliminación de filas que no sirven para el análisis, identificadas manualmente\n",
    "dataframe = dataframe.drop(['Publication Type', 'Addresses', 'Reprint Addresses', 'DOI Link', 'Times Cited, WoS Core', 'Date of Export', 'UT (Unique WOS ID)', 'Web of Science Record'], axis=1)\n",
    "\n",
    "\n",
    "# Paso 4: Eliminación de duplicados\n",
    "# Eliminar duplicados\n",
    "print(dataframe.shape)\n",
    "dataframe = dataframe.drop_duplicates()\n",
    "print(dataframe.shape)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Authors</th>\n",
       "      <th>Author Full Names</th>\n",
       "      <th>Source Title</th>\n",
       "      <th>Times Cited, All Databases</th>\n",
       "      <th>DOI</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Document Type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....</td>\n",
       "      <td>Delaglio, Frank (6603722428); Grzesiek, Stepha...</td>\n",
       "      <td>Journal of Biomolecular NMR</td>\n",
       "      <td>11644</td>\n",
       "      <td>10.1007/BF00197809</td>\n",
       "      <td>The NMRPipe system is a UNIX software environm...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Srivastava N.; Hinton G.; Krizhevsky A.; Sutsk...</td>\n",
       "      <td>Srivastava, Nitish (55697473300); Hinton, Geof...</td>\n",
       "      <td>Journal of Machine Learning Research</td>\n",
       "      <td>25310</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Deep neural nets with a large number of parame...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...</td>\n",
       "      <td>Jumper, John (36717725300); Evans, Richard (57...</td>\n",
       "      <td>Nature</td>\n",
       "      <td>6981</td>\n",
       "      <td>10.1038/s41586-021-03819-2</td>\n",
       "      <td>Proteins are essential to life, and understand...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Wild S.; Roglic G.; Green A.; Sicree R.; King H.</td>\n",
       "      <td>Wild, Sarah (7103086309); Roglic, Gojka (67016...</td>\n",
       "      <td>Diabetes Care</td>\n",
       "      <td>12069</td>\n",
       "      <td>10.2337/diacare.27.5.1047</td>\n",
       "      <td>OBJECTIVE - The goal of this study was to esti...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Cover T.M.; Thomas J.A.</td>\n",
       "      <td>Cover, Thomas M. (7102102743); Thomas, Joy A. ...</td>\n",
       "      <td>Elements of Information Theory</td>\n",
       "      <td>33728</td>\n",
       "      <td>10.1002/047174882X</td>\n",
       "      <td>The latest edition of this classic is updated ...</td>\n",
       "      <td>Book</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1808</th>\n",
       "      <td>Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...</td>\n",
       "      <td>Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...</td>\n",
       "      <td>APPLIED SCIENCES-BASEL</td>\n",
       "      <td>0</td>\n",
       "      <td>10.3390/app122412946</td>\n",
       "      <td>The language inclusion checking of timed autom...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1809</th>\n",
       "      <td>Roelens, B; Poels, G</td>\n",
       "      <td>Roelens, Ben; Poels, Geert</td>\n",
       "      <td>BUSINESS &amp; INFORMATION SYSTEMS ENGINEERING</td>\n",
       "      <td>7</td>\n",
       "      <td>10.1007/s12599-014-0363-z</td>\n",
       "      <td>Business models (BM) are the central concept t...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>Pulvermuller, F</td>\n",
       "      <td>Pulvermueller, Friedemann</td>\n",
       "      <td>PROGRESS IN NEUROBIOLOGY</td>\n",
       "      <td>104</td>\n",
       "      <td>10.1016/j.pneurobio.2017.07.001</td>\n",
       "      <td>Neurocognitive and neurolinguistics theories m...</td>\n",
       "      <td>Review</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1811</th>\n",
       "      <td>Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...</td>\n",
       "      <td>Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...</td>\n",
       "      <td>ACM TRANSACTIONS ON GRAPHICS</td>\n",
       "      <td>21</td>\n",
       "      <td>10.1145/3272127.3275035</td>\n",
       "      <td>We introduce a novel framework for using natur...</td>\n",
       "      <td>Article; Proceedings Paper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1812</th>\n",
       "      <td>Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...</td>\n",
       "      <td>Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...</td>\n",
       "      <td>ACM SIGPLAN NOTICES</td>\n",
       "      <td>322</td>\n",
       "      <td>10.1145/780822.781133</td>\n",
       "      <td>We present nesC, a programming language for ne...</td>\n",
       "      <td>Article; Proceedings Paper</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1813 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                Authors   \n",
       "0     Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....  \\\n",
       "1     Srivastava N.; Hinton G.; Krizhevsky A.; Sutsk...   \n",
       "2     Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...   \n",
       "3      Wild S.; Roglic G.; Green A.; Sicree R.; King H.   \n",
       "4                               Cover T.M.; Thomas J.A.   \n",
       "...                                                 ...   \n",
       "1808  Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...   \n",
       "1809                               Roelens, B; Poels, G   \n",
       "1810                                    Pulvermuller, F   \n",
       "1811  Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...   \n",
       "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
       "\n",
       "                                      Author Full Names   \n",
       "0     Delaglio, Frank (6603722428); Grzesiek, Stepha...  \\\n",
       "1     Srivastava, Nitish (55697473300); Hinton, Geof...   \n",
       "2     Jumper, John (36717725300); Evans, Richard (57...   \n",
       "3     Wild, Sarah (7103086309); Roglic, Gojka (67016...   \n",
       "4     Cover, Thomas M. (7102102743); Thomas, Joy A. ...   \n",
       "...                                                 ...   \n",
       "1808  Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...   \n",
       "1809                         Roelens, Ben; Poels, Geert   \n",
       "1810                          Pulvermueller, Friedemann   \n",
       "1811  Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...   \n",
       "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
       "\n",
       "                                    Source Title  Times Cited, All Databases   \n",
       "0                    Journal of Biomolecular NMR                       11644  \\\n",
       "1           Journal of Machine Learning Research                       25310   \n",
       "2                                         Nature                        6981   \n",
       "3                                  Diabetes Care                       12069   \n",
       "4                 Elements of Information Theory                       33728   \n",
       "...                                          ...                         ...   \n",
       "1808                      APPLIED SCIENCES-BASEL                           0   \n",
       "1809  BUSINESS & INFORMATION SYSTEMS ENGINEERING                           7   \n",
       "1810                    PROGRESS IN NEUROBIOLOGY                         104   \n",
       "1811                ACM TRANSACTIONS ON GRAPHICS                          21   \n",
       "1812                         ACM SIGPLAN NOTICES                         322   \n",
       "\n",
       "                                  DOI   \n",
       "0                  10.1007/BF00197809  \\\n",
       "1                                 NaN   \n",
       "2          10.1038/s41586-021-03819-2   \n",
       "3           10.2337/diacare.27.5.1047   \n",
       "4                  10.1002/047174882X   \n",
       "...                               ...   \n",
       "1808             10.3390/app122412946   \n",
       "1809        10.1007/s12599-014-0363-z   \n",
       "1810  10.1016/j.pneurobio.2017.07.001   \n",
       "1811          10.1145/3272127.3275035   \n",
       "1812            10.1145/780822.781133   \n",
       "\n",
       "                                               Abstract   \n",
       "0     The NMRPipe system is a UNIX software environm...  \\\n",
       "1     Deep neural nets with a large number of parame...   \n",
       "2     Proteins are essential to life, and understand...   \n",
       "3     OBJECTIVE - The goal of this study was to esti...   \n",
       "4     The latest edition of this classic is updated ...   \n",
       "...                                                 ...   \n",
       "1808  The language inclusion checking of timed autom...   \n",
       "1809  Business models (BM) are the central concept t...   \n",
       "1810  Neurocognitive and neurolinguistics theories m...   \n",
       "1811  We introduce a novel framework for using natur...   \n",
       "1812  We present nesC, a programming language for ne...   \n",
       "\n",
       "                   Document Type  \n",
       "0                        Article  \n",
       "1                        Article  \n",
       "2                        Article  \n",
       "3                        Article  \n",
       "4                           Book  \n",
       "...                          ...  \n",
       "1808                     Article  \n",
       "1809                     Article  \n",
       "1810                      Review  \n",
       "1811  Article; Proceedings Paper  \n",
       "1812  Article; Proceedings Paper  \n",
       "\n",
       "[1813 rows x 7 columns]"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                Authors   \n",
      "0     Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....  \\\n",
      "1     Srivastava N.; Hinton G.; Krizhevsky A.; Sutsk...   \n",
      "2     Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...   \n",
      "3      Wild S.; Roglic G.; Green A.; Sicree R.; King H.   \n",
      "4                               Cover T.M.; Thomas J.A.   \n",
      "...                                                 ...   \n",
      "1808  Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...   \n",
      "1809                               Roelens, B; Poels, G   \n",
      "1810                                    Pulvermuller, F   \n",
      "1811  Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...   \n",
      "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
      "\n",
      "                                      Author Full Names   \n",
      "0     Delaglio, Frank (6603722428); Grzesiek, Stepha...  \\\n",
      "1     Srivastava, Nitish (55697473300); Hinton, Geof...   \n",
      "2     Jumper, John (36717725300); Evans, Richard (57...   \n",
      "3     Wild, Sarah (7103086309); Roglic, Gojka (67016...   \n",
      "4     Cover, Thomas M. (7102102743); Thomas, Joy A. ...   \n",
      "...                                                 ...   \n",
      "1808  Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...   \n",
      "1809                         Roelens, Ben; Poels, Geert   \n",
      "1810                          Pulvermueller, Friedemann   \n",
      "1811  Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...   \n",
      "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
      "\n",
      "                                    Source Title  Times Cited, All Databases   \n",
      "0                    Journal of Biomolecular NMR                       11644  \\\n",
      "1           Journal of Machine Learning Research                       25310   \n",
      "2                                         Nature                        6981   \n",
      "3                                  Diabetes Care                       12069   \n",
      "4                 Elements of Information Theory                       33728   \n",
      "...                                          ...                         ...   \n",
      "1808                      APPLIED SCIENCES-BASEL                           0   \n",
      "1809  BUSINESS & INFORMATION SYSTEMS ENGINEERING                           7   \n",
      "1810                    PROGRESS IN NEUROBIOLOGY                         104   \n",
      "1811                ACM TRANSACTIONS ON GRAPHICS                          21   \n",
      "1812                         ACM SIGPLAN NOTICES                         322   \n",
      "\n",
      "                                  DOI   \n",
      "0                  10.1007/BF00197809  \\\n",
      "1                                 NaN   \n",
      "2          10.1038/s41586-021-03819-2   \n",
      "3           10.2337/diacare.27.5.1047   \n",
      "4                  10.1002/047174882X   \n",
      "...                               ...   \n",
      "1808             10.3390/app122412946   \n",
      "1809        10.1007/s12599-014-0363-z   \n",
      "1810  10.1016/j.pneurobio.2017.07.001   \n",
      "1811          10.1145/3272127.3275035   \n",
      "1812            10.1145/780822.781133   \n",
      "\n",
      "                                               Abstract   \n",
      "0     The NMRPipe system is a UNIX software environm...  \\\n",
      "1     Deep neural nets with a large number of parame...   \n",
      "2     Proteins are essential to life, and understand...   \n",
      "3     OBJECTIVE - The goal of this study was to esti...   \n",
      "4     The latest edition of this classic is updated ...   \n",
      "...                                                 ...   \n",
      "1808  The language inclusion checking of timed autom...   \n",
      "1809  Business models (BM) are the central concept t...   \n",
      "1810  Neurocognitive and neurolinguistics theories m...   \n",
      "1811  We introduce a novel framework for using natur...   \n",
      "1812  We present nesC, a programming language for ne...   \n",
      "\n",
      "                   Document Type  \n",
      "0                        Article  \n",
      "1                        Article  \n",
      "2                        Article  \n",
      "3                        Article  \n",
      "4                           Book  \n",
      "...                          ...  \n",
      "1808                     Article  \n",
      "1809                     Article  \n",
      "1810                      Review  \n",
      "1811  Article; Proceedings Paper  \n",
      "1812  Article; Proceedings Paper  \n",
      "\n",
      "[1813 rows x 7 columns]\n",
      "(1813, 7)\n",
      "Index(['Authors', 'Author Full Names', 'Source Title',\n",
      "       'Times Cited, All Databases', 'DOI', 'Abstract', 'Document Type'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "source": [
    "print(dataframe)\n",
    "print(dataframe.shape)\n",
    "\n",
    "# Publication Type\n",
    "# Addresses\n",
    "# Reprint Addresses\t\n",
    "# DOI Link\t\n",
    "# Times Cited, WoS Core\n",
    "# Date of Export\n",
    "#UT (Unique WOS ID)\n",
    "#Web of Science Record\n",
    "\n",
    "print(dataframe.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1813, 7)\n",
      "(1811, 7)\n"
     ]
    }
   ],
   "source": [
    "print(dataframe.shape)\n",
    "dataframe = dataframe.drop_duplicates(subset=['DOI'])\n",
    "print(dataframe.shape)\n",
    "\n",
    "# contar los valores nulos en DOI\n",
    "dataframe['DOI'].isnull().sum()\n",
    "\n",
    "# realizar un drop de los valores nulos en DOI\n",
    "dataframe = dataframe.dropna(subset=['DOI'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Times Cited, All Databases</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>1810.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mean</th>\n",
       "      <td>1428.613260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>std</th>\n",
       "      <td>3002.283907</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25%</th>\n",
       "      <td>4.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50%</th>\n",
       "      <td>95.500000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75%</th>\n",
       "      <td>1793.750000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>max</th>\n",
       "      <td>44908.000000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       Times Cited, All Databases\n",
       "count                 1810.000000\n",
       "mean                  1428.613260\n",
       "std                   3002.283907\n",
       "min                      0.000000\n",
       "25%                      4.000000\n",
       "50%                     95.500000\n",
       "75%                   1793.750000\n",
       "max                  44908.000000"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataframe.describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Authors</th>\n",
       "      <th>Author Full Names</th>\n",
       "      <th>Source Title</th>\n",
       "      <th>Times Cited, All Databases</th>\n",
       "      <th>DOI</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Document Type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Lecun Y.; Bengio Y.; Hinton G.</td>\n",
       "      <td>Lecun, Yann (55666793600); Bengio, Yoshua (700...</td>\n",
       "      <td>Nature</td>\n",
       "      <td>44908</td>\n",
       "      <td>10.1038/nature14539</td>\n",
       "      <td>Deep learning allows computational models that...</td>\n",
       "      <td>Review</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           Authors   \n",
       "12  Lecun Y.; Bengio Y.; Hinton G.  \\\n",
       "\n",
       "                                    Author Full Names Source Title   \n",
       "12  Lecun, Yann (55666793600); Bengio, Yoshua (700...       Nature  \\\n",
       "\n",
       "    Times Cited, All Databases                  DOI   \n",
       "12                       44908  10.1038/nature14539  \\\n",
       "\n",
       "                                             Abstract Document Type  \n",
       "12  Deep learning allows computational models that...        Review  "
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "most_cited_paper = dataframe[dataframe['Times Cited, All Databases'] == dataframe['Times Cited, All Databases'].max()]\n",
    "most_cited_paper\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABCAAAAK+CAYAAABzUG9sAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAC0E0lEQVR4nOzdd3xP5///8eebyJbEzLASe49q7VKE2HurVaXDCjVqj2q1dpVSNUKr1F4ltWlJY0aL1ogYRaKKRKggef/+8Mv5ekuQ4HxS+rjfbu/bTa5zneu8znmnn9vnPHPOdVmsVqtVAAAAAAAAJkqX1gUAAAAAAICXHwEEAAAAAAAwHQEEAAAAAAAwHQEEAAAAAAAwHQEEAAAAAAAwHQEEAAAAAAAwHQEEAAAAAAAwHQEEAAAAAAAwHQEEAAAAXlhXrlzR6NGjtXfv3rQuBQDwBAQQAAAAeCFZrVZ16tRJO3fuVOnSpf+nx7ZYLBo1atT/5Fi+vr7q3Lnz/+RYAGAmAggAAJDmLBZLij47duwwvZaZM2eqZcuWyp07tywWy2Nv/K5fv67u3bsrW7ZscnFxUfXq1XXw4EHTa8R9EydO1NmzZ7Vq1SrZ29unaS179uzRqFGjdP369TStAwD+zezSugAAAIBvvvnG5ueFCxdq8+bNSdqLFCliei2fffaZbty4oXLlyunSpUuP7JeQkKD69evr8OHDGjBggLJmzaovv/xSb7zxhg4cOKACBQqYXut/WVxcnO7cuaMNGzbI3d39f378f/75R3Z2//d/pffs2aPRo0erc+fO8vDw+J/XAwAvAgIIAACQ5t58802bn3/55Rdt3rw5Sfv/ws6dO42nH1xdXR/Zb/ny5dqzZ4+WLVumFi1aSJJatWqlggULauTIkfruu+/+VyX/T928eVMuLi5pXYYcHBw0dOjQNDu+o6Njmh0bAF5UvIIBAABeCDdv3tQHH3ygXLlyycHBQYUKFdLEiRNltVpt+lksFvXs2VOLFi1SoUKF5OjoqLJly2rXrl0pOk6ePHlksVie2G/58uXy9PRUs2bNjLZs2bKpVatWWrNmjeLi4h67v6+vrxo0aKBNmzapdOnScnR0VNGiRbVy5UqbflevXlX//v1VokQJubq6ys3NTXXr1tXhw4dt+u3YsUMWi0Xff/+9hgwZIi8vL7m4uKhRo0Y6f/58kuOHhoaqTp06cnd3l7Ozs6pVq6bdu3fb9Bk1apQsFouOHTumdu3aKVOmTKpSpYokKTIyUl26dFHOnDnl4OAgb29vNW7cWGfOnHnseXfu3Fmurq46d+6cGjRoIFdXV+XIkUMzZsyQJP3222+qUaOGXFxclCdPniRBTkqvhyR98cUXKlasmJydnZUpUya9+uqrKQqGbt++rVGjRqlgwYJydHSUt7e3mjVrpvDwcKPPg3NAjBo1SgMGDJAk+fn5Ga8MJV6L+fPnq0aNGsqePbscHBxUtGhRzZw5M8lxrVarxo4dq5w5c8rZ2VnVq1fX0aNHk63x9OnTatmypTJnzixnZ2dVqFBBP/zww3O7BgBgBp6AAAAA/3pWq1WNGjXS9u3b1bVrV5UuXVo//vijBgwYoAsXLmjKlCk2/Xfu3Knvv/9evXv3loODg7788kvVqVNHe/fuVfHixZ9LTYcOHdIrr7yidOls/55Trlw5zZ49WydOnFCJEiUeO8bJkyfVunVrvfvuu+rUqZPmz5+vli1bKjg4WLVq1ZJ0/0Zz9erVatmypfz8/BQVFaWvvvpK1apV07Fjx+Tj42Mz5scffyyLxaJBgwbp8uXLmjp1qvz9/RUWFiYnJydJ0rZt21S3bl2VLVtWI0eOVLp06Yyb5J9++knlypWzGbNly5YqUKCAPvnkEyPwad68uY4ePapevXrJ19dXly9f1ubNm3Xu3Dn5+vo+9rzj4+NVt25dVa1aVePHj9eiRYvUs2dPubi4aOjQoWrfvr2aNWumWbNmqWPHjqpYsaL8/PyM67Fq1Sq1atXKuB4zZ85Mcj2+/vpr9e7dWy1atFCfPn10+/Zt/frrrwoNDVW7du0eW1uDBg20detWtWnTRn369NGNGze0efNmHTlyRPny5UuyT7NmzXTixAktXrxYU6ZMUdasWSXdD6Sk+/OKFCtWTI0aNZKdnZ3WrVun999/XwkJCerRo4cxzogRIzR27FjVq1dP9erV08GDB1W7dm3duXPH5nhRUVGqVKmSbt26pd69eytLlixasGCBGjVqpOXLl6tp06bPdA0AwDRWAACAf5kePXpYH/y/KatXr7ZKso4dO9amX4sWLawWi8V66tQpo02SVZJ1//79RtvZs2etjo6O1qZNm6aqDhcXF2unTp0eue2tt95K0v7DDz9YJVmDg4MfO3aePHmskqwrVqww2qKjo63e3t7WMmXKGG23b9+2xsfH2+wbERFhdXBwsI4ZM8Zo2759u1WSNUeOHNaYmBijfenSpVZJ1s8//9xqtVqtCQkJ1gIFClgDAgKsCQkJRr9bt25Z/fz8rLVq1TLaRo4caZVkbdu2rc3xr127ZpVknTBhwmPPMTmdOnWySrJ+8sknNuM5OTlZLRaLdcmSJUb7H3/8YZVkHTlypNH2zz//WO/du2czZnh4eJLr0bhxY2uxYsVSXd+8efOskqyTJ09Osu3B6/VwXRMmTLBKskZERCTZ79atW0naAgICrHnz5jV+vnz5stXe3t5av359m+MMGTLEKsnm9zAwMNAqyfrTTz8ZbTdu3LD6+flZfX19jd+Xp70GAGAWXsEAAAD/ehs2bFD69OnVu3dvm/YPPvhAVqtVGzdutGmvWLGiypYta/ycO3duNW7cWD/++KPi4+OfS03//POPHBwckrQnzg3wzz//PHEMHx8f46/VkuTm5qaOHTvq0KFDioyMlHR/roPEpyzi4+P1999/y9XVVYUKFUp2xY2OHTsqY8aMxs8tWrSQt7e3NmzYIEkKCwvTyZMn1a5dO/3999+6cuWKrly5ops3b6pmzZratWuXEhISbMZ89913bX52cnKSvb29duzYoWvXrj3xPJPz9ttvG//28PBQoUKF5OLiolatWhnthQoVkoeHh06fPm20OTo6Kn369MbPcXFx8vHxUZEiRWyuh4eHh/7880/t27cvVXWtWLFCWbNmVa9evZJsS8mrOclJfPJEkqKjo3XlyhVVq1ZNp0+fVnR0tCRpy5YtunPnjnr16mVznMDAwCTjbdiwQeXKlTNeh5EkV1dXde/eXWfOnNGxY8ckPf01AACzEEAAAIB/vbNnz8rHx8fmxlr6v1Uxzp49a9Oe3AoUBQsW1K1bt/TXX389l5qcnJySnefh9u3bxvYnyZ8/f5Kb2oIFC0qSMX9AQkKCpkyZogIFCsjBwUFZs2ZVtmzZ9Ouvvxo3rw96+NwtFovy589vjHfy5ElJUqdOnZQtWzabz5w5cxQXF5dk3MTXHxI5ODjos88+08aNG+Xp6Wm8SpEYmjyJo6Oj8XpCInd3d+XMmTPJ9XB3d7cJOaxWq2bNmqXSpUvL1dVVjo6OcnJyUlhYmE3dgwYNkqurq8qVK6cCBQqoR48eSea4SE54eLgKFSpks8LFs9q9e7f8/f3l4uIiDw8PZcuWTUOGDJEko+bE3+GHv79s2bIpU6ZMNm1nz55VoUKFkhzn4f8envYaAIBZCCAAAACegre3d7LLdCa2PTw3w9P65JNP1K9fP1WtWlXffvutfvzxR23evFnFihVL8qRCSiTuM2HCBG3evDnZz8OrfyQXpgQGBurEiRMaN26cHB0dNXz4cBUpUkSHDh16Yg0PPsGQknbrAxONfvbZZ3rvvfdUpUoVLV68WLt371ZISIhKlChhcz2KFCmi48ePa8mSJapSpYpWrFihKlWqaOTIkU+s73kKDw9XzZo1deXKFU2ePFk//PCDNm/erL59+0rSU32HKfVvuQYAkIhJKAEAwL9enjx5tGXLFt24ccPmKYg//vjD2P6gxL/yP+jEiRNydnZO8pf3p1W6dGn99NNPSkhIsJmIMjQ0VM7OzsaTDI9z6tQpWa1Wm7/6nzhxQpKMiRyXL1+u6tWra+7cuTb7Xr9+3Zjs8EEPn7vVatWpU6dUsmRJSTImUXRzc5O/v38KzvTR8uXLpw8++EAffPCBTp48qdKlS2vSpEn69ttvn2ncx/n+++/l7++v6dOn27RfuXJFmTNntmlzcXFR69at1bp1a925c0fNmjXTxx9/rMGDBz9yGc18+fIpNDRUd+/eVYYMGVJc16Nez1i3bp3i4uK0du1a5c6d22jfvn27Tb/E3+GTJ08qb968Rvtff/2V5DWXPHny6Pjx40mOldx/D09zDQDALDwBAQAA/vXq1aun+Pj4JDedU6ZMkcViUd26dW3aQ0JCbOYDOH/+vNasWaPatWs/8q/sqdWiRQtFRUXZLJt55coVLVu2TA0bNkx2foiHXbx4UatWrTJ+jomJ0cKFC1W6dGl5eXlJuv9UgPWhpUaXLVumCxcuJDvmwoULdePGDePn5cuX69KlS8Y1Klu2rPLly6eJEycqNjY2yf4peUXl1q1bxqsmifLly6eMGTM+cfnRZ2WxWHT37l2btsWLFyd5GuXvv/+2+dne3l5FixaV1WpNsv+DmjdvritXriT5XZOU5Ht4kIuLi6T7wdCDEn/fHtw3Ojpa8+fPt+nn7++vDBky6IsvvrDpO3Xq1CTHqlevnvbu3auQkBCj7ebNm5o9e7Z8fX1VtGhRSU9/DQDALDwBAQAA/vUaNmyo6tWra+jQoTpz5oxKlSqlTZs2ac2aNQoMDEyyNGLx4sUVEBBgswynJI0ePfqJx1q3bp0OHz4sSbp7965+/fVXjR07VpLUqFEj40mCFi1aqEKFCurSpYuOHTumrFmz6ssvv1R8fHyKjiPdn++ha9eu2rdvnzw9PTVv3jxFRUXZ3Jw2aNBAY8aMUZcuXVSpUiX99ttvWrRokc1fyR+UOXNmValSRV26dFFUVJSmTp2q/Pnzq1u3bpKkdOnSac6cOapbt66KFSumLl26KEeOHLpw4YK2b98uNzc3rVu37rF1nzhxQjVr1lSrVq1UtGhR2dnZadWqVYqKilKbNm1SdO5Pq379+ho7dqy6dOmiihUr6rffftN3332X5Hegdu3a8vLyUuXKleXp6anff/9d06dPV/369ZPMJfKgjh07auHCherXr5/27t2r119/XTdv3tSWLVv0/vvvq3Hjxsnulzjp6dChQ9WmTRtlyJBBDRs2VO3atWVvb6+GDRvqnXfeUWxsrL7++mtlz57dJjTJli2b+vfvr3HjxqlBgwaqV6+eDh06pI0bNyZ50uXDDz/U4sWLVbduXfXu3VuZM2fWggULFBERoRUrVhhP5DztNQAA06TR6hsAAACP9PAynFbr/WUG+/bta/Xx8bFmyJDBWqBAAeuECRNsliy0Wu8vj9ijRw/rt99+ay1QoIDVwcHBWqZMGev27dtTdOzEZSKT+8yfP9+m79WrV61du3a1ZsmSxers7GytVq2add++fSk6Tp48eaz169e3/vjjj9aSJUtaHRwcrIULF7YuW7bMpt/t27etH3zwgdXb29vq5ORkrVy5sjUkJMRarVo1a7Vq1Yx+ictwLl682Dp48GBr9uzZrU5OTtb69etbz549m+T4hw4dsjZr1syaJUsWq4ODgzVPnjzWVq1aWbdu3Wr0SVyG86+//rLZ98qVK9YePXpYCxcubHVxcbG6u7tby5cvb126dOkTz7tTp05WFxeXJO3VqlVLdsnIxOv04PUIDAy0ent7W52dna2vv/66de/evUmux1dffWWtWrWqcX758uWzDhgwwBodHf3EGm/dumUdOnSo1c/Pz5ohQwarl5eXtUWLFtbw8HCjjx5ahtNqtVo/+ugja44cOazp0qWzWZJz7dq11pIlS1odHR2tvr6+1s8++8xY7vPBZTvj4+Oto0ePNr7rN954w3rkyBFrnjx5kiwHGx4ebm3RooXVw8PD6ujoaC1Xrpx1/fr1Nn2e5RoAgBksVutjniUDAAB4wVgsFvXo0SPZR+j/TXx9fVW8eHGtX7/+uYy3Y8cOVa9eXcuWLVOLFi2ey5gAADxPzAEBAAAAAABMRwABAAAAAABMRwABAAAAAABMxxwQAAAAAADAdDwBAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATGeX1gUAeDElJCTo4sWLypgxoywWS1qXAwAAACCNWK1W3bhxQz4+PkqX7tHPORBAAHgqFy9eVK5cudK6DAAAAAD/EufPn1fOnDkfuZ0AAsBTyZgxo6T7/yPj5uaWxtUAAAAASCsxMTHKlSuXcY/wKAQQAJ5K4msXbm5uBBAAAAAAnvhqNpNQAgAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0zEHBIBnUnXYYqV3cErrMgAAAID/jAMTOqZ1CU+FJyAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCBecLt27VLDhg3l4+Mji8Wi1atX22y3Wq0aMWKEvL295eTkJH9/f508efKZxkzpuI0aNVLu3Lnl6Ogob29vdejQQRcvXrTp8+OPP6pChQrKmDGjsmXLpubNm+vMmTNJjrdgwQJVqVIlRdckJUaNGqXSpUs/t/FSonfv3ipbtqwcHByeeOxTp04pY8aM8vDwsGkPCgqSxWKx+Tg6Otr0GTVqlAoXLiwXFxdlypRJ/v7+Cg0Ntelz8OBB1apVSx4eHsqSJYu6d++u2NjY53GaAAAAAJAsAogX3M2bN1WqVCnNmDEj2e3jx4/XtGnTNGvWLIWGhsrFxUUBAQG6ffv2U4+Z0nGrV6+upUuX6vjx41qxYoXCw8PVokULY3tERIQaN26sGjVqKCwsTD/++KOuXLmiZs2aJTnemjVr1KhRo5Rckv8pq9Wqe/fupbj/W2+9pdatWz+2z927d9W2bVu9/vrryW53c3PTpUuXjM/Zs2dtthcsWFDTp0/Xb7/9pp9//lm+vr6qXbu2/vrrL0nSxYsX5e/vr/z58ys0NFTBwcE6evSoOnfunOLzAAAAAIDUIoB4wdWtW1djx45V06ZNk2yzWq2aOnWqhg0bpsaNG6tkyZJauHChLl68mOxTDSkZMzXj9u3bVxUqVFCePHlUqVIlffjhh/rll1909+5dSdKBAwcUHx+vsWPHKl++fHrllVfUv39/hYWFGX0k6fbt29q0aZMRQPj6+uqTTz7RW2+9pYwZMyp37tyaPXu2TY2DBg1SwYIF5ezsrLx582r48OHGmEFBQRo9erQOHz5sPEUQFBSkM2fOyGKxKCwszBjn+vXrslgs2rFjhyRpx44dslgs2rhxo/E0w88//6yEhASNGzdOfn5+cnJyUqlSpbR8+XKbmqZNm6YePXoob968j7z2kjRs2DAVLlxYrVq1Sna7xWKRl5eX8fH09LTZ3q5dO/n7+ytv3rwqVqyYJk+erJiYGP3666+SpPXr1ytDhgyaMWOGChUqpNdee02zZs3SihUrdOrUqcfWBgAAAABPiwDiJRYREaHIyEj5+/sbbe7u7ipfvrxCQkL+p+NevXpVixYtUqVKlZQhQwZJUtmyZZUuXTrNnz9f8fHxio6O1jfffCN/f3+jjyRt3bpVOXLkUOHChY22SZMm6dVXX9WhQ4f0/vvv67333tPx48eN7RkzZlRQUJCOHTumzz//XF9//bWmTJkiSWrdurU++OADFStWzHiK4ElPJTzsww8/1Keffqrff/9dJUuW1Lhx47Rw4ULNmjVLR48eVd++ffXmm29q586dqRp327ZtWrZs2WOfPomNjVWePHmUK1cuNW7cWEePHn1k3zt37mj27Nlyd3dXqVKlJElxcXGyt7dXunT/95+/k5OTJOnnn39+5FhxcXGKiYmx+QAAAABAShFAvMQiIyMlKclfyD09PY1tZo87aNAgubi4KEuWLDp37pzWrFljbPPz89OmTZs0ZMgQOTg4yMPDQ3/++aeWLl1qM0Zyr1/Uq1dP77//vvLnz69BgwYpa9as2r59u7F92LBhqlSpknx9fdWwYUP179/fGNfJyUmurq6ys7MzniJIvAFPqTFjxqhWrVrKly+fXFxc9Mknn2jevHkKCAhQ3rx51blzZ7355pv66quvUjzm33//rc6dOysoKEhubm7J9ilUqJDmzZunNWvW6Ntvv1VCQoIqVaqkP//806bf+vXr5erqKkdHR02ZMkWbN29W1qxZJUk1atRQZGSkJkyYoDt37ujatWv68MMPJUmXLl16ZH3jxo2Tu7u78cmVK1eKzw0AAAAACCBgqgEDBujQoUPatGmT0qdPr44dO8pqtUq6H2R069ZNnTp10r59+7Rz507Z29urRYsWRh+r1ap169YlCSBKlixp/DvxlYTLly8bbd9//70qV64sLy8vubq6atiwYTp37txzO69XX33V+PepU6d069Yt1apVS66ursZn4cKFCg8PT/GY3bp1U7t27VS1atVH9qlYsaI6duyo0qVLq1q1alq5cqWyZcuWJOioXr26wsLCtGfPHtWpU0etWrUyrk+xYsW0YMECTZo0Sc7OzvLy8pKfn588PT1tnop42ODBgxUdHW18zp8/n+JzAwAAAAC7tC4A5vHy8pIkRUVFydvb22iPiop6phUgUjNu1qxZlTVrVhUsWFBFihRRrly59Msvv6hixYqaMWOG3N3dNX78eKP/t99+q1y5cik0NFQVKlTQ3r17de/ePVWqVMlm3Adf0ZDuhxAJCQmSpJCQELVv316jR49WQECA3N3dtWTJEk2aNOmx55V4850YfkiymYviQS4uLsa/E1eP+OGHH5QjRw6bfg4ODo895oO2bdumtWvXauLEiUYdCQkJsrOz0+zZs/XWW28l2SdDhgwqU6ZMkrkbXFxclD9/fuXPn18VKlRQgQIFNHfuXA0ePFjS/Xki2rVrp6ioKLm4uMhisWjy5MmPnZ/CwcEhVecDAAAAAA8igHiJ+fn5ycvLS1u3bjWCgZiYGIWGhuq99977n4+bGBDExcVJkm7dupXkL+7p06e36btmzRrVr1/faE+JPXv2KE+ePBo6dKjR9vBKEfb29oqPj7dpy5Ytm6T7ryGUKVNGkmwmpHyUokWLysHBQefOnVO1atVSXOfDQkJCbGpas2aNPvvsM+3ZsydJsJEoPj5ev/32m+rVq/fYsRMSEozr/qDE12jmzZsnR0dH1apV66nrBwAAAIDHIYB4wcXGxtr89TsiIkJhYWHKnDmzcufOrcDAQI0dO1YFChSQn5+fhg8fLh8fHzVp0sTYp2bNmmratKl69uyZojEtFssTxw0NDdW+fftUpUoVZcqUSeHh4Ro+fLjy5cunihUrSpLq16+vKVOmaMyYMWrbtq1u3LihIUOGKE+ePEYAsHbtWo0ZMyZV16RAgQI6d+6clixZotdee00//PCDVq1aZdPH19fXOK+cOXMqY8aMcnJyUoUKFfTpp5/Kz89Ply9f1rBhw554vIwZM6p///7q27evEhISVKVKFUVHR2v37t1yc3NTp06dJN1/VSM2NlaRkZH6559/jHCjaNGisre3V5EiRWzG3b9/v9KlS6fixYsbbWPGjFGFChWUP39+Xb9+XRMmTNDZs2f19ttvS7q/hOrHH3+sRo0aydvbW1euXNGMGTN04cIFtWzZ0hhn+vTpqlSpklxdXbV582YNGDBAn376qTw8PFJ1rQEAAAAgpQggXnD79+9X9erVjZ/79esnSerUqZOCgoI0cOBA3bx5U927d9f169dVpUoVBQcHy9HR0dgnPDxcV65cSfGYkp44rrOzs1auXKmRI0fq5s2b8vb2Vp06dTRs2DDjMf4aNWrou+++0/jx4zV+/Hg5OzurYsWKCg4OlpOTk8LDw3Xq1CkFBASk6po0atRIffv2Vc+ePRUXF6f69etr+PDhGjVqlNGnefPmWrlypapXr67r169r/vz56ty5s+bNm6euXbuqbNmyKlSokMaPH6/atWs/8ZgfffSRsmXLpnHjxun06dPy8PDQK6+8oiFDhhh93n77bZtVMRJDloiICPn6+qbo3K5du6Zu3bopMjJSmTJlUtmyZbVnzx4VLVpU0v0nSP744w8tWLBAV65cUZYsWfTaa6/pp59+UrFixYxx9u7dq5EjRyo2NlaFCxfWV199pQ4dOqSoBgAAAAB4Ghbrgy+8A/8ikydP1pYtW7Rhw4a0LgXJiImJub+8Z69ZSu+QulVEAAAAADy9AxM6pnUJNhLvDaKjox+5op/EKhj4F8uZM6cxaSIAAAAA4MXGKxj412rVqlValwAAAAAAeE54AgIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJiOAAIAAAAAAJjOLq0LAPBi2zW2rdzc3NK6DAAAAAD/cjwBAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATGeX1gUAeLGd/7SCMjqmT+syAAAA8B+We8RvaV0CUoAnIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIAAAAAAAgOkIIPBS8fX11dSpU9O6jP+poKAgeXh4pHUZAAAAAPBYBBAvqF27dqlhw4by8fGRxWLR6tWrbbZbrVaNGDFC3t7ecnJykr+/v06ePPlMY0pSbGysevbsqZw5c8rJyUlFixbVrFmzkvQLCQlRjRo15OLiIjc3N1WtWlX//POPsb1Ro0bKnTu3HB0d5e3trQ4dOujixYvG9lGjRslisST5uLi4JDnW6NGj9eabb0qS9u3bp+7duz/2PM1y9epVtW/fXm5ubvLw8FDXrl0VGxtr08dqtWrixIkqWLCgHBwclCNHDn388cfG9s6dOyd73sWKFXvkcVu3bq0TJ07YtO3YsUOvvPKKHBwclD9/fgUFBdlsT8l3DQAAAADPEwHEC+rmzZsqVaqUZsyYkez28ePHa9q0aZo1a5ZCQ0Pl4uKigIAA3b59+6nHlKR+/fopODhY3377rX7//XcFBgaqZ8+eWrt2rdEnJCREderUUe3atbV3717t27dPPXv2VLp0//frVr16dS1dulTHjx/XihUrFB4erhYtWhjb+/fvr0uXLtl8ihYtqpYtWyapac2aNWrUqJEkKVu2bHJ2dn70hTNR+/btdfToUW3evFnr16/Xrl27koQhffr00Zw5czRx4kT98ccfWrt2rcqVK2ds//zzz23O+fz588qcOXOy553IyclJ2bNnN36OiIhQ/fr1Vb16dYWFhSkwMFBvv/22fvzxR6NPSr5rAAAAAHieLFar1ZrWReDZWCwWrVq1Sk2aNJF0/6/sPj4++uCDD9S/f39JUnR0tDw9PRUUFKQ2bdqkesxExYsXV+vWrTV8+HCjrWzZsqpbt67Gjh0rSapQoYJq1aqljz76KMXnsHbtWjVp0kRxcXHKkCFDku2HDx9W6dKltWvXLr3++utG+/nz55U/f3799ddfcnNzk6+vrwIDAxUYGCir1arRo0dr3rx5ioqKUpYsWdSiRQtNmzZNkvTll19qypQpOn/+vNzd3fX6669r+fLlSY4dExMjT09PrVy5UnXr1jXaV61apY4dOyoqKkpnz55V0aJFtW/fPr366quSpODgYNWrV09//vmnfHx89Pvvv6tkyZI6cuSIChUqlKLrsnr1ajVr1kwRERHKkydPsn2CgoIUGBio69evS5IGDRqkH374QUeOHDH6tGnTRtevX1dwcHCS/R/1XT9JTEyM3N3ddWRwEWV0TJ+qfQEAAIDnKfeI39K6hP+0xHuD6Ohoubm5PbIfT0C8hCIiIhQZGSl/f3+jzd3dXeXLl1dISMgzjV2pUiWtXbtWFy5ckNVq1fbt23XixAnVrl1bknT58mWFhoYqe/bsqlSpkjw9PVWtWjX9/PPPjxzz6tWrWrRokSpVqpRs+CBJc+bMUcGCBW3CB+l+cPHGG28k+0u+YsUKTZkyRV999ZVOnjyp1atXq0SJEpKk/fv3q3fv3hozZoyOHz+u4OBgVa1aNdlju7m5qUGDBvruu+9s2hctWqQmTZrI2dlZISEh8vDwMMIHSfL391e6dOkUGhoqSVq3bp3y5s2r9evXy8/PT76+vnr77bd19erVR16buXPnyt/f/5HhQ3JCQkJsvntJCggIeObvPi4uTjExMTYfAAAAAEgpAoiXUGRkpCTJ09PTpt3T09PY9rS++OILFS1aVDlz5pS9vb3q1KmjGTNmGDfvp0+flnR/Dodu3bopODhYr7zyimrWrJlkDopBgwbJxcVFWbJk0blz57RmzZpkj3n79m0tWrRIXbt2TbLtwdcvHnbu3Dl5eXnJ399fuXPnVrly5dStWzdjm4uLixo0aKA8efKoTJky6t279yPPu3379lq9erVu3bol6X7C98MPP6h9+/aS7l/zB1+DkCQ7OztlzpzZuOanT5/W2bNntWzZMi1cuFBBQUE6cOCAzasnD7p48aI2btyot99++5F1JScyMjLZ7z4mJsZmHo7UGjdunNzd3Y1Prly5nnosAAAAAP89BBBIlS+++EK//PKL1q5dqwMHDmjSpEnq0aOHtmzZIklKSEiQJL3zzjvq0qWLypQpoylTpqhQoUKaN2+ezVgDBgzQoUOHtGnTJqVPn14dO3ZUcm8ErVq1Sjdu3FCnTp1s2mNiYrRz585HBhAtW7bUP//8o7x586pbt25atWqV7t27J0mqVauW8uTJo7x586pDhw5atGiRES4kp169esqQIYMx18WKFSvk5uaW5EmDx0lISFBcXJwWLlyo119/XW+88Ybmzp2r7du36/jx40n6L1iwQB4eHql+NcIsgwcPVnR0tPE5f/58WpcEAAAA4AVCAPES8vLykiRFRUXZtEdFRRnbnsY///yjIUOGaPLkyWrYsKFKliypnj17qnXr1po4caIkydvbW5JUtGhRm32LFCmic+fO2bRlzZpVBQsWVK1atbRkyRJt2LBBv/zyS5LjzpkzRw0aNEjyV/2NGzeqaNGij/xLfK5cuXT8+HF9+eWXcnJy0vvvv6+qVavq7t27ypgxow4ePKjFixfL29tbI0aMUKlSpYx5FB5mb2+vFi1aGK9hfPfdd2rdurXs7Owk3b/mly9fttnn3r17unr1qnHNvb29ZWdnp4IFC9pcF0lJro3VatW8efPUoUMH2dvbJ1vTo3h5eSX73bu5ucnJySlVYz3IwcFBbm5uNh8AAAAASCkCiJeQn5+fvLy8tHXrVqMtJiZGoaGhqlix4lOPe/fuXd29e9dmNQtJSp8+vfHkg6+vr3x8fJL8Rf/EiROPnccgcf+4uDib9oiICG3fvv2Rr180btz4sTU7OTmpYcOGmjZtmnbs2KGQkBD99tv9CWrs7Ozk7++v8ePH69dff9WZM2e0bdu2R47Vvn17BQcH6+jRo9q2bZvx+oUkVaxYUdevX9eBAweMtm3btikhIUHly5eXJFWuXFn37t1TeHi40Sdx+cyHr83OnTt16tSpZM/7SSpWrGjz3UvS5s2bn+m7BwAAAIBnZZfWBeDpxMbG6tSpU8bPERERCgsLU+bMmZU7d24FBgZq7NixKlCggPz8/DR8+HD5+PjYPM5fs2ZNNW3aVD179kzRmG5ubqpWrZoGDBggJycn5cmTRzt37tTChQs1efJkSfdXVBgwYIBGjhypUqVKqXTp0lqwYIH++OMPY4WJ0NBQ7du3T1WqVFGmTJkUHh6u4cOHK1++fElukufNmydvb2+b1Sek+08XbNy40VjlIzlBQUGKj49X+fLl5ezsrG+//daoe/369Tp9+rSqVq2qTJkyacOGDUpISDBWp5g+fbpWrVplcyNftWpVeXl5qX379vLz8zOCBen+kwx16tRRt27dNGvWLN29e1c9e/ZUmzZt5OPjI+n+pJSvvPKK3nrrLU2dOlUJCQnq0aOHatWqZfNUhHR/8sny5curePHiSc4rudoe9O6772r69OkaOHCg3nrrLW3btk1Lly7VDz/8YPR50ncNAAAAAM8bAcQLav/+/apevbrxc79+/SRJnTp1UlBQkAYOHKibN2+qe/fuun79uqpUqaLg4GA5Ojoa+4SHh+vKlSspHlOSlixZosGDB6t9+/a6evWq8uTJo48//ljvvvuusV9gYKBu376tvn376urVqypVqpQ2b96sfPnySZKcnZ21cuVKjRw5Ujdv3pS3t7fq1KmjYcOGycHBwRgnISFBQUFB6ty5s9Knt13mcefOnXJ1ddUrr7zyyGvk4eGhTz/9VP369VN8fLxKlCihdevWKUuWLPLw8NDKlSs1atQo3b59WwUKFNDixYtVrFgxSdKVK1dsnlSQ7ocrbdu21fjx4zVixIgkx1u0aJF69uypmjVrKl26dGrevLmx5KckpUuXTuvWrVOvXr1UtWpVubi4qG7dupo0aZLNONHR0VqxYoU+//zzZM8rudoe5Ofnpx9++EF9+/bV559/rpw5c2rOnDkKCAgw+qTkuwYAAACA58liTW7WP+Bfrnfv3rp3756+/PJLm3Zvb2999NFHqV45AqmXuNbvkcFFlNEx/ZN3AAAAAEySe8RvaV3Cf1rivUF0dPRj54rjCQi8kIoXL27zusatW7e0e/duRUVFGU8xAAAAAAD+PQgg8ELq3r27zc+zZ8/WRx99pMDAQCZbBAAAAIB/IQIIvBQCAwMVGBiY1mUAAAAAAB6BZTgBAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDpCCAAAAAAAIDp7NK6AAAvtlwf/iI3N7e0LgMAAADAvxxPQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANMRQAAAAAAAANPZpXUBAF5stWbVkp0T/1MCAAD+XXb32p3WJQB4CE9AAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAvAR27dqlhg0bysfHRxaLRatXr7bZbrVaNWLECHl7e8vJyUn+/v46efLkM42Z0nGvXr2q9u3by83NTR4eHuratatiY2ON7Tt27FDjxo3l7e0tFxcXlS5dWosWLUq2ptGjR+vNN9+UJN2+fVs9evRQlixZ5OrqqubNmysqKirZ/f7++2/lzJlTFotF169fN9o7d+4si8WS5FOsWLFUXYfkxrBYLJowYUKSvnFxcSpdurQsFovCwsJstv3444+qUKGCMmbMqGzZsql58+Y6c+aMTZ9FixapVKlScnZ2lre3t9566y39/fffxvavv/5ar7/+ujJlyqRMmTLJ399fe/fuNbbfvHlT+fLlU79+/WzGPXPmjNzc3PT1118new0BAAAA4FkRQLwEbt68qVKlSmnGjBnJbh8/frymTZumWbNmKTQ0VC4uLgoICNDt27efesyUjtu+fXsdPXpUmzdv1vr167Vr1y51797d2L5nzx6VLFlSK1as0K+//qouXbqoY8eOWr9+fZLjrVmzRo0aNZIk9e3bV+vWrdOyZcu0c+dOXbx4Uc2aNUu2zq5du6pkyZJJ2j///HNdunTJ+Jw/f16ZM2dWy5YtU3UdHhzj0qVLmjdvniwWi5o3b56k78CBA+Xj45OkPSIiQo0bN1aNGjUUFhamH3/8UVeuXLE5p927d6tjx47q2rWrjh49qmXLlmnv3r3q1q2b0WfHjh1q27attm/frpCQEOXKlUu1a9fWhQsXJEkuLi6aP3++vvjiC/3000+S7gdJXbp0UeXKlW3GAgAAAIDnyWK1Wq1pXQSeH4vFolWrVqlJkyaS7t9c+vj46IMPPlD//v0lSdHR0fL09FRQUJDatGmT6jFTOu7vv/+uokWLat++fXr11VclScHBwapXr57+/PPPZG/EJal+/fry9PTUvHnzjLbz588rf/78+uuvv2S1WpUtWzZ99913atGihSTpjz/+UJEiRRQSEqIKFSoY+82cOVPff/+9RowYoZo1a+ratWvy8PBI9rirV69Ws2bNFBERoTx58qToOiSnSZMmunHjhrZu3WrTvnHjRvXr108rVqxQsWLFdOjQIZUuXVqStHz5crVt21ZxcXFKl+5+Lrhu3To1btxYcXFxypAhgyZOnKiZM2cqPDzcGPOLL77QZ599pj///DPZWuLj45UpUyZNnz5dHTt2NNr79euntWvX6vDhw/r66681evRoHTlyRDly5HjsuT0oJiZG7u7uKvdZOdk52aV4PwAAgP+F3b12p3UJwH9G4r1BdHS03NzcHtmPJyBechEREYqMjJS/v7/R5u7urvLlyyskJMTUcUNCQuTh4WGED5Lk7++vdOnSKTQ09JFjR0dHK3PmzDZta9eu1RtvvCE3NzcdOHBAd+/etTl24cKFlTt3bptzOnbsmMaMGaOFCxcaN/WPM3fuXPn7+ycbPqRUVFSUfvjhB3Xt2jVJe7du3fTNN9/I2dk5yX5ly5ZVunTpNH/+fMXHxys6OlrffPON/P39lSFDBklSxYoVdf78eW3YsEFWq1VRUVFavny56tWr98h6bt26pbt37ya5nh9//LHs7Oz05ptvasiQIfriiy+eGD7ExcUpJibG5gMAAAAAKUUA8ZKLjIyUJHl6etq0e3p6GtvMGjcyMlLZs2e32W5nZ6fMmTM/8thLly7Vvn371KVLF5v2B1+/iIyMlL29fZInGR48dlxcnNq2basJEyYod+7cTzyfixcvauPGjXr77bef2PdxFixYoIwZM9q8OmG1WtW5c2e9++67NmHMg/z8/LRp0yYNGTJEDg4O8vDw0J9//qmlS5cafSpXrqxFixapdevWsre3l5eXl9zd3R/7esigQYPk4+NjE9ZIkpOTkz7//HOtXr1ab7zxhjG3xuOMGzdO7u7uxidXrlxP3AcAAAAAEhFA4F9j+/bt6tKli77++mubiSBjYmK0c+dOI4BIicGDB6tIkSIpurGW7gcHHh4eT3y94knmzZun9u3by9HR0Wj74osvdOPGDQ0ePPiR+0VGRqpbt27q1KmT9u3bp507d8re3l4tWrRQ4ltSx44dU58+fTRixAgdOHBAwcHBOnPmjN59991kx/z000+1ZMkSrVq1yqaeRHPnzpWzs7N+++03RUdHP/HcBg8erOjoaONz/vz5J+4DAAAAAIkIIF5yXl5ekpRkhYioqChjm1njenl56fLlyzbb7927p6tXryY59s6dO9WwYUNNmTLFZq4C6f7cCUWLFjX+4u7l5aU7d+7YrGjx8LG3bdumZcuWyc7OTnZ2dqpZs6YkKWvWrBo5cqTNflarVfPmzVOHDh1kb2+f2kth+Omnn3T8+PEkT1Fs27ZNISEhcnBwkJ2dnfLnzy9JevXVV9WpUydJ0owZM+Tu7q7x48erTJkyqlq1qr799ltt3brVeF1l3Lhxqly5sgYMGKCSJUsqICBAX375pebNm6dLly7ZHHPixIn69NNPtWnTpmQn4Pz++++1fv167dmzRxkzZlTfvn2feH4ODg5yc3Oz+QAAAABAShFAvOT8/Pzk5eVlMyFiTEyMQkNDVbFiRVPHrVixoq5fv64DBw4YfbZt26aEhASVL1/eaNuxY4fq16+vzz77zGaFjERr1qxR48aNjZ/Lli2rDBky2Bz7+PHjOnfunHHsFStW6PDhwwoLC1NYWJjmzJkj6X5I0KNHD5vxd+7cqVOnTiWZtyG15s6dq7Jly6pUqVI27dOmTbOpZcOGDZLuhwAff/yxpPtzNTw8T0X69OklSQkJCU/s8+BcsuPHj9dHH32k4ODgZF/5iIqKUo8ePTR27FiVKlVKQUFBWrhwoTZu3Pgspw8AAAAAj8XU9S+B2NhYnTp1yvg5IiJCYWFhypw5s3Lnzq3AwECNHTtWBQoUkJ+fn4YPHy4fHx+b1w1q1qyppk2bqmfPnika02KxPHHcIkWKqE6dOurWrZtmzZqlu3fvqmfPnmrTpo2xAsb27dvVoEED9enTR82bNzfmcLC3t1fmzJl17949bdy40VhpQ7o/2WXXrl3Vr18/Zc6cWW5uburVq5cqVqxorICRL18+m2t05coVo6aH546YO3euypcvr+LFi6f62iaKiYnRsmXLNGnSpCRjPDwHhaurq1Fjzpw5Jd1f+WPKlCkaM2aM2rZtqxs3bmjIkCHKkyePypQpI0lq2LChunXrppkzZyogIECXLl1SYGCgypUrZ1zPzz77TCNGjNB3330nX19f43q6uroax+3evbuKFCmiwMBASVK5cuU0YMAAde/eXUeOHJG7u3uScwAAAACAZ8UTEC+B/fv3q0yZMsaNar9+/VSmTBmNGDFCkjRw4ED16tVL3bt312uvvabY2FgFBwfbzAsQHh5u3KSnZMyUjrto0SIVLlxYNWvWVL169VSlShXNnj3b2L5gwQLdunVL48aNk7e3t/FJnMRx586dcnV11SuvvGJzzlOmTFGDBg3UvHlzVa1aVV5eXlq5cmWqr110dLRWrFjxyKcfUnIdJGnJkiWyWq1q27ZtqmuQpBo1aui7777T6tWrVaZMGdWpU0cODg4KDg6Wk5OTJKlz586aPHmypk+fruLFi6tly5YqVKiQzXnPnDlTd+7cUYsWLWyu58SJEyVJCxcu1JYtWzR//nybpylGjx4tDw+PFL2KAQAAAABPw2J98Nlt4F+md+/eunfvnr788su0LgUPSVzrt9xn5WTnxMNUAADg32V3r91pXQLwn5F4bxAdHf3YueK4a8C/WvHixZ9prgoAAAAAwL8DAQT+1ZKblBIAAAAA8OJhDggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6AggAAAAAAGA6u7QuAMCLbfO7m+Xm5pbWZQAAAAD4l+MJCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDoCCAAAAAAAYDq7tC4AwIvt5zp15WLH/5QAAIBHq7ZrZ1qXAOBfgCcgAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggABN07txZTZo0SesyAAAAAOBfgwDiJbRr1y41bNhQPj4+slgsWr16tc12q9WqESNGyNvbW05OTvL399fJkyefacyUjtuoUSPlzp1bjo6O8vb2VocOHXTx4kWbPj/++KMqVKigjBkzKlu2bGrevLnOnDmT5HgLFixQlSpVUnRNzHLmzBlZLBaFhYU981grV65UrVq1lC1bNrm5ualixYr68ccfk/SbMWOGfH195ejoqPLly2vv3r0222fPnq033nhDbm5uslgsun79epIxDh48qFq1asnDw0NZsmRR9+7dFRsb+8znAAAAAACPQgDxErp586ZKlSqlGTNmJLt9/PjxmjZtmmbNmqXQ0FC5uLgoICBAt2/ffuoxUzpu9erVtXTpUh0/flwrVqxQeHi4WrRoYWyPiIhQ48aNVaNGDYWFhenHH3/UlStX1KxZsyTHW7NmjRo1apSSS/JC2LVrl2rVqqUNGzbowIEDql69uho2bKhDhw4Zfb7//nv169dPI0eO1MGDB1WqVCkFBATo8uXLRp9bt26pTp06GjJkSLLHuXjxovz9/ZU/f36FhoYqODhYR48eVefOnc0+RQAAAAD/YRar1WpN6yJgHovFolWrVhmvA1itVvn4+OiDDz5Q//79JUnR0dHy9PRUUFCQ2rRpk+oxn2XctWvXqkmTJoqLi1OGDBm0fPlytW3bVnFxcUqX7n4+tm7dOjVu3NjoI0m3b99W1qxZtX//fhUuXFi+vr7q3r27Tp06pWXLlilTpkwaNmyYunfvLun+kwp+fn76/vvv9cUXX2j//v0qXry4Fi1apOjoaL333nv6448/9Prrr2vhwoXKli2bUeOcOXM0adIkRUREyNfXV71799b7779vXIsHVatWTTt27FDnzp11/fp1ValSRZMmTdKdO3fUpk0bTZ061TiHlChWrJhat26tESNGSJLKly+v1157TdOnT5ckJSQkKFeuXOrVq5c+/PBDm3137Nih6tWr69q1a/Lw8DDaZ8+ereHDh+vSpUvGNf7tt99UsmRJnTx5Uvnz509RbTExMXJ3d9cPFSvJxc4uxecEAAD+e6rt2pnWJQAwUeK9QXR0tNzc3B7Zjycg/mMiIiIUGRkpf39/o83d3V3ly5dXSEjI/3Tcq1evatGiRapUqZJxU162bFmlS5dO8+fPV3x8vKKjo/XNN9/I39/f5sZ969atypEjhwoXLmy0TZo0Sa+++qoOHTqk999/X++9956OHz9uc8yRI0dq2LBhOnjwoOzs7NSuXTsNHDhQn3/+uX766SedOnXKuNmXpEWLFmnEiBH6+OOP9fvvv+uTTz7R8OHDtWDBAkkyXn/YsmWLLl26pJUrVxr7bt++XeHh4dq+fbsWLFigoKAgBQUFpfiaJiQk6MaNG8qcObMk6c6dOzpw4IDNNU6XLp38/f1T9d3FxcXJ3t7eCB8kycnJSZL0888/P3a/mJgYmw8AAAAApBQBxH9MZGSkJMnT09Om3dPT09hm9riDBg2Si4uLsmTJonPnzmnNmjXGNj8/P23atElDhgyRg4ODPDw89Oeff2rp0qU2YyT3+kW9evX0/vvvK3/+/Bo0aJCyZs2q7du32/Tp37+/AgICVKRIEfXp00cHDhzQ8OHDVblyZZUpU0Zdu3a12WfkyJGaNGmSmjVrJj8/PzVr1kx9+/bVV199JUnGkxJZsmSRl5eXERZIUqZMmTR9+nQVLlxYDRo0UP369bV169YUX9OJEycqNjZWrVq1kiRduXJF8fHxz/zd1ahRQ5GRkZowYYLu3Lmja9euGU9PXLp06ZH7jRs3Tu7u7sYnV65cKT4mAAAAABBA4H9uwIABOnTokDZt2qT06dOrY8eOSnwTKDIyUt26dVOnTp20b98+7dy5U/b29mrRooXRx2q1at26dUkCiJIlSxr/tlgs8vLyspkb4eE+iTfyJUqUsGlL3OfmzZsKDw9X165d5erqanzGjh2r8PDwJ55nsWLFlD59euNnb2/vJPU8ynfffafRo0dr6dKlyp49e4r2SalixYppwYIFmjRpkpydneXl5SU/Pz95enraPBXxsMGDBys6Otr4nD9//rnWBQAAAODlxovb/zFeXl6SpKioKHl7exvtUVFRKl269P9k3KxZsypr1qwqWLCgihQpoly5cumXX35RxYoVNWPGDLm7u2v8+PFG/2+//Va5cuVSaGioKlSooL179+revXuqVKmSzbgPz61gsViUkJDwyD6J8zc83Ja4T+KqEF9//bXKly9vM86DwcKjpKSe5CxZskRvv/22li1bZvO6RdasWZU+fXpFRUXZ9I+KijKuf0q1a9dO7dq1U1RUlFxcXGSxWDR58mTlzZv3kfs4ODjIwcEhVccBAAAAgEQ8AfEf4+fnJy8vL5tXAWJiYhQaGqqKFSv+z8dNvCGPi4uTdH8Fh4f/Cp94s5/Yd82aNapfv36KQoBn4enpKR8fH50+fVr58+e3+fj5+UmS7O3tJUnx8fHP5ZiLFy9Wly5dtHjxYtWvX99mm729vcqWLWtzjRMSErR169an/u48PT3l6uqq77//Xo6OjqpVq9Yz1Q8AAAAAj8ITEC+h2NhYnTp1yvg5IiJCYWFhypw5s3Lnzq3AwECNHTtWBQoUkJ+fn4YPHy4fHx+bVS1q1qyppk2bqmfPnika02KxPHHc0NBQ7du3T1WqVFGmTJkUHh6u4cOHK1++fMYNdP369TVlyhSNGTNGbdu21Y0bNzRkyBDlyZNHZcqUkXR/5YwxY8aYfBXvGz16tHr37i13d3fVqVNHcXFx2r9/v65du6Z+/fope/bscnJyUnBwsHLmzClHR0e5u7unaOzBgwfrwoULWrhwoaT7r1106tRJn3/+ucqXL2/M6+Dk5GSM2a9fP3Xq1EmvvvqqypUrp6lTp+rmzZvq0qWLMW5kZKQiIyON7+u3335TxowZlTt3bmOOiunTp6tSpUpydXXV5s2bNWDAAH366ac2q2UAAAAAwPPEExAvof3796tMmTLGDXu/fv1UpkwZY3WHgQMHqlevXurevbtee+01xcbGKjg4WI6OjsYY4eHhunLlSorHTMm4zs7OWrlypWrWrKlChQqpa9euKlmypHbu3Gk82l+jRg199913Wr16tcqUKaM6derIwcFBwcHBcnJyUnh4uE6dOqWAgABzL+L/9/bbb2vOnDmaP3++SpQooWrVqikoKMh4AsLOzk7Tpk3TV199JR8fHzVu3DjFY1+6dEnnzp0zfp49e7bu3bunHj16yNvb2/j06dPH6NO6dWtNnDhRI0aMUOnSpRUWFqbg4GCbiSlnzZqlMmXKqFu3bpKkqlWrqkyZMlq7dq3RZ+/evapVq5ZKlCih2bNn66uvvlLv3r2f+joBAAAAwJNYrIkz+wEvgMmTJ2vLli3asGFDWpfyn5e41u8PFSvJxY6HqQAAwKNV27UzrUsAYKLEe4Po6Gi5ubk9sh9PQOCFkjNnTg0ePDitywAAAAAApBJ/tsQLpVWrVmldAgAAAADgKfAEBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMJ1dWhcA4MVWJXij3Nzc0roMAAAAAP9yPAEBAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMZ5fWBQB4sX01ZKOcHJzTugwAAPAv1nNSw7QuAcC/AE9AAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA0xFAAAAAAAAA06UqgIiPj9euXbt0/fp1k8oBAAAAAAAvo1QFEOnTp1ft2rV17do1s+oBAAAAAAAvoVS/glG8eHGdPn3ajFoAAAAAAMBLKtUBxNixY9W/f3+tX79ely5dUkxMjM0HAAAAAADgYXap3aFevXqSpEaNGslisRjtVqtVFotF8fHxz686AAAAAADwUkh1ALF9+3Yz6gAAAAAAAC+xVAcQ1apVM6MOAAAAAADwEkv1HBCS9NNPP+nNN99UpUqVdOHCBUnSN998o59//vm5FgcAAAAAAF4OqQ4gVqxYoYCAADk5OengwYOKi4uTJEVHR+uTTz557gUCAAAAAIAX31OtgjFr1ix9/fXXypAhg9FeuXJlHTx48LkWBwAAAAAAXg6pDiCOHz+uqlWrJml3d3fX9evXn0dNAAAAAADgJZPqAMLLy0unTp1K0v7zzz8rb968z6UoAAAAAADwckl1ANGtWzf16dNHoaGhslgsunjxohYtWqT+/fvrvffeM6NGAAAAAADwgkt1APHhhx+qXbt2qlmzpmJjY1W1alW9/fbbeuedd9SrVy8zakQa2bVrlxo2bCgfHx9ZLBatXr3aZrvVatWIESPk7e0tJycn+fv76+TJk880piStXLlStWvXVpYsWWSxWBQWFpakz+3bt9WjRw9lyZJFrq6uat68uaKiomz6bN26VZUqVVLGjBnl5eWlQYMG6d69e0nG2rlzp3LlyiVJGjVqlCwWi82ncOHCjz2nN954I8k+FotF9evXlyTdvXtXgwYNUokSJeTi4iIfHx917NhRFy9eTDLWDz/8oPLly8vJyUmZMmVSkyZNjG1BQUHJHsdisejy5ctGvxkzZqhIkSJycnJSoUKFtHDhQptjHD16VM2bN5evr68sFoumTp362PMDAAAAgOch1QGExWLR0KFDdfXqVR05ckS//PKL/vrrL3300Udm1Ic0dPPmTZUqVUozZsxIdvv48eM1bdo0zZo1S6GhoXJxcVFAQIBu37791GMm9qlSpYo+++yzR/bp27ev1q1bp2XLlmnnzp26ePGimjVrZmw/fPiw6tWrpzp16ujQoUP6/vvvtXbtWn344YdJxlqzZo0aNmxo/FysWDFdunTJ+DxpedmVK1fa9D9y5IjSp0+vli1bSpJu3bqlgwcPavjw4Tp48KBWrlyp48ePq1GjRjbjrFixQh06dFCXLl10+PBh7d69W+3atTO2t27d2uY4ly5dUkBAgKpVq6bs2bNLkmbOnKnBgwdr1KhROnr0qEaPHq0ePXpo3bp1xji3bt1S3rx59emnn8rLy+ux5wYAAAAAz4vFarVan3bn8+fPS5Lx12O8vCwWi1atWmX8Rd5qtcrHx0cffPCB+vfvL+n+Uqyenp4KCgpSmzZtUj3mw86cOSM/Pz8dOnRIpUuXNtqjo6OVLVs2fffdd2rRooUk6Y8//lCRIkUUEhKiChUqaMiQIdq8ebP27dtn7Ldu3Tq1atVKly9fVsaMGY32/Pnza/r06apTp45GjRql1atXJ/vURUpNnTpVI0aM0KVLl+Ti4pJsn3379qlcuXI6e/ascufOrXv37snX11ejR49W165dU3Scv/76Szly5NDcuXPVoUMHSVKlSpVUuXJlTZgwwej3wQcfKDQ0NNkgxdfXV4GBgQoMDEz1ecbExMjd3V3jeyyRk4NzqvcHAAD/HT0nNXxyJwAvrMR7g+joaLm5uT2yX6qfgLh3756GDx8ud3d3+fr6ytfXV+7u7ho2bJju3r37TEXjxREREaHIyEj5+/sbbe7u7ipfvrxCQkJMPfaBAwd09+5dm2MXLlxYuXPnNo4dFxcnR0dHm/2cnJx0+/ZtHThwwGg7evSoLl++rBo1ahhtJ0+elI+Pj/Lmzav27dvr3Llzqapv7ty5atOmzSPDB+l+iGKxWOTh4SFJOnjwoC5cuKB06dKpTJky8vb2Vt26dXXkyJFHjrFw4UI5OzsbIczjznvv3r3P/N9nXFycYmJibD4AAAAAkFKpDiB69eql2bNna/z48Tp06JAOHTqk8ePHa+7cuerdu7cZNeJfKDIyUpLk6elp0+7p6WlsM/PY9vb2xs17cscOCAjQnj17tHjxYsXHx+vChQsaM2aMJOnSpUvGPmvWrFFAQIDs7e0lSeXLl1dQUJCCg4M1c+ZMRURE6PXXX9eNGzdSVNvevXt15MgRvf3224/sc/v2bQ0aNEht27Y10sHTp09Luj8HxbBhw7R+/XplypRJb7zxhq5evZrsOHPnzlW7du3k5ORktAUEBGjOnDk6cOCArFar9u/frzlz5uju3bu6cuVKis7hUcaNGyd3d3fjw5NPAAAAAFIj1QHEd999p6CgIL3zzjsqWbKkSpYsqXfeeUdz587Vd999Z0aNQKrVrl1bEyZM0LvvvisHBwcVLFhQ9erVkySlS/d/v/Zr1qyxmYuhbt26atmypUqWLKmAgABt2LBB169f19KlS1N03Llz56pEiRIqV65cstvv3r2rVq1ayWq1aubMmUZ7QkKCJGno0KFq3ry5ypYtq/nz58tisWjZsmVJxgkJCdHvv/+e5HWN4cOHq27duqpQoYIyZMigxo0bq1OnTknO+2kMHjxY0dHRxifxFSwAAAAASIlU35E4ODjI19c3Sbufn5/xV2S8/BInL3x45YmoqCjTJzb08vLSnTt3dP369cceu1+/frp+/brOnTunK1euqHHjxpKkvHnzSrr/JMShQ4eM1SqS4+HhoYIFC+rUqVNPrOvmzZtasmTJI+dwSAwfzp49q82bN9u8G+Xt7S1JKlq0qNHm4OCgvHnzJvsKyJw5c1S6dGmVLVvWpt3JyUnz5s3TrVu3dObMGZ07d06+vr7KmDGjsmXL9sRzeBwHBwe5ubnZfAAAAAAgpVIdQPTs2VMfffSR4uLijLa4uDh9/PHH6tmz53MtDv9efn5+8vLy0tatW422mJgYhYaGqmLFiqYeu2zZssqQIYPNsY8fP65z584lObbFYpGPj4+cnJy0ePFi5cqVS6+88oqk+5NSVqpUSZkzZ37ksWJjYxUeHm4EBI+zbNkyxcXF6c0330yyLTF8OHnypLZs2aIsWbIkOScHBwcdP37cZp8zZ84oT548SWpaunTpYyerzJAhg3LmzKn06dNryZIlatCgwTM/AQEAAAAAz8IutTscOnRIW7duVc6cOVWqVClJ95c8vHPnjmrWrGmzFOLKlSufX6X4n4uNjbX5y39ERITCwsKUOXNm5c6dW4GBgRo7dqwKFCggPz8/DR8+XD4+PjarWtSsWVNNmzY1wqknjSlJV69e1blz53Tx4kVJMm7Kvby85OXlJXd3d3Xt2lX9+vVT5syZ5ebmpl69eqlixYqqUKGCMfaECRNUp04dpUuXTitXrtSnn36qpUuXKn369JKktWvXJlkKs3///mrYsKHy5MmjixcvauTIkUqfPr3atm1r9OnYsaNy5MihcePG2ew7d+5cNWnSJEm4cPfuXbVo0UIHDx7U+vXrFR8fb8xVkTlzZtnb28vNzU3vvvuuRo4cqVy5cilPnjzGShaJy3km+v7773Xv3r1kg44TJ05o7969Kl++vK5du6bJkyfryJEjWrBggdHnzp07OnbsmPHvCxcuKCwsTK6ursqfP3+SMQEAAADgeUh1AOHh4aHmzZvbtDEZ3ctp//79ql69uvFzv379JEmdOnVSUFCQBg4cqJs3b6p79+66fv26qlSpouDgYJtVGMLDw20mP3zSmNL9YKBLly5Gn8QlPUeOHKlRo0ZJkqZMmaJ06dKpefPmiouLU0BAgL788kub+jdu3KiPP/5YcXFxKlWqlNasWaO6detKuv+6xNatWzV16lSbff7880+1bdtWf//9t7Jly6YqVarol19+sXl94dy5c0meJjh+/Lh+/vlnbdq0Kcl1vHDhgtauXStJNsuJStL27dv1xhtvSLofmNjZ2alDhw76559/VL58eW3btk2ZMmWy2Wfu3Llq1qxZkkk4JSk+Pl6TJk3S8ePHlSFDBlWvXl179uyxeW3q4sWLKlOmjPHzxIkTNXHiRFWrVk07duxIMiYAAAAAPA8Wq9VqTesigP+1lStXatiwYcaTAEi9xLV+x/dYIicH57QuBwAA/Iv1nNQwrUsAYKLEe4Po6OjHzhXHS+H4T3J1ddVnn32W1mUAAAAAwH9Gql/BkKTly5dr6dKlOnfunO7cuWOz7eDBg8+lMMBMtWvXTusSAAAAAOA/JdVPQEybNk1dunSRp6enDh06pHLlyilLliw6ffq08X49AAAAAADAg1IdQHz55ZeaPXu2vvjiC9nb22vgwIHavHmzevfurejoaDNqBAAAAAAAL7hUBxDnzp1TpUqVJElOTk66ceOGJKlDhw5avHjx860OAAAAAAC8FFIdQHh5eenq1auSpNy5c+uXX36RJEVERIgFNQAAAAAAQHJSHUDUqFFDa9eulSR16dJFffv2Va1atdS6dWs1bdr0uRcIAAAAAABefKleBWP27NlKSEiQJPXo0UNZsmTRnj171KhRI73zzjvPvUAAAAAAAPDiS3UAkS5dOqVL938PTrRp00Zt2rR5rkUBAAAAAICXS6oDCEm6du2a5s6dq99//12SVLRoUXXp0kWZM2d+rsUBAAAAAICXQ6rngNi1a5f8/Pw0bdo0Xbt2TdeuXdO0adPk5+enXbt2mVEjAAAAAAB4waX6CYgePXqoVatWmjlzptKnTy9Jio+P1/vvv68ePXrot99+e+5FAgAAAACAF1uqn4A4deqUPvjgAyN8kKT06dOrX79+OnXq1HMtDgAAAAAAvBxSHUC88sorxtwPD/r9999VqlSp51IUAAAAAAB4uaT6FYzevXurT58+OnXqlCpUqCBJ+uWXXzRjxgx9+umn+vXXX42+JUuWfH6VAgAAAACAF1aqA4i2bdtKkgYOHJjsNovFIqvVKovFovj4+GevEAAAAAAAvPBSHUBERESYUQcAAAAAAHiJpTqAyJMnjxl1AAAAAACAl1iqAwgAeNA7n9SVm5tbWpcBAAAA4F8u1atgAAAAAAAApBYBBAAAAAAAMB0BBAAAAAAAMB0BBAAAAAAAMF2KJqHMlCmTLBZLiga8evXqMxUEAAAAAABePikKIKZOnWpyGQAAAAAA4GWWogCiU6dOZtcBAAAAAABeYk81B0R4eLiGDRumtm3b6vLly5KkjRs36ujRo8+1OAAAAAAA8HJIdQCxc+dOlShRQqGhoVq5cqViY2MlSYcPH9bIkSOfe4EAAAAAAODFl+oA4sMPP9TYsWO1efNm2dvbG+01atTQL7/88lyLAwAAAAAAL4dUBxC//fabmjZtmqQ9e/bsunLlynMpCgAAAAAAvFxSNAnlgzw8PHTp0iX5+fnZtB86dEg5cuR4boUBeDFM6NZBjhkypHUZAAC81IZ+uzytSwCAZ5bqJyDatGmjQYMGKTIyUhaLRQkJCdq9e7f69++vjh07mlEjAAAAAAB4waU6gPjkk09UuHBh5cqVS7GxsSpatKiqVq2qSpUqadiwYWbUCAAAAAAAXnCpfgXD3t5eX3/9tYYPH64jR44oNjZWZcqUUYECBcyoDwAAAAAAvARSHUAkyp07t3Lnzv08awEAAAAAAC+pFAUQ/fr1S/GAkydPfupiAAAAAADAyylFAcShQ4dsfj548KDu3bunQoUKSZJOnDih9OnTq2zZss+/QgAAAAAA8MJLUQCxfft249+TJ09WxowZtWDBAmXKlEmSdO3aNXXp0kWvv/66OVUCAAAAAIAXWqpXwZg0aZLGjRtnhA+SlClTJo0dO1aTJk16rsUBAAAAAICXQ6oDiJiYGP31119J2v/66y/duHHjuRQFAAAAAABeLqkOIJo2baouXbpo5cqV+vPPP/Xnn39qxYoV6tq1q5o1a2ZGjQAAAAAA4AWX6mU4Z82apf79+6tdu3a6e/fu/UHs7NS1a1dNmDDhuRcIAAAAAABefKkOIJydnfXll19qwoQJCg8PlyTly5dPLi4uz704AAAAAADwckh1AJHIxcVFJUuWfJ61AAAAAACAl1Sq54AAAAAAAABILQIIAAAAAABgOgIIAAAAAABgOgIIAAAAAABguhRPQrl27doU9WvUqNFTFwMAAAAAAF5OKQ4gmjRp8sQ+FotF8fHxz1IPkGZ8fX0VGBiowMDAtC4FAAAAAF46KX4FIyEh4YkfwoeXx65du9SwYUP5+PjIYrFo9erVNtutVqtGjBghb29vOTk5yd/fXydPnnymMVM67tWrV9W+fXu5ubnJw8NDXbt2VWxsrLF9x44daty4sby9veXi4qLSpUtr0aJFydY0evRovfnmm5Kkffv2qXv37im4OqmX0pqWLVumwoULy9HRUSVKlNCGDRtstqfk+pw4cUKNGzdW1qxZ5ebmpipVqmj79u3J1vX3338rZ86cslgsun79+nM7XwAAAAB4GHNAIFk3b95UqVKlNGPGjGS3jx8/XtOmTdOsWbMUGhoqFxcXBQQE6Pbt2089ZkrHbd++vY4eParNmzdr/fr12rVrl01wsGfPHpUsWVIrVqzQr7/+qi5duqhjx45av359kuOtWbPGeG0oW7ZscnZ2fuK1eRopqWnPnj1q27atunbtqkOHDqlJkyZq0qSJjhw5YvRJyfVp0KCB7t27p23btunAgQMqVaqUGjRooMjIyCR1de3aVSVLljTlnAEAAADgQRar1WpNzQ7Lli3T4sWLdeLECUlSwYIF1a5dO7Vo0cKUApH2LBaLVq1aZbyGY7Va5ePjow8++ED9+/eXJEVHR8vT01NBQUFq06ZNqsdM6bi///67ihYtqn379unVV1+VJAUHB6tevXr6888/5ePjk+zx6tevL09PT82bN89oO3/+vPLnz6+//vpLbm5uSV7BuH79ugYNGqTVq1crOjpa+fPn16effqoGDRpIknbv3q2hQ4dq7969cnBwULly5bRkyRJlypQpRdf14Zpat26tmzdv2oQSFSpUUOnSpTVr1qwUXZ8rV64oW7Zs2rVrl15//XVJ0o0bN+Tm5qbNmzfL39/fGHvmzJn6/vvvNWLECNWsWVPXrl2Th4dHimqXpJiYGLm7u2tYq0ZyzJAhxfsBAIDUG/rt8rQuAQAeKfHeIDo6Wm5ubo/sl6pXMFq3bq3WrVvr2LFjyp8/v/Lnz6+jR4+qdevWatOmjVKZZeAFFRERocjISJubWXd3d5UvX14hISGmjhsSEiIPDw8jfJAkf39/pUuXTqGhoY8cOzo6WpkzZ7ZpW7t2rd54441k/wNJSEhQ3bp1tXv3bn377bc6duyYPv30U6VPn16SFBYWppo1a6po0aIKCQnRzz//rIYNG6bqNaSHawoJCbE5d0kKCAgwzj0l1ydLliwqVKiQFi5cqJs3b+revXv66quvlD17dpUtW9bY79ixYxozZowWLlyodOl4EAoAAACA+VI8CeXnn3+uLVu2aO3atcZfgBOtXbtWXbp00eeff84Efv8BiY/ye3p62rR7enom+5j/8xw3MjJS2bNnt9luZ2enzJkzP/LYS5cu1b59+/TVV1/ZtK9Zs0aNGzdOdp8tW7Zo7969+v3331WwYEFJUt68eY3t48eP16uvvqovv/zSaCtWrFhKTvORNUVGRj7x3BPbHtXHYrFoy5YtatKkiTJmzKh06dIpe/bsCg4ONp7MiIuLU9u2bTVhwgTlzp1bp0+fTlHNcXFxiouLM36OiYlJ8fkCAAAAQIr/9Dl//nxNmDAhSfgg3V96c/z48TaPtwP/Btu3b1eXLl309ddf2wQEMTEx2rlz5yOXjQ0LC1POnDmN8CG57TVr1nyuNT0PVqtVPXr0UPbs2fXTTz9p7969atKkiRo2bKhLly5JkgYPHqwiRYoYk2+m1Lhx4+Tu7m58cuXK9VxrBwAAAPByS3EAcfLkySSPhz8oJasg4OXg5eUlSYqKirJpj4qKMraZNa6Xl5cuX75ss/3evXu6evVqkmPv3LlTDRs21JQpU9SxY0ebbRs3blTRokUfeRPt5OT02FqftP1RHleTl5fXE889se1RfbZt26b169dryZIlqly5sl555RV9+eWXcnJy0oIFC4w+y5Ytk52dnezs7IwgJWvWrBo5cuQjax88eLCio6ONz/nz55/qGgAAAAD4b0pxAOHk5PTYZfpiYmLk6Oj4PGrCv5yfn5+8vLy0detWoy0mJkahoaGqWLGiqeNWrFhR169f14EDB4w+27ZtU0JCgsqXL2+07dixQ/Xr19dnn32W7NKaj3v9QpJKliypP//805hsNbntD9aZEk+qqWLFiknG3Lx5s3HuKbk+t27dkqQk8zqkS5dOCQkJkqQVK1bo8OHDCgsLU1hYmObMmSNJ+umnn9SjR49H1u/g4CA3NzebDwAAAACkVIoDiIoVK2rmzJmP3D5jxoxnuvnEv0tsbKxxgyrdnwAxLCxM586dk8ViUWBgoMaOHau1a9fqt99+U8eOHeXj42OzqkXNmjU1ffr0FI0pKUXjFilSRHXq1FG3bt20d+9e7d69Wz179lSbNm2MFTC2b9+u+vXrq3fv3mrevLkiIyMVGRmpq1evSrr/xMTGjRsf+fqFJFWrVk1Vq1ZV8+bNtXnzZkVERGjjxo0KDg6WdP9pgH379un999/Xr7/+qj/++EMzZ87UlStXJEnTp0+3eUXjSTVJUp8+fRQcHKxJkybpjz/+0KhRo7R//3717NkzxdenYsWKypQpkzp16qTDhw/rxIkTGjBggCIiIlS/fn1JUr58+VS8eHHj4+fnZ1zbh+fXAAAAAIDnJcUBxNChQzV37ly1atVKe/fuVUxMjKKjo/XLL7+oZcuWmjdvnoYOHWpmrfgf2r9/v8qUKaMyZcpIkvr166cyZcpoxIgRkqSBAweqV69e6t69u1577TXFxsYqODjY5imY8PBw44Y8JWOmdNxFixapcOHCqlmzpurVq6cqVapo9uzZxvYFCxbo1q1bGjdunLy9vY1Ps2bNJN1/DcLV1VWvvPLKY6/BihUr9Nprr6lt27YqWrSoBg4caKxyUbBgQW3atEmHDx9WuXLlVLFiRa1Zs0Z2dvfndb1y5YrCw8NTXJMkVapUSd99951mz56tUqVKafny5Vq9erWKFy+e4uuTNWtWBQcHKzY2VjVq1NCrr76qn3/+WWvWrFGpUqUee74AAAAAYCaLNRVrZ65atUrdu3e3+autJGXKlElfffWVmjdv/twLBJ633r176969ezYrWEiSt7e3PvroI7399ttpVNmLJXGt32GtGskxQ4a0LgcAgJfa0G+Xp3UJAPBIifcG0dHRj31VO8XLcEpS06ZNFRAQoB9//NGYcLJgwYKqXbu2nJ2dn61i4H+kePHiNq8L3bp1S7t371ZUVNRzX5UCAAAAAHBfqgIISXJ2dlbTpk3NqAX4n3h4AsjZs2fro48+UmBgIPOYAAAAAIBJUhVAJCQkKCgoSCtXrtSZM2dksVjk5+enFi1aqEOHDrJYLGbVCZgmMDBQgYGBaV0GAAAAALzUUjwJpdVqVaNGjfT222/rwoULKlGihIoVK6azZ8+qc+fOPBUBAAAAAAAeKcVPQAQFBWnXrl3aunWrqlevbrNt27ZtatKkiRYuXKiOHTs+9yIBAAAAAMCLLcVPQCxevFhDhgxJEj5IUo0aNfThhx9q0aJFz7U4AAAAAADwckhxAPHrr7+qTp06j9xet25dHT58+LkUBQAAAAAAXi4pDiCuXr0qT0/PR2739PTUtWvXnktRAAAAAADg5ZLiACI+Pl52do+eMiJ9+vS6d+/ecykKAAAAAAC8XFI8CaXValXnzp3l4OCQ7Pa4uLjnVhQAAAAAAHi5pDiA6NSp0xP7sAIGAAAAAABITooDiPnz55tZBwAAAAAAeImleA4IAAAAAACAp0UAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATGexWq3WtC4CwIsnJiZG7u7uio6OlpubW1qXAwAAACCNpPTegCcgAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6QggAAAAAACA6ezSugAAL7bjE3bK1dElrcsAAECSVGRojbQuAQDwCDwBAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAAQAAAAAATEcAkQK7du1Sw4YN5ePjI4vFotWrV9tst1qtGjFihLy9veXk5CR/f3+dPHnymcaUpJUrV6p27drKkiWLLBaLwsLCHjme1WpV3bp1kx3LYrEk+SxZsiTJGAsWLFCVKlVSfOzw8HA1bdpU2bJlk5ubm1q1aqWoqKgk/X744QeVL19eTk5OypQpk5o0aWKzvXfv3ipbtqwcHBxUunTpJPvfvn1bnTt3VokSJWRnZ5dk/0SLFi1SqVKl5OzsLG9vb7311lv6+++/je13797VmDFjlC9fPjk6OqpUqVIKDg5OdqzUHnvGjBkqUqSInJycVKhQIS1cuDBJn+vXr6tHjx7y9vaWg4ODChYsqA0bNhjbn9fvREq+l4MHD6pWrVry8PBQlixZ1L17d8XGxj72WgAAAADAsyCASIGbN2+qVKlSmjFjRrLbx48fr2nTpmnWrFkKDQ2Vi4uLAgICdPv27aceM7FPlSpV9Nlnnz2xxqlTp8pisTxy+/z583Xp0iXjk9yN9Jo1a9SoUaMUHfvmzZuqXbu2LBaLtm3bpt27d+vOnTtq2LChEhISjH4rVqxQhw4d1KVLFx0+fFi7d+9Wu3btkoz31ltvqXXr1skeKz4+Xk5OTurdu7f8/f2T7bN792517NhRXbt21dGjR7Vs2TLt3btX3bp1M/oMGzZMX331lb744gsdO3ZM7777rpo2bapDhw498rql5NgzZ87U4MGDNWrUKB09elSjR49Wjx49tG7dOqPPnTt3VKtWLZ05c0bLly/X8ePH9fXXXytHjhw21/RZfydS8r1cvHhR/v7+yp8/v0JDQxUcHKyjR4+qc+fOjzwuAAAAADwru7Qu4EVQt25d1a1bN9ltVqtVU6dO1bBhw9S4cWNJ0sKFC+Xp6anVq1erTZs2qR4zUYcOHSRJZ86ceWy/sLAwTZo0Sfv375e3t3eyfTw8POTl5fXIMW7fvq1Nmzbpk08+SdGxd+/erTNnzujQoUNyc3OTdP8JikyZMmnbtm3y9/fXvXv31KdPH02YMEFdu3Y19i1atKjNWNOmTZMk/fXXX/r111+THMvFxUUzZ840jnv9+vUkfUJCQuTr66vevXtLkvz8/PTOO+/Y3Kh/8803Gjp0qOrVqydJeu+997RlyxZNmjRJ3377bbLnmZJjf/PNN3rnnXeMACVv3rzat2+fPvvsMzVs2FCSNG/ePF29elV79uxRhgwZJEm+vr424zyP34mUfC/r169XhgwZNGPGDKVLdz+DnDVrlkqWLKlTp04pf/78j60BAAAAAJ4GT0A8o4iICEVGRtr8ddzd3V3ly5dXSEiI6ce/deuW2rVrpxkzZjw2YOjRo4eyZs2qcuXKad68ebJarTbbt27dqhw5cqhw4cIpOm5cXJwsFoscHByMNkdHR6VLl04///yzpPuP+V+4cEHp0qVTmTJl5O3trbp16+rIkSNPcaaPV7FiRZ0/f14bNmyQ1WpVVFSUli9fboQNiTU7Ojra7Ofk5GTU+7QeNe7evXt19+5dSdLatWtVsWJF9ejRQ56enipevLg++eQTxcfHP9Oxk6vlSd9LXFyc7O3tjfAhsV5Jj70WcXFxiomJsfkAAAAAQEoRQDyjyMhISZKnp6dNu6enp7HNTH379lWlSpWMpy+SM2bMGC1dulSbN29W8+bN9f777+uLL76w6fPg6xcpUaFCBbm4uGjQoEG6deuWbt68qf79+ys+Pl6XLl2SJJ0+fVqSNGrUKA0bNkzr169XpkyZ9MYbb+jq1atPcbaPVrlyZS1atEitW7eWvb29vLy85O7ubvM6Q0BAgCZPnqyTJ08qISFBmzdv1sqVK416n1ZAQIDmzJmjAwcOyGq1av/+/ZozZ47u3r2rK1euSLp/LZYvX674+Hht2LBBw4cP16RJkzR27NhnOvbDUvK91KhRQ5GRkZowYYLu3Lmja9eu6cMPP5Skx16LcePGyd3d3fjkypXrudYOAAAA4OVGAPECW7t2rbZt26apU6c+tt/w4cNVuXJllSlTRoMGDdLAgQM1YcIEY7vVatW6detSFUBky5ZNy5Yt07p16+Tq6ip3d3ddv35dr7zyivGX9cQ5B4YOHarmzZurbNmymj9/viwWi5YtW5b6E36MY8eOqU+fPhoxYoQOHDig4OBgnTlzRu+++67R5/PPP1eBAgVUuHBh2dvbq2fPnurSpYvNkwBPY/jw4apbt64qVKigDBkyqHHjxurUqZMk2VyL7Nmza/bs2Spbtqxat26toUOHatasWc907Iel5HspVqyYFixYoEmTJsnZ2VleXl7y8/OTp6fnY6/F4MGDFR0dbXzOnz//XGsHAAAA8HIjgHhGia89PLzKQFRU1GNfiXgetm3bpvDwcHl4eMjOzk52dven9GjevLneeOONR+5Xvnx5/fnnn4qLi5Mk7d27V/fu3VOlSpVSdfzatWsrPDxcly9f1pUrV/TNN9/owoULyps3ryQZ81E8OOeDg4OD8ubNq3PnzqXqWE8ybtw4Va5cWQMGDFDJkiUVEBCgL7/8UvPmzTP+qp8tWzatXr1aN2/e1NmzZ/XHH3/I1dXVqPdpOTk5ad68ebp165bOnDmjc+fOydfXVxkzZlS2bNkk3b8WBQsWVPr06Y39ihQposjISN25c+eZjv+wJ30vktSuXTtFRkbqwoUL+vvvvzVq1Cj99ddfj70WDg4OcnNzs/kAAAAAQEoRQDwjPz8/eXl5aevWrUZbTEyMQkNDVbFiRVOP/eGHH+rXX39VWFiY8ZGkKVOmaP78+Y/cLywsTJkyZTLmCVizZo3q169vc3OcGlmzZpWHh4e2bdumy5cvG09SJC6tefz4caPv3bt3debMGeXJk+epjvUot27dSvLX+8TzeXi+C0dHR+XIkUP37t3TihUrHvv6SmpkyJBBOXPmVPr06bVkyRI1aNDAqKly5co6deqUzQohJ06ckLe3t+zt7Z/L8R/2qO/lQZ6ennJ1ddX3338vR0dH1apVy5RaAAAAAIBVMFIgNjZWp06dMn6OiIhQWFiYMmfOrNy5cyswMFBjx45VgQIF5Ofnp+HDh8vHx8dmqcuaNWuqadOm6tmzZ4rGlKSrV6/q3LlzunjxoiQZN/JeXl42n4flzp1bfn5+kqR169YpKipKFSpUkKOjozZv3qxPPvlE/fv3N/qvXbtWY8aMsRnjSceW7i/tWaRIEWXLlk0hISHq06eP+vbtq0KFCkmS3Nzc9O6772rkyJHKlSuX8uTJY7z60bJlS+NYp06dUmxsrCIjI/XPP/8YQUrRokWNm/Njx47pzp07unr1qm7cuGH0KV26tCSpYcOG6tatm2bOnKmAgABdunRJgYGBKleunHx8fCRJoaGhunDhgkqXLq0LFy5o1KhRSkhI0MCBA41apk+frlWrVtkESk869okTJ7R3716VL19e165d0+TJk3XkyBEtWLDAGOO9997T9OnT1adPH/Xq1UsnT57UJ598Yqza8bx+J1LyvSSeZ6VKleTq6qrNmzdrwIAB+vTTT+Xh4SEAAAAAMAMBRArs379f1atXN37u16+fJKlTp04KCgrSwIEDdfPmTXXv3l3Xr19XlSpVFBwcbLMyQnh4uDEhYUrGlO4HA126dDH6JC7pOXLkSI0aNSpFtScut9i3b19ZrVblz59fkydPVrdu3Yy6Tp06pYCAAJv9UnLs48ePa/Dgwbp69ap8fX01dOhQ9e3b12acCRMmyM7OTh06dNA///yj8uXLa9u2bcqUKZPR5+2339bOnTuNn8uUKSPp/g144lKV9erV09mzZ5P0SXy6oXPnzrpx44amT5+uDz74QB4eHqpRo4bNMpy3b9/WsGHDdPr0abm6uqpevXr65ptvbG66r1y5ovDwcJtzeNKx4+PjNWnSJB0/flwZMmRQ9erVtWfPHptlNnPlyqUff/xRffv2VcmSJZUjRw716dNHgwYNMvo8r9+JlHwve/fu1ciRIxUbG6vChQvrq6++Mpb4BAAAAAAzWKwPP5+O/5TJkydry5Yt2rBhQ1qXghdMTEyM3N3dtXfYWrk6uqR1OQAASJKKDK2R1iUAwH9O4r1BdHT0Y+eKYw6I/7icOXNq8ODBaV0GAAAAAOAlxysY/3GtWrVK6xIAAAAAAP8BPAEBAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMRwABAAAAAABMZ5fWBQB4sRUaUE1ubm5pXQYAAACAfzmegAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKYjgAAAAAAAAKazS+sCALzYxo0bJwcHh7QuAwDwHzdq1Ki0LgEA8AQ8AQEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAAEAAAAAAExHAIF/NV9fX02dOtX42WKxaPXq1WlWDwAAAADg6RBAmGDXrl1q2LChfHx8kr1htlqtGjFihLy9veXk5CR/f3+dPHnymcZM6bgHDx5UrVq15OHhoSxZsqh79+6KjY01tv/999+qU6eOfHx85ODgoFy5cqlnz56KiYlJcrwFCxaoSpUqKb8wL5idO3cqV65caV1GEsuWLVPhwoXl6OioEiVKaMOGDTbbo6Ki1LlzZ/n4+MjZ2Vl16tSx+T24evWqevXqpUKFCsnJyUm5c+dW7969FR0d/b8+FQAAAAD/IQQQJrh586ZKlSqlGTNmJLt9/PjxmjZtmmbNmqXQ0FC5uLgoICBAt2/ffuoxUzLuxYsX5e/vr/z58ys0NFTBwcE6evSoOnfubIyRLl06NW7cWGvXrtWJEycUFBSkLVu26N13301yvDVr1qhRo0YpvCovnjVr1qhhw4ZpXYaNPXv2qG3bturatasOHTqkJk2aqEmTJjpy5Iik+yFUkyZNdPr0aa1Zs0aHDh1Snjx55O/vr5s3b0q6/3tw8eJFTZw4UUeOHFFQUJCCg4PVtWvXtDw1AAAAAC85AggT1K1bV2PHjlXTpk2TbLNarZo6daqGDRumxo0bq2TJklq4cKEuXrz42FcLHjdmSsddv369MmTIoBkzZqhQoUJ67bXXNGvWLK1YsUKnTp2SJGXKlEnvvfeeXn31VeXJk0c1a9bU+++/r59++snmeLdv39amTZuMAMLX11djx45Vx44d5erqqjx58mjt2rX666+/1LhxY7m6uqpkyZLav3+/zTgrVqxQsWLF5ODgIF9fX02aNOmJ1/fSpUuqW7eunJyclDdvXi1fvtxm+/nz59WqVSt5eHgoc+bMaty4sc6cOSNJ2rRpkxwdHXX9+nWbffr06aMaNWrYtK1du9Y4vzfeeEO9evVSYGCgMmXKJE9PT3399de6efOmunTpoowZMyp//vzauHGjsf+1a9fUvn17ZcuWTU5OTipQoIDmz58vSTpz5owsFouWLl2q119/XU5OTnrttdd04sQJ7du3T6+++qpcXV1Vt25d/fXXX8aYn3/+uerUqaMBAwaoSJEi+uijj/TKK69o+vTpkqSTJ0/ql19+0cyZM/Xaa6+pUKFCmjlzpv755x8tXrxYklS8eHGtWLFCDRs2VL58+VSjRg19/PHHWrdune7du/fE6w8AAAAAT4MA4n8sIiJCkZGR8vf3N9rc3d1Vvnx5hYSEmDpuXFyc7O3tlS7d/33tTk5OkqSff/452XEvXryolStXqlq1ajbtW7duVY4cOVS4cGGjbcqUKapcubIOHTqk+vXrq0OHDurYsaPefPNNHTx4UPny5VPHjh1ltVolSQcOHFCrVq3Upk0b/fbbbxo1apSGDx+uoKCgx57r8OHD1bx5cx0+fFjt27dXmzZt9Pvvv0uS7t69q4CAAGXMmFE//fSTdu/eLVdXV9WpU0d37txRzZo15eHhoRUrVhjjxcfH6/vvv1f79u2NtqNHj+ry5cs2ocSCBQuUNWtW7d27V7169dJ7772nli1bqlKlSjp48KBq166tDh066NatW0adx44d08aNG/X7779r5syZypo1q825jBw5UsOGDdPBgwdlZ2endu3aaeDAgfr888/1008/6dSpU/p/7d15WBXl///xF8guAuICuOCGCrjgbqCmBoZGZuVHzfy45VKm5ZYLJWqZuWammbaKlWZZrmUqmVgpoZikopGSiqVgqYCYgsL8/vDL/DwhiOb5YPV8XNe5Ls/Me+6558zd6Tov7pmZPHmyWR8XF2dxjiUpPDzc4hxLkpOTk7ne1tZWjo6ORZ5jScrMzJSbm5vs7OyKrMnJyVFWVpbFCwAAAABKigDifywtLU2S5OXlZbHcy8vLXGetdu+55x6lpaVpzpw5ys3N1blz5zRx4kRJV2cVXKt3795ycXFR1apV5ebmprffftti/fUuv7jvvvv0+OOPq27dupo8ebKysrLUsmVL9ejRQ/Xq1dOECRN06NAhpaenS5LmzZun0NBQRUVFqV69ehowYIBGjBihOXPmFHusPXr00ODBg1WvXj1NmzZNLVq00MKFCyVJH330kfLz8/X222+rUaNGCggI0NKlS5WamqrY2FiVKVNGjzzyiFasWGG2t3XrVmVkZKh79+4WxxceHi4HBwdzWVBQkCZNmqS6desqMjJSTk5OqlixooYMGWIe85kzZ7Rv3z5JUmpqqpo2baoWLVqoZs2aCgsLK3RJxzPPPKPw8HAFBARo5MiR2rNnj6KiotSmTRs1bdpUgwYN0rZt2yzOc3Hn2N/fX76+voqMjNS5c+eUm5urWbNm6Zdffil0jgv8/vvvmjZtmoYOHVrs5z5jxgy5u7ubrzvx/hgAAAAA7lwEEP8iDRo00LJly/Tyyy/LxcVF3t7eqlWrlry8vCxmRUhXZzN8//33WrdunVJSUjRmzBhznWEY2rBhQ6EAonHjxua/C34kN2rUqNCy06dPS5IOHTqkNm3aWLTRpk0bHT58WHl5eUUeR3BwcKH3BTMgfvjhBx05ckTlypWTq6urXF1d5enpqUuXLiklJUWS1KdPH8XGxurkyZOSpOXLlysiIkIeHh5mm9cLWK49vjJlyqhChQrFHt+wYcO0cuVKNWnSROPHj9fOnTsLHUtJPrOC9krC3t5eq1ev1k8//SRPT0+5uLho27Zt6tKlS6FzLElZWVmKiIhQYGCgpk6dWmzbkZGRyszMNF8nTpwocb8AAAAAoOj51rAKb29vSVefVODj42MuT09PV5MmTaze7qOPPqpHH31U6enpKlu2rGxsbDRv3jzVrl27UHve3t7y9/eXp6en2rVrp6ioKPn4+GjXrl26cuWKQkJCLLaxt7c3/21jY1Pksvz8/Fs+zhvJzs5W8+bNtXz58kLrKlWqJElq2bKl6tSpo5UrV2rYsGFas2aNxWUfp06dMi8juda1xyJdPZ7ijq9Lly46fvy4Nm7cqJiYGIWGhmr48OGaO3fuddss6jO79vPy9vY2Z5AUSE9PN8+/JDVv3lyJiYnKzMxUbm6uKlWqpNatW6tFixYW250/f16dO3dWuXLltGbNmkLH92eOjo5ydHQstgYAAAAAisIMiP+xWrVqydvbW1u3bjWXZWVlKT4+vtBf9q3ZrpeXl1xdXfXRRx/JyclJnTp1KrLtgh/ABfcXWLdunSIiIlSmTJlb7q8kBQQEaMeOHRbLduzYoXr16hXb9nfffVfofUBAgCSpWbNmOnz4sCpXriw/Pz+Ll7u7u7lNnz59tHz5cm3YsEG2trYWYcOGDRsUEhIiT0/Pv3R80tXQo3///vrggw80f/58vfnmm3+pveDgYItzLEkxMTHXPcfu7u6qVKmSDh8+rISEBHXr1s1cl5WVpXvvvVcODg5av369xT0jAAAAAMAamAFhBdnZ2eZTJaSrN4hMTEyUp6enfH19NWrUKL344ouqW7euatWqpaioKFWpUkUPPviguU1oaKgeeughjRgxokRt2tjYlKjd1157TSEhIXJ1dVVMTIzGjRunmTNnmpcfbNy4Uenp6WrZsqVcXV2VlJSkcePGqU2bNqpZs6akq0+HeOGFF/7y5zR27Fi1bNlS06ZNU69evRQXF6fXXntNr7/+erHbrVq1Si1atFDbtm21fPly7dq1S++8846kq8HCnDlz1K1bN73wwguqVq2ajh8/rtWrV2v8+PGqVq2aWTd16lRNnz5d//nPfyz+sn/t0y/+ismTJ6t58+Zq0KCBcnJy9Nlnn5lBya0aOXKk2rdvr5dfflkRERFauXKlEhISLIKNVatWqVKlSvL19dX+/fs1cuRIPfjgg7r33nsl/f/w4Y8//tAHH3xgcUPJSpUq/eVgCQAAAACuhwDCChISEtSxY0fzfcH9E/r376/o6GiNHz9eFy5c0NChQ5WRkaG2bdtq06ZNFn+FTklJ0e+//17iNiWVqN1du3ZpypQpys7Olr+/v9544w317dvXXO/s7Ky33npLo0ePVk5OjqpXr66HH37YvFllSkqKjhw5ovDw8L/8OTVr1kwff/yxJk+erGnTpsnHx0cvvPCCBgwYUOx2zz//vFauXKknn3xSPj4++vDDDxUYGChJcnFx0ddff60JEybo4Ycf1vnz51W1alWFhobKzc3NbMPPz0+tWrXSrl27NH/+fHP5hQsXtHXrVotlt8rBwUGRkZE6duyYnJ2d1a5dO61cufIvtRkSEqIVK1Zo0qRJevbZZ1W3bl2tXbtWDRs2NGtOnTqlMWPGmJfj9OvXT1FRUeb677//XvHx8ZKufg7XOnr0qBk0AQAAAMDtZGMUPBMRKIF58+bpyy+/1MaNG0u7K1axevVqTZo0SQcPHiztrtzxsrKy5O7urokTJ3JvCABAqbvRzZQBANZT8NsgMzPT4g+/f8Y9IHBTqlWrpsjIyNLuhtW4urpq1qxZpd0NAAAAAPjH4RIM3JSePXuWdhesquA+CQAAAACA24sZEAAAAAAAwOoIIAAAAAAAgNURQAAAAAAAAKsjgAAAAAAAAFZHAAEAAAAAAKyOAAIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDqCCAAAAAAAIDVEUAAAAAAAACrI4AAAAAAAABWRwABAAAAAACsjgACAAAAAABYHQEEAAAAAACwOhvDMIzS7gSAv5+srCy5u7srMzNTbm5upd0dAAAAAKWkpL8NmAEBAAAAAACsjgACAAAAAABYHQEEAAAAAACwOgIIAAAAAABgdQQQAAAAAADA6gggAAAAAACA1RFAAAAAAAAAqyOAAAAAAAAAVkcAAQAAAAAArI4AAgAAAAAAWB0BBAAAAAAAsDoCCAAAAAAAYHV2pd0BAH9vq9d0lItLmdLuBgDc8Xr22FXaXQAAoFQxAwIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDqCCAAAAAAAIDVEUAAAAAAAACrI4AAAAAAAABWRwABAAAAAACsjgACAAAAAABYHQEEAAAAAACwOgIIAAAAAABgdQQQAAAAAADA6gggAAAAAACA1RFAAAAAAAAAqyOAAAAAAAAAVkcAAQAAAAAArI4AAgAAAAAAWB0BBAAAAAAAsDoCCKAYx44dk42NjRITE0u7KwAAAADwt0YA8Q/w9ddfq2vXrqpSpYpsbGy0du1ai/WGYWjy5Mny8fGRs7OzwsLCdPjw4WLbXLx4sRo3biw3Nze5ubkpODhYX3zxhUXNm2++qQ4dOsjNzU02NjbKyMgo1M706dMVEhIiFxcXeXh4XHdfNjY2hV4rV64sVLds2TK1bdu22H4XSEpKUvfu3VWzZk3Z2Nho/vz5RdYOHDhQkyZNKlG7kvT000+refPmcnR0VJMmTYqtPXLkiMqVK1fksV/rRudRklavXq17771XFSpUKDYYiYuL0z333KOyZcvKzc1Nd999ty5evCjpaqgyaNAg1apVS87OzqpTp46mTJmi3NzcG/YRAAAAAG4VAcQ/wIULFxQUFKRFixZdd/3s2bO1YMECLVmyRPHx8SpbtqzCw8N16dKlItusVq2aZs6cqT179ighIUH33HOPunXrpqSkJLPmjz/+UOfOnfXss88W2U5ubq569OihYcOGFXsMS5cu1alTp8zXgw8+WKhm3bp1euCBB4pt59q+1a5dWzNnzpS3t3eRdXl5efrss89K3G6Bxx57TL169Sq25vLly+rdu7fatWtXojZvdB4Latq2batZs2YVWRMXF6fOnTvr3nvv1a5du7R7926NGDFCtrZX/3P/8ccflZ+frzfeeENJSUl65ZVXtGTJkmLPIwAAAAD8VXal3QH8dV26dFGXLl2uu84wDM2fP1+TJk1St27dJEnvvfeevLy8tHbtWj3yyCPX3a5r164W76dPn67Fixfru+++U4MGDSRJo0aNkiTFxsYW2bfnn39ekhQdHV3sMXh4eBQbFFy6dElbtmzRSy+9pGeffVZbt25VfHy8RU1QUJC6d++uyZMnq2XLlmrZsqUkaeLEiUW2u3PnTtnb25u1u3bt0uOPP65Dhw6pYcOGeu655wpts2DBAknSb7/9pn379hXZ9qRJk+Tv76/Q0FDt3Lmz6IP/P8WdxwJ9+/aVdHUWQ1FGjx6tp59+2uK469evb/67c+fO6ty5s/m+du3aSk5O1uLFizV37twb9hMAAAAAbgUzIP7hjh49qrS0NIWFhZnL3N3d1bp1a8XFxZWojby8PK1cuVIXLlxQcHCwVfo5fPhwVaxYUa1atdK7774rwzAs1m/dulVVq1aVv7+/+vTpo127diklJcVcn5SUpH379unRRx+9qf2uX79eXbt2lY2NjbKzs3X//fcrMDBQe/bs0dSpU/XMM8/c0vF89dVXWrVqVbGzGazh9OnTio+PV+XKlRUSEiIvLy+1b99e3377bbHbZWZmytPTs9ianJwcZWVlWbwAAAAAoKQIIP7h0tLSJEleXl4Wy728vMx1Rdm/f79cXV3l6OioJ554QmvWrFFgYOBt7+MLL7ygjz/+WDExMerevbuefPJJLVy40KLm2ssvGjRooKCgIK1YscJcv3z5crVu3Vp+fn43te9r212xYoXy8/P1zjvvqEGDBrr//vs1bty4mz6eM2fOaMCAAYqOjpabm9tNb/9X/Pzzz5KkqVOnasiQIdq0aZOaNWum0NDQIu/7ceTIES1cuFCPP/54sW3PmDFD7u7u5qt69eq3vf8AAAAA/rkIIFCk+vXrKzExUfHx8Ro2bJj69++vgwcP3vb9REVFqU2bNmratKkmTJig8ePHa86cOeZ6wzC0YcMGi/s09OnTxwwgDMPQhx9+qD59+tzUfg8dOqSTJ08qNDTUfN+4cWM5OTmZNbcy42PIkCF69NFHdffdd193/TfffCNXV1fztXz58pveR1Hy8/MlSY8//rgGDhyopk2b6pVXXlH9+vX17rvvFqr/9ddf1blzZ/Xo0UNDhgwptu3IyEhlZmaarxMnTty2fgMAAAD45+MeEP9wBfdVSE9Pl4+Pj7k8PT39hk9wcHBwMGcUNG/eXLt379arr76qN954w2r9laTWrVtr2rRpysnJkaOjo3bt2qUrV64oJCTErOndu7cmTJig77//XhcvXtSJEydueFPIP1u/fr06depkETjcDl999ZXWr19v3k/BMAzl5+fLzs5Ob775pnr37m3x9Io/z075KwrO8Z9nqgQEBCg1NdVi2cmTJ9WxY0eFhITozTffvGHbjo6OcnR0vG19BQAAAPDvQgDxD1erVi15e3tr69atZuCQlZVlzmq4Gfn5+crJybFCLy0lJiaqfPny5o/ddevWKSIiQmXKlDFrqlWrpvbt22v58uW6ePGiOnXqpMqVK9/UftatW6ehQ4ea7wMCAvT+++/r0qVLZijx3Xff3XT/4+LilJeXZ7GfWbNmaefOnapataqcnZ1v+lKRkqpZs6aqVKmi5ORki+U//fSTxQ0uf/31V3Xs2FHNmzfX0qVLzSdkAAAAAIC1EED8A2RnZ+vIkSPm+6NHjyoxMVGenp7y9fXVqFGj9OKLL6pu3bqqVauWoqKiVKVKFYtHXYaGhuqhhx7SiBEjJF2dbt+lSxf5+vrq/PnzWrFihWJjY7V582Zzm7S0NKWlpZn73r9/v8qVKydfX1/zhoapqak6e/asUlNTlZeXZ/7l38/PT66urtqwYYPS09N11113ycnJSTExMXrppZcsbv64fv16vfDCC4WOu0+fPpoyZYpyc3P1yiuvWKzLzc01LxfJzc3Vr7/+qsTERLm6usrPz0+nT59WQkKC1q9fb27z6KOP6rnnntOQIUMUGRmpY8eOXfepEEeOHFF2drbS0tJ08eJF85gCAwPl4OCggIAAi/qEhATZ2tqqYcOG1z+B/+dG51GS+VmePHlSksygwdvbW97e3rKxsdG4ceM0ZcoUBQUFqUmTJlq2bJl+/PFHffLJJ5Kuhg8dOnRQjRo1NHfuXP3222/mPot7EgkAAAAA/BUEEP8ACQkJ6tixo/l+zJgxkqT+/fsrOjpa48eP14ULFzR06FBlZGSobdu22rRpk8WlBykpKfr999/N96dPn1a/fv106tQpubu7q3Hjxtq8ebM6depk1ixZssR8zKYk854HS5cu1YABAyRJkydP1rJly8yapk2bSpK2bdumDh06yN7eXosWLdLo0aNlGIb8/Pw0b948834EKSkpOnLkiMLDwwsd93/+8x+NGDFCZcqUsQhTpKuXFxTsS5Lmzp2ruXPnqn379oqNjdWGDRvUqlUrVaxY0awpCESeeOIJNW3aVIGBgZo1a5a6d+9u0fbgwYO1ffv2Qsd09OhR1axZs1A/S+pG51G6GsYMHDjQrCl4jOqUKVM0depUSVcfj3rp0iWNHj1aZ8+eVVBQkGJiYlSnTh1JUkxMjI4cOaIjR46oWrVqFn3489NHAAAAAOB2sTH4xYE72Lx58/Tll19q48aNt7XdBx54QG3bttX48eNva7v/JllZWXJ3d9fS6GZycSlz4w0A4F+uZ49dpd0FAACsouC3QWZmZrFPAuTCb9zRqlWrpsjIyNvebtu2bdW7d+/b3i4AAAAA4Pq4BAN3tJ49e1qlXWY+AAAAAMD/FjMgAAAAAACA1RFAAAAAAAAAqyOAAAAAAAAAVkcAAQAAAAAArI4AAgAAAAAAWB0BBAAAAAAAsDoCCAAAAAAAYHUEEAAAAAAAwOoIIAAAAAAAgNURQAAAAAAAAKsjgAAAAAAAAFZHAAEAAAAAAKyOAAIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDq7Eq7AwD+3h5+aJvc3NxKuxsAAAAA7nDMgAAAAAAAAFZHAAEAAAAAAKyOAAIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDqCCAAAAAAAIDVEUAAAAAAAACrI4AAAAAAAABWRwABAAAAAACsjgACAAAAAABYnV1pdwDA31vI2i9VxqVsaXcDwL/ED/8JL+0uAACAW8QMCAAAAAAAYHUEEAAAAAAAwOoIIAAAAAAAgNURQAAAAAAAAKsjgAAAAAAAAFZHAAEAAAAAAKyOAAIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDqCCAAAAAAAIDVEUAAAAAAAACrI4AAAAAAAABWRwABAAAAAACsjgACAAAAAABYHQEEAAAAAACwOgIIAAAAAABgdQQQAAAAAADA6gggAAAAAACA1RFA4G8vNzdXfn5+2rlz521td+LEiXrqqadua5sAAAAA8G9VqgHE119/ra5du6pKlSqysbHR2rVrLdYbhqHJkyfLx8dHzs7OCgsL0+HDh/9SmyVtd/r06QoJCZGLi4s8PDyK3F90dLQaN24sJycnVa5cWcOHDy9Us337dlWvXl2S9Ntvv2nYsGHy9fWVo6OjvL29FR4erh07dpj1ly5d0vDhw1WhQgW5urqqe/fuSk9Pt2jz6aefVvPmzeXo6KgmTZpct2+GYWju3LmqV6+eHB0dVbVqVU2fPt2iZvny5QoKCpKLi4t8fHz02GOP6cyZM0UerySdP39eo0aNUo0aNeTs7KyQkBDt3r270L5v9BmfPXtWffr0kZubmzw8PDRo0CBlZ2eb62NjY9WtWzf5+PiobNmyatKkiZYvX16oP0uWLFGtWrUUEhJS4raTk5PVsWNHeXl5ycnJSbVr19akSZN0+fJls+aZZ57RsmXL9PPPP1vsLzU1VREREXJxcVHlypU1btw4XblyxVx/6tQpPfroo6pXr55sbW01atSoQn1evXq1WrRoIQ8PD/PY3n///UJ1hw4d0gMPPCB3d3eVLVtWLVu2VGpqqrk+LS1Nffv2lbe3t8qWLatmzZrp008/LdTOn+3evVuhoaHy8PBQ+fLlFR4erh9++OGG2wEAAADArSrVAOLChQsKCgrSokWLrrt+9uzZWrBggZYsWaL4+HiVLVtW4eHhunTp0i23WdJ2c3Nz1aNHDw0bNqzIdubNm6fnnntOEydOVFJSkr788kuFh4cXqlu3bp26du0qSerevbv27t2rZcuW6aefftL69evVoUMHix/9o0eP1oYNG7Rq1Spt375dJ0+e1MMPP1yo3ccee0y9evUqsn8jR47U22+/rblz5+rHH3/U+vXr1apVK3P9jh071K9fPw0aNEhJSUlatWqVdu3apSFDhhTZpiQNHjxYMTExev/997V//37de++9CgsL06+//mrWlOQz7tOnj5KSkhQTE6PPPvtMX3/9tYYOHWqu37lzpxo3bqxPP/1U+/bt08CBA9WvXz999tlnZo1hGHrttdc0aNAgiz7eqG17e3v169dPW7ZsUXJysubPn6+33npLU6ZMMWsqVqyo8PBwLV682FyWl5eniIgI5ebmaufOnVq2bJmio6M1efJksyYnJ0eVKlXSpEmTFBQUdN3P0NPTU88995zi4uLMYxs4cKA2b95s1qSkpKht27by9/dXbGys9u3bp6ioKDk5OZk1/fr1U3JystavX6/9+/fr4YcfVs+ePbV3794iz192drY6d+4sX19fxcfH69tvv1W5cuUUHh5uEcAAAAAAwO1kYxiGUdqdkCQbGxutWbNGDz74oKSrPyyrVKmisWPH6plnnpEkZWZmysvLS9HR0XrkkUduus1baTc6OlqjRo1SRkaGxfJz586patWq2rBhg0JDQ4vth5+fn1577TXdddddKl++vGJjY9W+ffvr1mZmZqpSpUpasWKF/vOf/0iSfvzxRwUEBCguLk533XWXRf3UqVO1du1aJSYmWiw/dOiQGjdurAMHDqh+/frX3dfcuXO1ePFipaSkmMsWLlyoWbNm6ZdffrnuNhcvXlS5cuW0bt06RUREmMubN2+uLl266MUXXyzRZ3zo0CEFBgZq9+7datGihSRp06ZNuu+++/TLL7+oSpUq191/RESEvLy89O6770qSEhIS1Lp1a2VkZKhcuXLmsd9K22PGjNHu3bv1zTffmMvee+89Pffcczpx4oQk6YsvvtD999+vkydPysvLS9LVGRgTJkzQb7/9JgcHB4s2O3TooCZNmmj+/PnX3ee1mjVrpoiICE2bNk2S9Mgjj8je3v66MyMKuLq6avHixerbt6+5rEKFCpo1a5YGDx583W0SEhLMmRQFM3P279+vxo0b6/Dhw/Lz87thXyUpKytL7u7uarDsU5VxKVuibQDgr/rhP4WDfgAAULoKfhtkZmbKzc2tyLo79h4QR48eVVpamsLCwsxl7u7uat26teLi4kq93ZiYGOXn5+vXX39VQECAqlWrpp49e5o/VAskJSXp9OnTuueee+Tq6ipXV1etXbtWOTk51213z549unz5skX//P395evre1P927Bhg2rXrq3PPvtMtWrVUs2aNTV48GCdPXvWrAkODtaJEye0ceNGGYah9PR0ffLJJ7rvvvuKbPfKlSvKy8uz+Cu8JDk7O+vbb7+VVLLPOC4uTh4eHmZAIElhYWGytbVVfHx8kfvPzMyUp6en+f6bb75RvXr1zPDhVts+cuSINm3aVCgYatWqlX755RcdO3bMbLtRo0Zm+CBJ4eHhysrKUlJSUpH9Lo5hGNq6dauSk5N19913S5Ly8/P1+eefq169egoPD1flypXVunXrQpcUhYSE6KOPPtLZs2eVn5+vlStX6tKlS+rQoUOR+6tfv74qVKigd955R7m5ubp48aLeeecdBQQEqGbNmkVul5OTo6ysLIsXAAAAAJTUHRtApKWlSZLFD72C9wXrSrPdn3/+Wfn5+XrppZc0f/58ffLJJzp79qw6deqk3Nxcs27dunUKDw+Xg4OD7OzsFB0drWXLlsnDw0Nt2rTRs88+q3379ln0z8HBodB9J26lf8ePH9eqVav03nvvKTo6Wnv27DFnVUhSmzZttHz5cvXq1UsODg7y9vaWu7t7sZevlCtXTsHBwZo2bZpOnjypvLw8ffDBB4qLi9OpU6fMYyjoc1HHkJaWpsqVK1ust7Ozk6enZ5HH+fHHH2v37t0aOHCguez48eOFZjTcTNshISFycnJS3bp11a5dO73wwgsW6wvaPn78uNn29Y7r2uMuqczMTLm6usrBwUERERFauHChOnXqJEk6ffq0srOzNXPmTHXu3FlbtmzRQw89pIcffljbt2+3+EwuX76sChUqyNHRUY8//rjWrFlT7CyGcuXKKTY2Vh988IGcnZ3l6uqqTZs26YsvvpCdnV2R282YMUPu7u7mq2D2BAAAAACUxB0bQNzp8vPzdfnyZS1YsEDh4eG666679OGHH+rw4cPatm2bWbdu3To98MAD5vvu3bvr5MmTWr9+vTp37qzY2Fg1a9ZM0dHRt71/OTk5eu+999SuXTt16NBB77zzjrZt26bk5GRJ0sGDBzVy5EhNnjxZe/bs0aZNm3Ts2DE98cQTkq7OLiiYteHq6mreAPL999+XYRiqWrWqHB0dtWDBAvXu3Vu2ttYbTtu2bdPAgQP11ltvqUGDBubyixcvFpqNcTM++ugjff/991qxYoU+//xzzZ0712K9s7OzJOmPP/645X0UpVy5ckpMTNTu3bs1ffp0jRkzRrGxsZKunj9J6tatm0aPHq0mTZpo4sSJuv/++7VkyRKzjaioKGVkZOjLL79UQkKCxowZo549e2r//v2SpC5dupjnr+Bzu3jxogYNGqQ2bdrou+++044dO9SwYUNFRETo4sWLRfY3MjJSmZmZ5uvPs30AAAAAoDhF/7mzlHl7e0uS0tPT5ePjYy5PT08v8qkP/8t2C7YNDAw0l1WqVEkVK1Y0n1Jw6tQp7d271+JeCZLk5OSkTp06qVOnToqKitLgwYM1ZcoUDRgwQN7e3srNzVVGRobFLIj09HSz7yXtn52dnerVq2cuCwgIkHT1KQ7169fXjBkz1KZNG40bN06S1LhxY5UtW1bt2rXTiy++qBYtWljcW6LgL/116tTR9u3bdeHCBWVlZcnHx0e9evVS7dq1JZXsM/b29tbp06ct+nzlyhWdPXu20HFu375dXbt21SuvvKJ+/fpZrKtYsaL5Y7vAzbRd8Ff8wMBA5eXlaejQoRo7dqzKlCkjSeYlK5UqVTLb3rVrl0UbBU8ouZnzI0m2trbmTIUmTZro0KFDmjFjhjp06KCKFSvKzs7OYnxJV89hwaUuKSkpeu2113TgwAEzXAgKCtI333yjRYsWacmSJXr77bfNUMHe3l6StGLFCh07dkxxcXFmaLRixQqVL19e69atK/L+Ko6OjnJ0dLypYwQAAACAAnfsDIhatWrJ29tbW7duNZdlZWUpPj5ewcHBpd5umzZtJMmcTSBd/bH6+++/q0aNGpKu3ochJCTE4p4F1xMYGKgLFy5IunozR3t7e4v+JScnKzU19ab7d+XKFYsbTP7000+SZPbvjz/+KDRroeCHt2EYcnZ2lp+fn/m69j4LklS2bFn5+Pjo3Llz2rx5s7p16yapZJ9xcHCwMjIytGfPHrPmq6++Un5+vlq3bm0ui42NVUREhGbNmmXxFIsCTZs21Y8//qhr76Va0rb/rGBWS8HsA0k6cOCA7O3tzR/4wcHB2r9/v0XAERMTIzc3t0Jhwc0qmLUiSQ4ODmrZsqXF+JKunsNrz5+k657DgmOoWrWqef7+fN5tbGzMbQreX3vsAAAAAHA7leoMiOzsbB05csR8f/ToUSUmJsrT01O+vr4aNWqUXnzxRdWtW1e1atVSVFSUqlSpYvFUi9DQUD300EMaMWJEidq0sbEpUbupqak6e/asUlNTlZeXZ84E8PPzk6urq+rVq6du3bpp5MiRevPNN+Xm5qbIyEj5+/urY8eOkqT169dbXH5x5swZ9ejRQ4899pgaN26scuXKKSEhQbNnzzZ/vLu7u2vQoEEaM2aMPD095ebmpqeeekrBwcEWT8A4cuSIsrOzlZaWposXL5r9CwwMlIODg8LCwtSsWTM99thjmj9/vvLz8zV8+HB16tTJnBXRtWtXDRkyRIsXL1Z4eLhOnTqlUaNGqVWrVkU+KUKSNm/eLMMwVL9+fR05ckTjxo2Tv7+/eW+GknzGAQEB6ty5s4YMGaIlS5bo8uXLGjFihB555BFz39u2bdP999+vkSNHqnv37uY9FhwcHMxQp2PHjsrOzlZSUpIaNmxY4raXL18ue3t7NWrUSI6OjkpISFBkZKR69eplzhSQrl6G0q5dO/NSjHvvvVeBgYHq27evZs+erbS0NE2aNEnDhw+3mB1QcD6ys7P122+/KTExUQ4ODmZIMWPGDLVo0UJ16tRRTk6ONm7cqPfff9/ikZ/jxo1Tr169dPfdd6tjx47atGmTNmzYYF6m4e/vLz8/Pz3++OOaO3euKlSooLVr15qPHi1Kp06dNG7cOA0fPlxPPfWU8vPzNXPmTNnZ2ZljFwAAAABut1J9DGdsbOx1f/D0799f0dHRMgxDU6ZM0ZtvvqmMjAy1bdtWr7/+usVlBTVr1tSAAQM0derUErUpqUTtDhgwQMuWLSvUzrZt28wnDGRlZWn06NFavXq1bG1t1b59e7366quqXr26Lly4YF4eUDDNPicnR1OnTtWWLVuUkpKiy5cvq3r16urRo4eeffZZ80fupUuXNHbsWH344YfKyclReHi4Xn/9dYsp/h06dLC4GWGBo0ePmk8yOHnypJ566ilt2bJFZcuWVZcuXfTyyy9bzMhYuHChlixZoqNHj8rDw0P33HOPZs2apapVq17vlEm6euPDyMhI/fLLL/L09FT37t01ffp0ubu7mzUl+YzPnj2rESNGaMOGDbK1tVX37t21YMECubq6FnsO2rdvb/4Il2Re/jFjxowSt/3RRx9p9uzZ+umnn2QYhmrUqKH//ve/Gj16tMU9Jfz9/TV16lSLyxKOHz+uYcOGKTY2VmXLllX//v3NH/AFrp1dUKBGjRrm0zQmTZqkjz76SL/88oucnZ3l7++vkSNHqlevXhbbvPvuu5oxY4Z++eUX1a9fX88//7wZVknS4cOHNXHiRH377bfKzs6Wn5+fnnnmGYvHcl5PTEyMnn/+eR04cEC2trZq2rSppk+fXugxr8XhMZwASgOP4QQA4M5T0sdwlmoA8U+2evVqTZo0SQcPHiztrvzj7du3T506dVJKSooZMNwOX3zxhcaOHat9+/YV+3SIfysCCAClgQACAIA7T0kDiDv2HhB/d66urpo1a1Zpd+NfoXHjxpo1a5aOHj16W9u9cOGCli5dSvgAAAAAALcBMyAA3BJmQAAoDcyAAADgzsMMCAAAAAAAcMcggAAAAAAAAFZHAAEAAAAAAKyOAAIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDqCCAAAAAAAIDVEUAAAAAAAACrI4AAAAAAAABWRwABAAAAAACsjgACAAAAAABYHQEEAAAAAACwOgIIAAAAAABgdQQQAAAAAADA6gggAAAAAACA1dmVdgcA/L3tfDBMbm5upd0NAAAAAHc4ZkAAAAAAAACrI4AAAAAAAABWRwABAAAAAACsjgACAAAAAABYHTehBHBLDMOQJGVlZZVyTwAAAACUpoLfBAW/EYpCAAHglpw5c0aSVL169VLuCQAAAIA7wfnz5+Xu7l7kegIIALfE09NTkpSamlrslwzwv5CVlaXq1avrxIkTPBYWpY7xiDsNYxJ3EsbjP5NhGDp//ryqVKlSbB0BBIBbYmt79RYy7u7u/M8Ddww3NzfGI+4YjEfcaRiTuJMwHv95SvJHSW5CCQAAAAAArI4AAgAAAAAAWB0BBIBb4ujoqClTpsjR0bG0uwIwHnFHYTziTsOYxJ2E8fjvZmPc6DkZAAAAAAAAfxEzIAAAAAAAgNURQAAAAAAAAKsjgAAAAAAAAFZHAAEAAAAAAKyOAALATVu0aJFq1qwpJycntW7dWrt27SrtLuFv6Ouvv1bXrl1VpUoV2djYaO3atRbrDcPQ5MmT5ePjI2dnZ4WFhenw4cMWNWfPnlWfPn3k5uYmDw8PDRo0SNnZ2RY1+/btU7t27eTk5KTq1atr9uzZhfqyatUq+fv7y8nJSY0aNdLGjRtv+/HizjVjxgy1bNlS5cqVU+XKlfXggw8qOTnZoubSpUsaPny4KlSoIFdXV3Xv3l3p6ekWNampqYqIiJCLi4sqV66scePG6cqVKxY1sbGxatasmRwdHeXn56fo6OhC/eE7FosXL1bjxo3l5uYmNzc3BQcH64svvjDXMx5RmmbOnCkbGxuNGjXKXMaYRIkZAHATVq5caTg4OBjvvvuukZSUZAwZMsTw8PAw0tPTS7tr+JvZuHGj8dxzzxmrV682JBlr1qyxWD9z5kzD3d3dWLt2rfHDDz8YDzzwgFGrVi3j4sWLZk3nzp2NoKAg47vvvjO++eYbw8/Pz+jdu7e5PjMz0/Dy8jL69OljHDhwwPjwww8NZ2dn44033jBrduzYYZQpU8aYPXu2cfDgQWPSpEmGvb29sX//fqt/BrgzhIeHG0uXLjUOHDhgJCYmGvfdd5/h6+trZGdnmzVPPPGEUb16dWPr1q1GQkKCcddddxkhISHm+itXrhgNGzY0wsLCjL179xobN240KlasaERGRpo1P//8s+Hi4mKMGTPGOHjwoLFw4UKjTJkyxqZNm8wavmNhGIaxfv164/PPPzd++uknIzk52Xj22WcNe3t748CBA4ZhMB5Renbt2mXUrFnTaNy4sTFy5EhzOWMSJUUAAeCmtGrVyhg+fLj5Pi8vz6hSpYoxY8aMUuwV/u7+HEDk5+cb3t7expw5c8xlGRkZhqOjo/Hhhx8ahmEYBw8eNCQZu3fvNmu++OILw8bGxvj1118NwzCM119/3ShfvryRk5Nj1kyYMMGoX7+++b5nz55GRESERX9at25tPP7447f1GPH3cfr0aUOSsX37dsMwro49e3t7Y9WqVWbNoUOHDElGXFycYRhXAzVbW1sjLS3NrFm8eLHh5uZmjr/x48cbDRo0sNhXr169jPDwcPM937EoSvny5Y23336b8YhSc/78eaNu3bpGTEyM0b59ezOAYEziZnAJBoASy83N1Z49exQWFmYus7W1VVhYmOLi4kqxZ/inOXr0qNLS0izGmru7u1q3bm2Otbi4OHl4eKhFixZmTVhYmGxtbRUfH2/W3H333XJwcDBrwsPDlZycrHPnzpk11+6noIYx/e+VmZkpSfL09JQk7dmzR5cvX7YYJ/7+/vL19bUYj40aNZKXl5dZEx4erqysLCUlJZk1xY01vmNxPXl5eVq5cqUuXLig4OBgxiNKzfDhwxUREVFo3DAmcTPsSrsDAP4+fv/9d+Xl5Vn8z0OSvLy89OOPP5ZSr/BPlJaWJknXHWsF69LS0lS5cmWL9XZ2dvL09LSoqVWrVqE2CtaVL19eaWlpxe4H/y75+fkaNWqU2rRpo4YNG0q6OlYcHBzk4eFhUfvn8Xi9cVSwrriarKwsXbx4UefOneM7Fqb9+/crODhYly5dkqurq9asWaPAwEAlJiYyHvE/t3LlSn3//ffavXt3oXV8R+JmEEAAAAD8n+HDh+vAgQP69ttvS7sr+JerX7++EhMTlZmZqU8++UT9+/fX9u3bS7tb+Bc6ceKERo4cqZiYGDk5OZV2d/A3xyUYAEqsYsWKKlOmTKG7Gqenp8vb27uUeoV/ooLxVNxY8/b21unTpy3WX7lyRWfPnrWouV4b1+6jqBrG9L/PiBEj9Nlnn2nbtm2qVq2audzb21u5ubnKyMiwqP/zeLzVsebm5iZnZ2e+Y2HBwcFBfn5+at68uWbMmKGgoCC9+uqrjEf8z+3Zs0enT59Ws2bNZGdnJzs7O23fvl0LFiyQnZ2dvLy8GJMoMQIIACXm4OCg5s2ba+vWreay/Px8bd26VcHBwaXYM/zT1KpVS97e3hZjLSsrS/Hx8eZYCw4OVkZGhvbs2WPWfPXVV8rPz1fr1q3Nmq+//lqXL182a2JiYlS/fn2VL1/erLl2PwU1jOl/D8MwNGLECK1Zs0ZfffVVoct2mjdvLnt7e4txkpycrNTUVIvxuH//fotQLCYmRm5ubgoMDDRrihtrfMeiOPn5+crJyWE84n8uNDRU+/fvV2Jiovlq0aKF+vTpY/6bMYkSK+27YAL4e1m5cqXh6OhoREdHGwcPHjSGDh1qeHh4WNzVGCiJ8+fPG3v37jX27t1rSDLmzZtn7N271zh+/LhhGFcfw+nh4WGsW7fO2Ldvn9GtW7frPoazadOmRnx8vPHtt98adevWtXgMZ0ZGhuHl5WX07dvXOHDggLFy5UrDxcWl0GM47ezsjLlz5xqHDh0ypkyZwmM4/2WGDRtmuLu7G7GxscapU6fM1x9//GHWPPHEE4avr6/x1VdfGQkJCUZwcLARHBxsri94xNy9995rJCYmGps2bTIqVap03UfMjRs3zjh06JCxaNGi6z5iju9YTJw40di+fbtx9OhRY9++fcbEiRMNGxsbY8uWLYZhMB5R+q59CoZhMCZRcgQQAG7awoULDV9fX8PBwcFo1aqV8d1335V2l/A3tG3bNkNSoVf//v0Nw7j6KM6oqCjDy8vLcHR0NEJDQ43k5GSLNs6cOWP07t3bcHV1Ndzc3IyBAwca58+ft6j54YcfjLZt2xqOjo5G1apVjZkzZxbqy8cff2zUq1fPcHBwMBo0aGB8/vnnVjtu3HmuNw4lGUuXLjVrLl68aDz55JNG+fLlDRcXF+Ohhx4yTp06ZdHOsWPHjC5duhjOzs5GxYoVjbFjxxqXL1+2qNm2bZvRpEkTw8HBwahdu7bFPgrwHYvHHnvMqFGjhuHg4GBUqlTJCA0NNcMHw2A8ovT9OYBgTKKkbAzDMEpn7gUAAAAAAPi34B4QAAAAAADA6gggAAAAAACA1RFAAAAAAAAAqyOAAAAAAAAAVkcAAQAAAAAArI4AAgAAAAAAWB0BBAAAAAAAsDoCCAAAgH+QY8eOycbGRomJiUXWxMbGysbGRhkZGX9pXx06dNCoUaNuapupU6eqSZMmf2m/xSnJ8QMASgcBBAAAwG2Wlpamp556SrVr15ajo6OqV6+url27auvWrbd1PwMGDNCDDz5osax69eo6deqUGjZseFv39Xfx5+O/XWELAOCvsyvtDgAAAPyTHDt2TG3atJGHh4fmzJmjRo0a6fLly9q8ebOGDx+uH3/80ar7L1OmjLy9va26jzvZv/34AeBOxgwIAACA2+jJJ5+UjY2Ndu3ape7du6tevXpq0KCBxowZo++++86smzdvnho1aqSyZcuqevXqevLJJ5WdnW2uj46OloeHhzZv3qyAgAC5urqqc+fOOnXqlKSrlzIsW7ZM69atk42NjWxsbBQbG3vdSxA2btyoevXqydnZWR07dtSxY8cs+nzmzBn17t1bVatWlYuLixo1aqQPP/zQoubChQvq16+fXF1d5ePjo5dffrlEn8fMmTPl5eWlcuXKadCgQbp06VKhmrffflsBAQFycnKSv7+/Xn/99WLbzM/P1+zZs+Xn5ydHR0f5+vpq+vTpkiwvwTh27Jg6duwoSSpfvrxsbGw0YMAASdKmTZvUtm1beXh4qEKFCrr//vuVkpJi7iM3N1cjRoyQj4+PnJycVKNGDc2YMaNExwwAuD4CCAAAgNvk7Nmz2rRpk4YPH66yZcsWWu/h4WH+29bWVgsWLFBSUpKWLVumr776SuPHj7eo/+OPPzR37ly9//77+vrrr5WamqpnnnlGkvTMM8+oZ8+eZihx6tQphYSEFNrniRMn9PDDD6tr165KTEzU4MGDNXHiRIuaS5cuqXnz5vr888914MABDR06VH379tWuXbvMmnHjxmn79u1at26dtmzZotjYWH3//ffFfh4ff/yxpk6dqpdeekkJCQny8fEpFC4sX75ckydP1vTp03Xo0CG99NJLioqK0rJly4psNzIyUjNnzlRUVJQOHjyoFStWyMvLq1Bd9erV9emnn0qSkpOTderUKb366quSrgYqY8aMUUJCgrZu3SpbW1s99NBDys/PlyQtWLBA69ev18cff6zk5GQtX75cNWvWLPZ4AQA3YAAAAOC2iI+PNyQZq1evvultV61aZVSoUMF8v3TpUkOSceTIEXPZokWLDC8vL/N9//79jW7dulm0c/ToUUOSsXfvXsMwDCMyMtIIDAy0qJkwYYIhyTh37lyR/YmIiDDGjh1rGIZhnD9/3nBwcDA+/vhjc/2ZM2cMZ2dnY+TIkUW2ERwcbDz55JMWy1q3bm0EBQWZ7+vUqWOsWLHCombatGlGcHDwddvMysoyHB0djbfeeuu66/98/Nu2bbvhsRqGYfz222+GJGP//v2GYRjGU089Zdxzzz1Gfn5+sdsBAEqOGRAAAAC3iWEYJa798ssvFRoaqqpVq6pcuXLq27evzpw5oz/++MOscXFxUZ06dcz3Pj4+On369E316dChQ2rdurXFsuDgYIv3eXl5mjZtmho1aiRPT0+5urpq8+bNSk1NlSSlpKQoNzfXoh1PT0/Vr1//L+37woULSklJ0aBBg+Tq6mq+XnzxRYvLIf7cZk5OjkJDQ2988MU4fPiwevfurdq1a8vNzc2c3VBwzAMGDFBiYqLq16+vp59+Wlu2bPlL+wMAcBNKAACA26Zu3bqysbG54Y0mjx07pvvvv1/Dhg3T9OnT5enpqW+//VaDBg1Sbm6uXFxcJEn29vYW29nY2NxUyFFSc+bM0auvvqr58+eb96UYNWqUcnNzb/u+rlVwz4u33nqrUFBRpkyZ627j7Ox8W/bdtWtX1ahRQ2+99ZaqVKmi/Px8NWzY0DzmZs2a6ejRo/riiy/05ZdfqmfPngoLC9Mnn3xyW/YPAP9GzIAAAAC4TTw9PRUeHq5FixbpwoULhdYXPApyz549ys/P18svv6y77rpL9erV08mTJ296fw4ODsrLyyu2JiAgwOJeDpIsboYpSTt27FC3bt303//+V0FBQapdu7Z++uknc32dOnVkb2+v+Ph4c9m5c+csaora97Xb/HnfXl5eqlKlin7++Wf5+flZvGrVqnXdNuvWrStnZ+cSP9LUwcFBkiw+pzNnzig5OVmTJk1SaGioAgICdO7cuULburm5qVevXnrrrbf00Ucf6dNPP9XZs2dLtF8AQGHMgAAAALiNFi1apDZt2qhVq1Z64YUX1LhxY125ckUxMTFavHixDh06JD8/P12+fFkLFy5U165dtWPHDi1ZsuSm91WzZk1t3rxZycnJqlChgtzd3QvVPPHEE3r55Zc1btw4DR48WHv27FF0dLRFTd26dfXJJ59o586dKl++vObNm6f09HQFBgZKklxdXTVo0CCNGzdOFSpUUOXKlfXcc8/J1rb4v2WNHDlSAwYMUIsWLdSmTRstX75cSUlJql27tlnz/PPP6+mnn5a7u7s6d+6snJwcJSQk6Ny5cxozZkyhNp2cnDRhwgSNHz9eDg4OatOmjX777TclJSVp0KBBhepr1KghGxsbffbZZ7rvvvvk7Oys8uXLq0KFCnrzzTfl4+Oj1NTUQjfmnDdvnnx8fNS0aVPZ2tpq1apV8vb2triRKADg5jADAgAA4DaqXbu2vv/+e3Xs2FFjx45Vw4YN1alTJ23dulWLFy+WJAUFBWnevHmaNWuWGjZsqOXLl9/SIx6HDBmi+vXrq0WLFqpUqZJ27NhRqMbX11effvqp1q5dq6CgIC1ZskQvvfSSRc2kSZPUrFkzhYeHq0OHDvL29taDDz5oUTNnzhy1a9dOXbt2VVhYmNq2bavmzZsX279evXopKipK48ePV/PmzXX8+HENGzbMombw4MF6++23tXTpUjVq1Ejt27dXdHR0kTMgJCkqKkpjx47V5MmTFRAQoF69ehV5b4yqVavq+eef18SJE+Xl5aURI0bI1tZWK1eu1J49e9SwYUONHj1ac+bMsdiuXLlymj17tlq0aKGWLVvq2LFj2rhx4w1DFwBA0WwMa1xICAAAAAAAcA0iXAAAAAAAYHUEEAAAAAAAwOoIIAAAAAAAgNURQAAAAAAAAKsjgAAAAAAAAFZHAAEAAAAAAKyOAAIAAAAAAFgdAQQAAAAAALA6AggAAAAAAGB1BBAAAAAAAMDqCCAAAAAAAIDVEUAAAAAAAACr+382sh0MgubCyQAAAABJRU5ErkJggg==",
      "text/plain": [
       "<Figure size 1000x800 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "\n",
    "top_cited_papers = dataframe.nlargest(10, 'Times Cited, All Databases')\n",
    "\n",
    "plt.figure(figsize=(10,8))\n",
    "sns.barplot(x='Times Cited, All Databases', y=top_cited_papers['DOI'], data=top_cited_papers, orient='h')\n",
    "plt.title('Top 10 papers más citados')\n",
    "plt.xlabel('Cantidad de citas')\n",
    "plt.ylabel('DOI del paper')\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index(['Authors', 'Author Full Names', 'Source Title',\n",
      "       'Times Cited, All Databases', 'DOI', 'Abstract', 'Document Type'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "source": [
    "print(dataframe.columns)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Authors</th>\n",
       "      <th>Author Full Names</th>\n",
       "      <th>Source Title</th>\n",
       "      <th>Times Cited, All Databases</th>\n",
       "      <th>DOI</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Document Type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....</td>\n",
       "      <td>Delaglio, Frank (6603722428); Grzesiek, Stepha...</td>\n",
       "      <td>Journal of Biomolecular NMR</td>\n",
       "      <td>11644</td>\n",
       "      <td>10.1007/BF00197809</td>\n",
       "      <td>The NMRPipe system is a UNIX software environm...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...</td>\n",
       "      <td>Jumper, John (36717725300); Evans, Richard (57...</td>\n",
       "      <td>Nature</td>\n",
       "      <td>6981</td>\n",
       "      <td>10.1038/s41586-021-03819-2</td>\n",
       "      <td>Proteins are essential to life, and understand...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Wild S.; Roglic G.; Green A.; Sicree R.; King H.</td>\n",
       "      <td>Wild, Sarah (7103086309); Roglic, Gojka (67016...</td>\n",
       "      <td>Diabetes Care</td>\n",
       "      <td>12069</td>\n",
       "      <td>10.2337/diacare.27.5.1047</td>\n",
       "      <td>OBJECTIVE - The goal of this study was to esti...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Cover T.M.; Thomas J.A.</td>\n",
       "      <td>Cover, Thomas M. (7102102743); Thomas, Joy A. ...</td>\n",
       "      <td>Elements of Information Theory</td>\n",
       "      <td>33728</td>\n",
       "      <td>10.1002/047174882X</td>\n",
       "      <td>The latest edition of this classic is updated ...</td>\n",
       "      <td>Book</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>LeCun Y.; Bottou L.; Bengio Y.; Haffner P.</td>\n",
       "      <td>LeCun, Yann (55666793600); Bottou, Léon (67017...</td>\n",
       "      <td>Proceedings of the IEEE</td>\n",
       "      <td>31945</td>\n",
       "      <td>10.1109/5.726791</td>\n",
       "      <td>Multilayer neural networks trained with the ba...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1808</th>\n",
       "      <td>Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...</td>\n",
       "      <td>Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...</td>\n",
       "      <td>APPLIED SCIENCES-BASEL</td>\n",
       "      <td>0</td>\n",
       "      <td>10.3390/app122412946</td>\n",
       "      <td>The language inclusion checking of timed autom...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1809</th>\n",
       "      <td>Roelens, B; Poels, G</td>\n",
       "      <td>Roelens, Ben; Poels, Geert</td>\n",
       "      <td>BUSINESS &amp; INFORMATION SYSTEMS ENGINEERING</td>\n",
       "      <td>7</td>\n",
       "      <td>10.1007/s12599-014-0363-z</td>\n",
       "      <td>Business models (BM) are the central concept t...</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>Pulvermuller, F</td>\n",
       "      <td>Pulvermueller, Friedemann</td>\n",
       "      <td>PROGRESS IN NEUROBIOLOGY</td>\n",
       "      <td>104</td>\n",
       "      <td>10.1016/j.pneurobio.2017.07.001</td>\n",
       "      <td>Neurocognitive and neurolinguistics theories m...</td>\n",
       "      <td>Review</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1811</th>\n",
       "      <td>Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...</td>\n",
       "      <td>Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...</td>\n",
       "      <td>ACM TRANSACTIONS ON GRAPHICS</td>\n",
       "      <td>21</td>\n",
       "      <td>10.1145/3272127.3275035</td>\n",
       "      <td>We introduce a novel framework for using natur...</td>\n",
       "      <td>Article; Proceedings Paper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1812</th>\n",
       "      <td>Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...</td>\n",
       "      <td>Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...</td>\n",
       "      <td>ACM SIGPLAN NOTICES</td>\n",
       "      <td>322</td>\n",
       "      <td>10.1145/780822.781133</td>\n",
       "      <td>We present nesC, a programming language for ne...</td>\n",
       "      <td>Article; Proceedings Paper</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1810 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                Authors   \n",
       "0     Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....  \\\n",
       "2     Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...   \n",
       "3      Wild S.; Roglic G.; Green A.; Sicree R.; King H.   \n",
       "4                               Cover T.M.; Thomas J.A.   \n",
       "5            LeCun Y.; Bottou L.; Bengio Y.; Haffner P.   \n",
       "...                                                 ...   \n",
       "1808  Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...   \n",
       "1809                               Roelens, B; Poels, G   \n",
       "1810                                    Pulvermuller, F   \n",
       "1811  Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...   \n",
       "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
       "\n",
       "                                      Author Full Names   \n",
       "0     Delaglio, Frank (6603722428); Grzesiek, Stepha...  \\\n",
       "2     Jumper, John (36717725300); Evans, Richard (57...   \n",
       "3     Wild, Sarah (7103086309); Roglic, Gojka (67016...   \n",
       "4     Cover, Thomas M. (7102102743); Thomas, Joy A. ...   \n",
       "5     LeCun, Yann (55666793600); Bottou, Léon (67017...   \n",
       "...                                                 ...   \n",
       "1808  Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...   \n",
       "1809                         Roelens, Ben; Poels, Geert   \n",
       "1810                          Pulvermueller, Friedemann   \n",
       "1811  Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...   \n",
       "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
       "\n",
       "                                    Source Title  Times Cited, All Databases   \n",
       "0                    Journal of Biomolecular NMR                       11644  \\\n",
       "2                                         Nature                        6981   \n",
       "3                                  Diabetes Care                       12069   \n",
       "4                 Elements of Information Theory                       33728   \n",
       "5                        Proceedings of the IEEE                       31945   \n",
       "...                                          ...                         ...   \n",
       "1808                      APPLIED SCIENCES-BASEL                           0   \n",
       "1809  BUSINESS & INFORMATION SYSTEMS ENGINEERING                           7   \n",
       "1810                    PROGRESS IN NEUROBIOLOGY                         104   \n",
       "1811                ACM TRANSACTIONS ON GRAPHICS                          21   \n",
       "1812                         ACM SIGPLAN NOTICES                         322   \n",
       "\n",
       "                                  DOI   \n",
       "0                  10.1007/BF00197809  \\\n",
       "2          10.1038/s41586-021-03819-2   \n",
       "3           10.2337/diacare.27.5.1047   \n",
       "4                  10.1002/047174882X   \n",
       "5                    10.1109/5.726791   \n",
       "...                               ...   \n",
       "1808             10.3390/app122412946   \n",
       "1809        10.1007/s12599-014-0363-z   \n",
       "1810  10.1016/j.pneurobio.2017.07.001   \n",
       "1811          10.1145/3272127.3275035   \n",
       "1812            10.1145/780822.781133   \n",
       "\n",
       "                                               Abstract   \n",
       "0     The NMRPipe system is a UNIX software environm...  \\\n",
       "2     Proteins are essential to life, and understand...   \n",
       "3     OBJECTIVE - The goal of this study was to esti...   \n",
       "4     The latest edition of this classic is updated ...   \n",
       "5     Multilayer neural networks trained with the ba...   \n",
       "...                                                 ...   \n",
       "1808  The language inclusion checking of timed autom...   \n",
       "1809  Business models (BM) are the central concept t...   \n",
       "1810  Neurocognitive and neurolinguistics theories m...   \n",
       "1811  We introduce a novel framework for using natur...   \n",
       "1812  We present nesC, a programming language for ne...   \n",
       "\n",
       "                   Document Type  \n",
       "0                        Article  \n",
       "2                        Article  \n",
       "3                        Article  \n",
       "4                           Book  \n",
       "5                        Article  \n",
       "...                          ...  \n",
       "1808                     Article  \n",
       "1809                     Article  \n",
       "1810                      Review  \n",
       "1811  Article; Proceedings Paper  \n",
       "1812  Article; Proceedings Paper  \n",
       "\n",
       "[1810 rows x 7 columns]"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Source Title</th>\n",
       "      <th>Author Full Names</th>\n",
       "      <th>Document Type</th>\n",
       "      <th>Times Cited, All Databases</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Authors</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1007/BF00197809</td>\n",
       "      <td>Journal of Biomolecular NMR</td>\n",
       "      <td>Delaglio, Frank (6603722428); Grzesiek, Stepha...</td>\n",
       "      <td>Article</td>\n",
       "      <td>11644</td>\n",
       "      <td>The NMRPipe system is a UNIX software environm...</td>\n",
       "      <td>Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1038/s41586-021-03819-2</td>\n",
       "      <td>Nature</td>\n",
       "      <td>Jumper, John (36717725300); Evans, Richard (57...</td>\n",
       "      <td>Article</td>\n",
       "      <td>6981</td>\n",
       "      <td>Proteins are essential to life, and understand...</td>\n",
       "      <td>Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.2337/diacare.27.5.1047</td>\n",
       "      <td>Diabetes Care</td>\n",
       "      <td>Wild, Sarah (7103086309); Roglic, Gojka (67016...</td>\n",
       "      <td>Article</td>\n",
       "      <td>12069</td>\n",
       "      <td>OBJECTIVE - The goal of this study was to esti...</td>\n",
       "      <td>Wild S.; Roglic G.; Green A.; Sicree R.; King H.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1002/047174882X</td>\n",
       "      <td>Elements of Information Theory</td>\n",
       "      <td>Cover, Thomas M. (7102102743); Thomas, Joy A. ...</td>\n",
       "      <td>Book</td>\n",
       "      <td>33728</td>\n",
       "      <td>The latest edition of this classic is updated ...</td>\n",
       "      <td>Cover T.M.; Thomas J.A.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>10.1109/5.726791</td>\n",
       "      <td>Proceedings of the IEEE</td>\n",
       "      <td>LeCun, Yann (55666793600); Bottou, Léon (67017...</td>\n",
       "      <td>Article</td>\n",
       "      <td>31945</td>\n",
       "      <td>Multilayer neural networks trained with the ba...</td>\n",
       "      <td>LeCun Y.; Bottou L.; Bengio Y.; Haffner P.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1808</th>\n",
       "      <td>10.3390/app122412946</td>\n",
       "      <td>APPLIED SCIENCES-BASEL</td>\n",
       "      <td>Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...</td>\n",
       "      <td>Article</td>\n",
       "      <td>0</td>\n",
       "      <td>The language inclusion checking of timed autom...</td>\n",
       "      <td>Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1809</th>\n",
       "      <td>10.1007/s12599-014-0363-z</td>\n",
       "      <td>BUSINESS &amp; INFORMATION SYSTEMS ENGINEERING</td>\n",
       "      <td>Roelens, Ben; Poels, Geert</td>\n",
       "      <td>Article</td>\n",
       "      <td>7</td>\n",
       "      <td>Business models (BM) are the central concept t...</td>\n",
       "      <td>Roelens, B; Poels, G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>10.1016/j.pneurobio.2017.07.001</td>\n",
       "      <td>PROGRESS IN NEUROBIOLOGY</td>\n",
       "      <td>Pulvermueller, Friedemann</td>\n",
       "      <td>Review</td>\n",
       "      <td>104</td>\n",
       "      <td>Neurocognitive and neurolinguistics theories m...</td>\n",
       "      <td>Pulvermuller, F</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1811</th>\n",
       "      <td>10.1145/3272127.3275035</td>\n",
       "      <td>ACM TRANSACTIONS ON GRAPHICS</td>\n",
       "      <td>Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...</td>\n",
       "      <td>Article; Proceedings Paper</td>\n",
       "      <td>21</td>\n",
       "      <td>We introduce a novel framework for using natur...</td>\n",
       "      <td>Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1812</th>\n",
       "      <td>10.1145/780822.781133</td>\n",
       "      <td>ACM SIGPLAN NOTICES</td>\n",
       "      <td>Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...</td>\n",
       "      <td>Article; Proceedings Paper</td>\n",
       "      <td>322</td>\n",
       "      <td>We present nesC, a programming language for ne...</td>\n",
       "      <td>Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1810 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  DOI   \n",
       "0                  10.1007/BF00197809  \\\n",
       "2          10.1038/s41586-021-03819-2   \n",
       "3           10.2337/diacare.27.5.1047   \n",
       "4                  10.1002/047174882X   \n",
       "5                    10.1109/5.726791   \n",
       "...                               ...   \n",
       "1808             10.3390/app122412946   \n",
       "1809        10.1007/s12599-014-0363-z   \n",
       "1810  10.1016/j.pneurobio.2017.07.001   \n",
       "1811          10.1145/3272127.3275035   \n",
       "1812            10.1145/780822.781133   \n",
       "\n",
       "                                    Source Title   \n",
       "0                    Journal of Biomolecular NMR  \\\n",
       "2                                         Nature   \n",
       "3                                  Diabetes Care   \n",
       "4                 Elements of Information Theory   \n",
       "5                        Proceedings of the IEEE   \n",
       "...                                          ...   \n",
       "1808                      APPLIED SCIENCES-BASEL   \n",
       "1809  BUSINESS & INFORMATION SYSTEMS ENGINEERING   \n",
       "1810                    PROGRESS IN NEUROBIOLOGY   \n",
       "1811                ACM TRANSACTIONS ON GRAPHICS   \n",
       "1812                         ACM SIGPLAN NOTICES   \n",
       "\n",
       "                                      Author Full Names   \n",
       "0     Delaglio, Frank (6603722428); Grzesiek, Stepha...  \\\n",
       "2     Jumper, John (36717725300); Evans, Richard (57...   \n",
       "3     Wild, Sarah (7103086309); Roglic, Gojka (67016...   \n",
       "4     Cover, Thomas M. (7102102743); Thomas, Joy A. ...   \n",
       "5     LeCun, Yann (55666793600); Bottou, Léon (67017...   \n",
       "...                                                 ...   \n",
       "1808  Wang, Ting; Shen, Yan; Chen, Tieming; Ji, Baiy...   \n",
       "1809                         Roelens, Ben; Poels, Geert   \n",
       "1810                          Pulvermueller, Friedemann   \n",
       "1811  Ma, Rui; Patil, Akshay Gadi; Fisher, Matthew; ...   \n",
       "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...   \n",
       "\n",
       "                   Document Type  Times Cited, All Databases   \n",
       "0                        Article                       11644  \\\n",
       "2                        Article                        6981   \n",
       "3                        Article                       12069   \n",
       "4                           Book                       33728   \n",
       "5                        Article                       31945   \n",
       "...                          ...                         ...   \n",
       "1808                     Article                           0   \n",
       "1809                     Article                           7   \n",
       "1810                      Review                         104   \n",
       "1811  Article; Proceedings Paper                          21   \n",
       "1812  Article; Proceedings Paper                         322   \n",
       "\n",
       "                                               Abstract   \n",
       "0     The NMRPipe system is a UNIX software environm...  \\\n",
       "2     Proteins are essential to life, and understand...   \n",
       "3     OBJECTIVE - The goal of this study was to esti...   \n",
       "4     The latest edition of this classic is updated ...   \n",
       "5     Multilayer neural networks trained with the ba...   \n",
       "...                                                 ...   \n",
       "1808  The language inclusion checking of timed autom...   \n",
       "1809  Business models (BM) are the central concept t...   \n",
       "1810  Neurocognitive and neurolinguistics theories m...   \n",
       "1811  We introduce a novel framework for using natur...   \n",
       "1812  We present nesC, a programming language for ne...   \n",
       "\n",
       "                                                Authors  \n",
       "0     Delaglio F.; Grzesiek S.; Vuister G.W.; Zhu G....  \n",
       "2     Jumper J.; Evans R.; Pritzel A.; Green T.; Fig...  \n",
       "3      Wild S.; Roglic G.; Green A.; Sicree R.; King H.  \n",
       "4                               Cover T.M.; Thomas J.A.  \n",
       "5            LeCun Y.; Bottou L.; Bengio Y.; Haffner P.  \n",
       "...                                                 ...  \n",
       "1808  Wang, T; Shen, Y; Chen, TM; Ji, BY; Zhu, TT; L...  \n",
       "1809                               Roelens, B; Poels, G  \n",
       "1810                                    Pulvermuller, F  \n",
       "1811  Ma, R; Patil, AG; Fisher, M; Li, MY; Pirk, S; ...  \n",
       "1812  Gay, D; Levis, P; von Behren, R; Welsh, M; Bre...  \n",
       "\n",
       "[1810 rows x 7 columns]"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataframe = dataframe[['DOI', 'Source Title', 'Author Full Names', 'Document Type', 'Times Cited, All Databases', 'Abstract','Authors']]\n",
    "\n",
    "dataframe\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Paper más publicado:  COMPUTER SPEECH AND LANGUAGE\n"
     ]
    }
   ],
   "source": [
    "most_published_paper = dataframe['Source Title'].value_counts().idxmax()\n",
    "print(\"Paper más publicado: \", most_published_paper)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Autor más prominente:  Wood, Simon N. (7401448389)\n",
      "Paper más relevante de autor prominente:  48\n"
     ]
    }
   ],
   "source": [
    "prominent_author = dataframe['Author Full Names'].value_counts().idxmax()\n",
    "print(\"Autor más prominente: \", prominent_author)\n",
    "most_relevant_paper_from_prominent_author = dataframe[dataframe['Author Full Names'] == prominent_author]['Times Cited, All Databases'].idxmax()\n",
    "print(\"Paper más relevante de autor prominente: \", most_relevant_paper_from_prominent_author)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: [(10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Delaglio F.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Grzesiek S.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Vuister G.W.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Zhu G.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Pfeifer J.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Bax A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Jumper J.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Evans R.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Pritzel A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Green T.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Figurnov M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Ronneberger O.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Tunyasuvunakool K.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Bates R.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Žídek A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Potapenko A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Bridgland A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Meyer C.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Kohl S.A.A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Ballard A.J.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Cowie A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Romera-Paredes B.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Nikolov S.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Jain R.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Adler J.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Back T.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Petersen S.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Reiman D.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Clancy E.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Zielinski M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Steinegger M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Pacholska M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Berghammer T.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Bodenstein S.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Silver D.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Vinyals O.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Senior A.W.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Kavukcuoglu K.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Kohli P.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Hassabis D.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Wild S.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Roglic G.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Green A.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Sicree R.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., King H.), (10.1002/047174882X, Cover, Thomas M. (7102102743); Thomas, Joy A. (7406570513), Elements of Information Theory, Book, 33728, The latest edition of this classic is updated with new problem sets and material. The Second Edition of this fundamental textbook maintains the book's tradition of clear, thought-provoking instruction. Readers are provided once again with an instructive mix of mathematics, physics, statistics, and information theory. All the essential topics in information theory are covered in detail, including entropy, data compression, channel capacity, rate distortion, network information theory, and hypothesis testing. The authors provide readers with a solid understanding of the underlying theory and applications. Problem sets and a telegraphic summary at the end of each chapter further assist readers. The historical notes that follow each chapter recap the main points. The Second Edition features: Chapters reorganized to improve teaching. 200 new problems. New material on source coding, portfolio theory, and feedback capacity. Updated references. Now current and enhanced, the Second Edition of Elements of Information Theory remains the ideal textbook for upper-level undergraduate and graduate courses in electrical engineering, statistics, and telecommunications. An Instructor's Manual presenting detailed solutions to all the problems in the book is available from the Wiley editorial department. © 2006 John Wiley & Sons, Inc.., Cover T.M.), (10.1002/047174882X, Cover, Thomas M. (7102102743); Thomas, Joy A. (7406570513), Elements of Information Theory, Book, 33728, The latest edition of this classic is updated with new problem sets and material. The Second Edition of this fundamental textbook maintains the book's tradition of clear, thought-provoking instruction. Readers are provided once again with an instructive mix of mathematics, physics, statistics, and information theory. All the essential topics in information theory are covered in detail, including entropy, data compression, channel capacity, rate distortion, network information theory, and hypothesis testing. The authors provide readers with a solid understanding of the underlying theory and applications. Problem sets and a telegraphic summary at the end of each chapter further assist readers. The historical notes that follow each chapter recap the main points. The Second Edition features: Chapters reorganized to improve teaching. 200 new problems. New material on source coding, portfolio theory, and feedback capacity. Updated references. Now current and enhanced, the Second Edition of Elements of Information Theory remains the ideal textbook for upper-level undergraduate and graduate courses in electrical engineering, statistics, and telecommunications. An Instructor's Manual presenting detailed solutions to all the problems in the book is available from the Wiley editorial department. © 2006 John Wiley & Sons, Inc.., Thomas J.A.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., LeCun Y.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., Bottou L.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., Bengio Y.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., Haffner P.), (10.1109/ICCV.2015.169, Girshick, Ross (35179333300), Proceedings of the IEEE International Conference on Computer Vision, Conference paper, 14787, This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN) for object detection. Fast R-CNN builds on previous work to efficiently classify object proposals using deep convolutional networks. Compared to previous work, Fast R-CNN employs several innovations to improve training and testing speed while also increasing detection accuracy. Fast R-CNN trains the very deep VGG16 network 9x faster than R-CNN, is 213x faster at test-time, and achieves a higher mAP on PASCAL VOC 2012. Compared to SPPnet, Fast R-CNN trains VGG16 3x faster, tests 10x faster, and is more accurate. Fast R-CNN is implemented in Python and C++ (using Caffe) and is available under the open-source MIT License at https://github.com/rbgirshick/fast-rcnn. © 2015 IEEE., Girshick R.), (10.1371/journal.pone.0061217, McMurdie, Paul J. (16245746800); Holmes, Susan (7203030601), PLoS ONE, Article, 8493, Background:The analysis of microbial communities through DNA sequencing brings many challenges: the integration of different types of data with methods from ecology, genetics, phylogenetics, multivariate statistics, visualization and testing. With the increased breadth of experimental designs now being pursued, project-specific statistical analyses are often needed, and these analyses are often difficult (or impossible) for peer researchers to independently reproduce. The vast majority of the requisite tools for performing these analyses reproducibly are already implemented in R and its extensions (packages), but with limited support for high throughput microbiome census data.Results:Here we describe a software project, phyloseq, dedicated to the object-oriented representation and analysis of microbiome census data in R. It supports importing data from a variety of common formats, as well as many analysis techniques. These include calibration, filtering, subsetting, agglomeration, multi-table comparisons, diversity analysis, parallelized Fast UniFrac, ordination methods, and production of publication-quality graphics; all in a manner that is easy to document, share, and modify. We show how to apply functions from other R packages to phyloseq-represented data, illustrating the availability of a large number of open source analysis techniques. We discuss the use of phyloseq with tools for reproducible research, a practice common in other fields but still rare in the analysis of highly parallel microbiome census data. We have made available all of the materials necessary to completely reproduce the analysis and figures included in this article, an example of best practices for reproducible research.Conclusions:The phyloseq project for R is a new open-source software package, freely available on the web from both GitHub and Bioconductor. © 2013 McMurdie, Holmes., McMurdie P.J.), (10.1371/journal.pone.0061217, McMurdie, Paul J. (16245746800); Holmes, Susan (7203030601), PLoS ONE, Article, 8493, Background:The analysis of microbial communities through DNA sequencing brings many challenges: the integration of different types of data with methods from ecology, genetics, phylogenetics, multivariate statistics, visualization and testing. With the increased breadth of experimental designs now being pursued, project-specific statistical analyses are often needed, and these analyses are often difficult (or impossible) for peer researchers to independently reproduce. The vast majority of the requisite tools for performing these analyses reproducibly are already implemented in R and its extensions (packages), but with limited support for high throughput microbiome census data.Results:Here we describe a software project, phyloseq, dedicated to the object-oriented representation and analysis of microbiome census data in R. It supports importing data from a variety of common formats, as well as many analysis techniques. These include calibration, filtering, subsetting, agglomeration, multi-table comparisons, diversity analysis, parallelized Fast UniFrac, ordination methods, and production of publication-quality graphics; all in a manner that is easy to document, share, and modify. We show how to apply functions from other R packages to phyloseq-represented data, illustrating the availability of a large number of open source analysis techniques. We discuss the use of phyloseq with tools for reproducible research, a practice common in other fields but still rare in the analysis of highly parallel microbiome census data. We have made available all of the materials necessary to completely reproduce the analysis and figures included in this article, an example of best practices for reproducible research.Conclusions:The phyloseq project for R is a new open-source software package, freely available on the web from both GitHub and Bioconductor. © 2013 McMurdie, Holmes., Holmes S.), (10.1137/07070111X, Kolda, Tamara G. (6602726792); Bader, Brett W. (14024152800), SIAM Review, Review, 6344, This survey provides an overview of higher-order tensor decompositions, their applications, and available software. A tensor is a multidimensional or N-way array. Decompositions of higher-order tensors (i.e., N-way arrays with N ≥ 3) have applications in psychometrics, chemometrics, signal processing, numerical linear algebra, computer vision, numerical analysis, data mining, neuroscience, graph analysis, and elsewhere. Two particular tensor decompositions can be considered to be higher-order extensions of the matrix singular value decomposition: CANDECOMP/PARAFAC (CP) decomposes a tensor as a sum of rank-one tensors, and the Tucker decomposition is a higher-order form of principal component analysis. There are many other tensor decompositions, including INDSCAL, PARAFAC2, CANDELINC, DEDICOM, and PARATUCK2 as well as nonnegative variants of all of the above. The N-way Toolbox, Tensor Toolbox, and Multilinear Engine are examples of software packages for working with tensors. © 2009 Society for Industrial and Applied Mathematics., Kolda T.G.), (10.1137/07070111X, Kolda, Tamara G. (6602726792); Bader, Brett W. (14024152800), SIAM Review, Review, 6344, This survey provides an overview of higher-order tensor decompositions, their applications, and available software. A tensor is a multidimensional or N-way array. Decompositions of higher-order tensors (i.e., N-way arrays with N ≥ 3) have applications in psychometrics, chemometrics, signal processing, numerical linear algebra, computer vision, numerical analysis, data mining, neuroscience, graph analysis, and elsewhere. Two particular tensor decompositions can be considered to be higher-order extensions of the matrix singular value decomposition: CANDECOMP/PARAFAC (CP) decomposes a tensor as a sum of rank-one tensors, and the Tucker decomposition is a higher-order form of principal component analysis. There are many other tensor decompositions, including INDSCAL, PARAFAC2, CANDELINC, DEDICOM, and PARATUCK2 as well as nonnegative variants of all of the above. The N-way Toolbox, Tensor Toolbox, and Multilinear Engine are examples of software packages for working with tensors. © 2009 Society for Industrial and Applied Mathematics., Bader B.W.), (10.1002/0471787779, Bazaraa, Mokhtar S. (6603978524); Sherali, Hanif D. (7006900646); Shetty, C. M. (6701714902), Nonlinear Programming: Theory and Algorithms, Book, 6116, COMPREHENSIVE COVERAGE OF NONLINEAR PROGRAMMING THEORY AND ALGORITHMS, THOROUGHLY REVISED AND EXPANDED Nonlinear Programming: Theory and Algorithms-now in an extensively updated Third Edition-addresses the problem of optimizing an objective function in the presence of equality and inequality constraints. Many realistic problems cannot be adequately represented as a linear program owing to the nature of the nonlinearity of the objective function and/or the nonlinearity of any constraints. The Third Edition begins with a general introduction to nonlinear programming with illustrative examples and guidelines for model construction. Concentration on the three major parts of nonlinear programming is provided: Convex analysis with discussion of topological properties of convex sets, separation and support of convex sets, polyhedral sets, extreme points and extreme directions of polyhedral sets, and linear programming Optimality conditions and duality with coverage of the nature, interpretation, and value of the classical Fritz John (FJ) and the Karush-Kuhn-Tucker (KKT) optimality conditions; the interrelationships between various proposed constraint qualifications; and Lagrangian duality and saddle point optimality conditions Algorithms and their convergence, with a presentation of algorithms for solving both unconstrained and constrained nonlinear programming problems Important features of the Third Edition include: New topics such as second interior point methods, nonconvex optimization, nondifferentiable optimization, and more Updated discussion and new applications in each chapter Detailed numerical examples and graphical illustrations Essential coverage of modeling and formulating nonlinear programs Simple numerical problems Advanced theoretical exercises The book is a solid reference for professionals as well as a useful text for students in the fields of operations research, management science, industrial engineering, applied mathematics, and also in engineering disciplines that deal with analytical optimization techniques. The logical and self-contained format uniquely covers nonlinear programming techniques with a great depth of information and an abundance of valuable examples and illustrations that showcase the most current advances in nonlinear problems. © 2006 John Wiley & Sons, Inc. All rights reserved., Bazaraa M.S.), (10.1002/0471787779, Bazaraa, Mokhtar S. (6603978524); Sherali, Hanif D. (7006900646); Shetty, C. M. (6701714902), Nonlinear Programming: Theory and Algorithms, Book, 6116, COMPREHENSIVE COVERAGE OF NONLINEAR PROGRAMMING THEORY AND ALGORITHMS, THOROUGHLY REVISED AND EXPANDED Nonlinear Programming: Theory and Algorithms-now in an extensively updated Third Edition-addresses the problem of optimizing an objective function in the presence of equality and inequality constraints. Many realistic problems cannot be adequately represented as a linear program owing to the nature of the nonlinearity of the objective function and/or the nonlinearity of any constraints. The Third Edition begins with a general introduction to nonlinear programming with illustrative examples and guidelines for model construction. Concentration on the three major parts of nonlinear programming is provided: Convex analysis with discussion of topological properties of convex sets, separation and support of convex sets, polyhedral sets, extreme points and extreme directions of polyhedral sets, and linear programming Optimality conditions and duality with coverage of the nature, interpretation, and value of the classical Fritz John (FJ) and the Karush-Kuhn-Tucker (KKT) optimality conditions; the interrelationships between various proposed constraint qualifications; and Lagrangian duality and saddle point optimality conditions Algorithms and their convergence, with a presentation of algorithms for solving both unconstrained and constrained nonlinear programming problems Important features of the Third Edition include: New topics such as second interior point methods, nonconvex optimization, nondifferentiable optimization, and more Updated discussion and new applications in each chapter Detailed numerical examples and graphical illustrations Essential coverage of modeling and formulating nonlinear programs Simple numerical problems Advanced theoretical exercises The book is a solid reference for professionals as well as a useful text for students in the fields of operations research, management science, industrial engineering, applied mathematics, and also in engineering disciplines that deal with analytical optimization techniques. The logical and self-contained format uniquely covers nonlinear programming techniques with a great depth of information and an abundance of valuable examples and illustrations that showcase the most current advances in nonlinear problems. © 2006 John Wiley & Sons, Inc. All rights reserved., Sherali H.D.), (10.1002/0471787779, Bazaraa, Mokhtar S. (6603978524); Sherali, Hanif D. (7006900646); Shetty, C. M. (6701714902), Nonlinear Programming: Theory and Algorithms, Book, 6116, COMPREHENSIVE COVERAGE OF NONLINEAR PROGRAMMING THEORY AND ALGORITHMS, THOROUGHLY REVISED AND EXPANDED Nonlinear Programming: Theory and Algorithms-now in an extensively updated Third Edition-addresses the problem of optimizing an objective function in the presence of equality and inequality constraints. Many realistic problems cannot be adequately represented as a linear program owing to the nature of the nonlinearity of the objective function and/or the nonlinearity of any constraints. The Third Edition begins with a general introduction to nonlinear programming with illustrative examples and guidelines for model construction. Concentration on the three major parts of nonlinear programming is provided: Convex analysis with discussion of topological properties of convex sets, separation and support of convex sets, polyhedral sets, extreme points and extreme directions of polyhedral sets, and linear programming Optimality conditions and duality with coverage of the nature, interpretation, and value of the classical Fritz John (FJ) and the Karush-Kuhn-Tucker (KKT) optimality conditions; the interrelationships between various proposed constraint qualifications; and Lagrangian duality and saddle point optimality conditions Algorithms and their convergence, with a presentation of algorithms for solving both unconstrained and constrained nonlinear programming problems Important features of the Third Edition include: New topics such as second interior point methods, nonconvex optimization, nondifferentiable optimization, and more Updated discussion and new applications in each chapter Detailed numerical examples and graphical illustrations Essential coverage of modeling and formulating nonlinear programs Simple numerical problems Advanced theoretical exercises The book is a solid reference for professionals as well as a useful text for students in the fields of operations research, management science, industrial engineering, applied mathematics, and also in engineering disciplines that deal with analytical optimization techniques. The logical and self-contained format uniquely covers nonlinear programming techniques with a great depth of information and an abundance of valuable examples and illustrations that showcase the most current advances in nonlinear problems. © 2006 John Wiley & Sons, Inc. All rights reserved., Shetty C.M.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Bustin S.A.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Benes V.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Garson J.A.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Hellemans J.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Huggett J.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Kubista M.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Mueller R.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Nolan T.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Pfaffl M.W.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Shipley G.L.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Vandesompele J.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Wittwer C.T.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., James Kent W.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Sugnet C.W.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Furey T.S.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Roskin K.M.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Pringle T.H.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Zahler A.M.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Haussler D.), (10.1038/nature14539, Lecun, Yann (55666793600); Bengio, Yoshua (7003958245); Hinton, Geoffrey (7006699573), Nature, Review, 44908, Deep learning allows computational models that are composed of multiple processing layers to learn representations of data with multiple levels of abstraction. These methods have dramatically improved the state-of-the-art in speech recognition, visual object recognition, object detection and many other domains such as drug discovery and genomics. Deep learning discovers intricate structure in large data sets by using the backpropagation algorithm to indicate how a machine should change its internal parameters that are used to compute the representation in each layer from the representation in the previous layer. Deep convolutional nets have brought about breakthroughs in processing images, video, speech and audio, whereas recurrent nets have shone light on sequential data such as text and speech. © 2015 Macmillan Publishers Limited. All rights reserved., Lecun Y.), (10.1038/nature14539, Lecun, Yann (55666793600); Bengio, Yoshua (7003958245); Hinton, Geoffrey (7006699573), Nature, Review, 44908, Deep learning allows computational models that are composed of multiple processing layers to learn representations of data with multiple levels of abstraction. These methods have dramatically improved the state-of-the-art in speech recognition, visual object recognition, object detection and many other domains such as drug discovery and genomics. Deep learning discovers intricate structure in large data sets by using the backpropagation algorithm to indicate how a machine should change its internal parameters that are used to compute the representation in each layer from the representation in the previous layer. Deep convolutional nets have brought about breakthroughs in processing images, video, speech and audio, whereas recurrent nets have shone light on sequential data such as text and speech. © 2015 Macmillan Publishers Limited. All rights reserved., Bengio Y.), (10.1038/nature14539, Lecun, Yann (55666793600); Bengio, Yoshua (7003958245); Hinton, Geoffrey (7006699573), Nature, Review, 44908, Deep learning allows computational models that are composed of multiple processing layers to learn representations of data with multiple levels of abstraction. These methods have dramatically improved the state-of-the-art in speech recognition, visual object recognition, object detection and many other domains such as drug discovery and genomics. Deep learning discovers intricate structure in large data sets by using the backpropagation algorithm to indicate how a machine should change its internal parameters that are used to compute the representation in each layer from the representation in the previous layer. Deep convolutional nets have brought about breakthroughs in processing images, video, speech and audio, whereas recurrent nets have shone light on sequential data such as text and speech. © 2015 Macmillan Publishers Limited. All rights reserved., Hinton G.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Tamura K.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Dudley J.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Nei M.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Kumar S.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Murshudov G.N.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Skubák P.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Lebedev A.A.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Pannu N.S.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Steiner R.A.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Nicholls R.A.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Winn M.D.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Long F.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Vagin A.A.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Bates D.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Mächler M.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Bolker B.M.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Walker S.C.), (10.1109/TPAMI.2013.50, Bengio, Yoshua (7003958245); Courville, Aaron (6507291186); Vincent, Pascal (57203214842), IEEE Transactions on Pattern Analysis and Machine Intelligence, Article, 7471, The success of machine learning algorithms generally depends on data representation, and we hypothesize that this is because different representations can entangle and hide more or less the different explanatory factors of variation behind the data. Although specific domain knowledge can be used to help design representations, learning with generic priors can also be used, and the quest for AI is motivating the design of more powerful representation-learning algorithms implementing such priors. This paper reviews recent work in the area of unsupervised feature learning and deep learning, covering advances in probabilistic models, autoencoders, manifold learning, and deep networks. This motivates longer term unanswered questions about the appropriate objectives for learning good representations, for computing representations (i.e., inference), and the geometrical connections between representation learning, density estimation, and manifold learning. © 1979-2012 IEEE., Bengio Y.), (10.1109/TPAMI.2013.50, Bengio, Yoshua (7003958245); Courville, Aaron (6507291186); Vincent, Pascal (57203214842), IEEE Transactions on Pattern Analysis and Machine Intelligence, Article, 7471, The success of machine learning algorithms generally depends on data representation, and we hypothesize that this is because different representations can entangle and hide more or less the different explanatory factors of variation behind the data. Although specific domain knowledge can be used to help design representations, learning with generic priors can also be used, and the quest for AI is motivating the design of more powerful representation-learning algorithms implementing such priors. This paper reviews recent work in the area of unsupervised feature learning and deep learning, covering advances in probabilistic models, autoencoders, manifold learning, and deep networks. This motivates longer term unanswered questions about the appropriate objectives for learning good representations, for computing representations (i.e., inference), and the geometrical connections between representation learning, density estimation, and manifold learning. © 1979-2012 IEEE., Courville A.), ...]\n",
      "\n",
      "[12874 rows x 0 columns]\n"
     ]
    }
   ],
   "source": [
    "# Dividir los nombres de los autores por '; '\n",
    "dataframe['Authors'] = dataframe['Authors'].str.split('; ')\n",
    "\n",
    "# Expandir la lista de autores a filas separadas\n",
    "dataframe = dataframe.explode('Authors')\n",
    "\n",
    "# Limpiar los espacios en blanco adicionales que puedan haber quedado después de la división\n",
    "dataframe['Authors'] = dataframe['Authors'].str.strip()\n",
    "\n",
    "# Reorganizar las columnas para que 'Authors' sea la última columna\n",
    "dataframe = dataframe[['DOI', 'Source Title', 'Author Full Names', 'Document Type', 'Times Cited, All Databases', 'Abstract', 'Authors']]\n",
    "\n",
    "# Establecer 'DOI' y 'Authors' como índices\n",
    "dataframe.set_index(['DOI','Author Full Names','Source Title', 'Document Type', 'Times Cited, All Databases' , 'Abstract','Authors'], inplace=True)\n",
    "\n",
    "# Imprimir el DataFrame\n",
    "print(dataframe)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>DOI</th>\n",
       "      <th>Author Full Names</th>\n",
       "      <th>Source Title</th>\n",
       "      <th>Document Type</th>\n",
       "      <th>Times Cited, All Databases</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Authors</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th rowspan=\"5\" valign=\"top\">10.1007/BF00197809</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045)</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">Journal of Biomolecular NMR</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">Article</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">11644</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V.</th>\n",
       "      <th>Delaglio F.</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Grzesiek S.</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Vuister G.W.</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Zhu G.</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pfeifer J.</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <th>...</th>\n",
       "      <th>...</th>\n",
       "      <th>...</th>\n",
       "      <th>...</th>\n",
       "      <th>...</th>\n",
       "      <th>...</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th rowspan=\"5\" valign=\"top\">10.1145/780822.781133</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">Gay, D; Levis, P; von Behren, R; Welsh, M; Brewer, E; Culler, D</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">ACM SIGPLAN NOTICES</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">Article; Proceedings Paper</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">322</th>\n",
       "      <th rowspan=\"5\" valign=\"top\">We present nesC, a programming language for networked embedded systems that represent a new design space for application developers. An example of a networked embedded system is a sensor network, which consists of (potentially) thousands of tiny, low-power motes, each of which execute concurrent, reactive programs that must operate with severe memory and power constraints. nesC's contribution is to support the special needs of this domain by exposing a programming model that incorporates event-driven execution, a flexible concurrency model, and component-oriented application design. Restrictions on the programming model allow the nesC compiler to perform whole-program analyses, including data-race detection (which improves reliability) and aggressive function inlining (which reduces resource consumption). nesC has been used to implement TinyOS, a small operating system for sensor networks, as well as several significant sensor applications. nesC and TinyOS have been adopted by a large number of sensor network research groups, and our experience and evaluation of the language shows that it is effective at supporting the complex, concurrent programming style demanded by this new class of deeply networked systems.</th>\n",
       "      <th>Levis, P</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>von Behren, R</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Welsh, M</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Brewer, E</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Culler, D</th>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>12874 rows × 0 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: []\n",
       "Index: [(10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Delaglio F.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Grzesiek S.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Vuister G.W.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Zhu G.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Pfeifer J.), (10.1007/BF00197809, Delaglio, Frank (6603722428); Grzesiek, Stephan (7005157322); Vuister, Geerten W. (7004551796); Zhu, Guang (55240289900); Pfeifer, John (7005432594); Bax, Ad (7102779045), Journal of Biomolecular NMR, Article, 11644, The NMRPipe system is a UNIX software environment of processing, graphics, and analysis tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments. The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control. In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction. Complete multidimensional processing schemes are constructed as simple UNIX shell scripts. The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing. The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks. © 1995 ESCOM Science Publishers B.V., Bax A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Jumper J.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Evans R.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Pritzel A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Green T.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Figurnov M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Ronneberger O.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Tunyasuvunakool K.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Bates R.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Žídek A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Potapenko A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Bridgland A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Meyer C.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Kohl S.A.A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Ballard A.J.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Cowie A.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Romera-Paredes B.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Nikolov S.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Jain R.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Adler J.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Back T.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Petersen S.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Reiman D.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Clancy E.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Zielinski M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Steinegger M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Pacholska M.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Berghammer T.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Bodenstein S.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Silver D.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Vinyals O.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Senior A.W.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Kavukcuoglu K.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Kohli P.), (10.1038/s41586-021-03819-2, Jumper, John (36717725300); Evans, Richard (57683818800); Pritzel, Alexander (57194211707); Green, Tim (57209233923); Figurnov, Michael (57194205652); Ronneberger, Olaf (6505888134); Tunyasuvunakool, Kathryn (57225994535); Bates, Russ (56708095100); Žídek, Augustin (57204800548); Potapenko, Anna (57220414079); Bridgland, Alex (57211696334); Meyer, Clemens (57203636305); Kohl, Simon A. A. (57204142298); Ballard, Andrew J. (35725270900); Cowie, Andrew (57226008279); Romera-Paredes, Bernardino (39862104900); Nikolov, Stanislav (57203636533); Jain, Rishub (57225987330); Adler, Jonas (57193711251); Back, Trevor (57193128349); Petersen, Stig (56535950000); Reiman, David (57225992380); Clancy, Ellen (57223722021); Zielinski, Michal (57210553368); Steinegger, Martin (55820768400); Pacholska, Michalina (57196071698); Berghammer, Tamas (57226016249); Bodenstein, Sebastian (57225991765); Silver, David (7202151417); Vinyals, Oriol (24342311100); Senior, Andrew W. (7103116795); Kavukcuoglu, Koray (25646533000); Kohli, Pushmeet (14035707300); Hassabis, Demis (15839738700), Nature, Article, 6981, Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the ‘protein folding problem’8—has been an important open research problem for more than 50 years9. Despite recent progress10–14, existing methods fall far short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm. © 2021, The Author(s)., Hassabis D.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Wild S.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Roglic G.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Green A.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., Sicree R.), (10.2337/diacare.27.5.1047, Wild, Sarah (7103086309); Roglic, Gojka (6701601112); Green, Anders (7404021380); Sicree, Richard (6603179704); King, Hilary (7201415459), Diabetes Care, Article, 12069, OBJECTIVE - The goal of this study was to estimate the prevalence of diabetes and the number of people of all ages with diabetes for years 2000 and 2030. RESEARCH DESIGN AND METHODS - Data on diabetes prevalence by age and sex from a limited number of countries were extrapolated to all 191 World Health Organization member states and applied to United Nations' population estimates for 2000 and 2030. Urban and rural populations were considered separately for developing countries. RESULTS - The prevalence of diabetes for all age-groups worldwide was estimated to be 2.8% in 2000 and 4.4% in 2030. The total number of people with diabetes is projected to rise from 171 million in 2000 to 366 million in 2030. The prevalence of diabetes is higher in men than women, but there are more women with diabetes than men. The urban population in developing countries is projected to double between 2000 and 2030. The most important demographic change to diabetes prevalence across the world appears to be the increase in the proportion of people >65 years of age. CONCLUSIONS - These findings indicate that the \"diabetes epidemic\" will continue even if levels of obesity remain constant. Given the increasing prevalence of obesity, it is likely that these figures provide an underestimate of future diabetes prevalence., King H.), (10.1002/047174882X, Cover, Thomas M. (7102102743); Thomas, Joy A. (7406570513), Elements of Information Theory, Book, 33728, The latest edition of this classic is updated with new problem sets and material. The Second Edition of this fundamental textbook maintains the book's tradition of clear, thought-provoking instruction. Readers are provided once again with an instructive mix of mathematics, physics, statistics, and information theory. All the essential topics in information theory are covered in detail, including entropy, data compression, channel capacity, rate distortion, network information theory, and hypothesis testing. The authors provide readers with a solid understanding of the underlying theory and applications. Problem sets and a telegraphic summary at the end of each chapter further assist readers. The historical notes that follow each chapter recap the main points. The Second Edition features: Chapters reorganized to improve teaching. 200 new problems. New material on source coding, portfolio theory, and feedback capacity. Updated references. Now current and enhanced, the Second Edition of Elements of Information Theory remains the ideal textbook for upper-level undergraduate and graduate courses in electrical engineering, statistics, and telecommunications. An Instructor's Manual presenting detailed solutions to all the problems in the book is available from the Wiley editorial department. © 2006 John Wiley & Sons, Inc.., Cover T.M.), (10.1002/047174882X, Cover, Thomas M. (7102102743); Thomas, Joy A. (7406570513), Elements of Information Theory, Book, 33728, The latest edition of this classic is updated with new problem sets and material. The Second Edition of this fundamental textbook maintains the book's tradition of clear, thought-provoking instruction. Readers are provided once again with an instructive mix of mathematics, physics, statistics, and information theory. All the essential topics in information theory are covered in detail, including entropy, data compression, channel capacity, rate distortion, network information theory, and hypothesis testing. The authors provide readers with a solid understanding of the underlying theory and applications. Problem sets and a telegraphic summary at the end of each chapter further assist readers. The historical notes that follow each chapter recap the main points. The Second Edition features: Chapters reorganized to improve teaching. 200 new problems. New material on source coding, portfolio theory, and feedback capacity. Updated references. Now current and enhanced, the Second Edition of Elements of Information Theory remains the ideal textbook for upper-level undergraduate and graduate courses in electrical engineering, statistics, and telecommunications. An Instructor's Manual presenting detailed solutions to all the problems in the book is available from the Wiley editorial department. © 2006 John Wiley & Sons, Inc.., Thomas J.A.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., LeCun Y.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., Bottou L.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., Bengio Y.), (10.1109/5.726791, LeCun, Yann (55666793600); Bottou, Léon (6701721644); Bengio, Yoshua (7003958245); Haffner, Patrick (6701644748), Proceedings of the IEEE, Article, 31945, Multilayer neural networks trained with the back-propagation algorithm constitute the best example of a successful gradient-based learning technique. Given an appropriate network architecture, gradient-based learning algorithms can be used to synthesize a complex decision surface that can classify high-dimensional patterns, such as handwritten characters, with minimal preprocessing. This paper reviews various methods applied to handwritten character recognition and compares them on a standard handwritten digit recognition task. Convolutional neural networks, which are specifically designed to deal with the variability of two dimensional (2-D) shapes, are shown to outperform all other techniques. Real-life document recognition systems are composed of multiple modules including field extraction, segmentation, recognition, and language modeling. A new learning paradigm, called graph transformer networks (GTN's), allows such multimodule systems to be trained globally using gradient-based methods so as to minimize an overall performance measure. Two systems for online handwriting recognition are described. Experiments demonstrate the advantage of global training, and the flexibility of graph transformer networks. A graph transformer network for reading a bank check is also described. It uses convolutional neural network character recognizers combined with global training techniques to provide record accuracy on business and personal checks. It is deployed commercially and reads several million checks per day. © 1998 IEEE., Haffner P.), (10.1109/ICCV.2015.169, Girshick, Ross (35179333300), Proceedings of the IEEE International Conference on Computer Vision, Conference paper, 14787, This paper proposes a Fast Region-based Convolutional Network method (Fast R-CNN) for object detection. Fast R-CNN builds on previous work to efficiently classify object proposals using deep convolutional networks. Compared to previous work, Fast R-CNN employs several innovations to improve training and testing speed while also increasing detection accuracy. Fast R-CNN trains the very deep VGG16 network 9x faster than R-CNN, is 213x faster at test-time, and achieves a higher mAP on PASCAL VOC 2012. Compared to SPPnet, Fast R-CNN trains VGG16 3x faster, tests 10x faster, and is more accurate. Fast R-CNN is implemented in Python and C++ (using Caffe) and is available under the open-source MIT License at https://github.com/rbgirshick/fast-rcnn. © 2015 IEEE., Girshick R.), (10.1371/journal.pone.0061217, McMurdie, Paul J. (16245746800); Holmes, Susan (7203030601), PLoS ONE, Article, 8493, Background:The analysis of microbial communities through DNA sequencing brings many challenges: the integration of different types of data with methods from ecology, genetics, phylogenetics, multivariate statistics, visualization and testing. With the increased breadth of experimental designs now being pursued, project-specific statistical analyses are often needed, and these analyses are often difficult (or impossible) for peer researchers to independently reproduce. The vast majority of the requisite tools for performing these analyses reproducibly are already implemented in R and its extensions (packages), but with limited support for high throughput microbiome census data.Results:Here we describe a software project, phyloseq, dedicated to the object-oriented representation and analysis of microbiome census data in R. It supports importing data from a variety of common formats, as well as many analysis techniques. These include calibration, filtering, subsetting, agglomeration, multi-table comparisons, diversity analysis, parallelized Fast UniFrac, ordination methods, and production of publication-quality graphics; all in a manner that is easy to document, share, and modify. We show how to apply functions from other R packages to phyloseq-represented data, illustrating the availability of a large number of open source analysis techniques. We discuss the use of phyloseq with tools for reproducible research, a practice common in other fields but still rare in the analysis of highly parallel microbiome census data. We have made available all of the materials necessary to completely reproduce the analysis and figures included in this article, an example of best practices for reproducible research.Conclusions:The phyloseq project for R is a new open-source software package, freely available on the web from both GitHub and Bioconductor. © 2013 McMurdie, Holmes., McMurdie P.J.), (10.1371/journal.pone.0061217, McMurdie, Paul J. (16245746800); Holmes, Susan (7203030601), PLoS ONE, Article, 8493, Background:The analysis of microbial communities through DNA sequencing brings many challenges: the integration of different types of data with methods from ecology, genetics, phylogenetics, multivariate statistics, visualization and testing. With the increased breadth of experimental designs now being pursued, project-specific statistical analyses are often needed, and these analyses are often difficult (or impossible) for peer researchers to independently reproduce. The vast majority of the requisite tools for performing these analyses reproducibly are already implemented in R and its extensions (packages), but with limited support for high throughput microbiome census data.Results:Here we describe a software project, phyloseq, dedicated to the object-oriented representation and analysis of microbiome census data in R. It supports importing data from a variety of common formats, as well as many analysis techniques. These include calibration, filtering, subsetting, agglomeration, multi-table comparisons, diversity analysis, parallelized Fast UniFrac, ordination methods, and production of publication-quality graphics; all in a manner that is easy to document, share, and modify. We show how to apply functions from other R packages to phyloseq-represented data, illustrating the availability of a large number of open source analysis techniques. We discuss the use of phyloseq with tools for reproducible research, a practice common in other fields but still rare in the analysis of highly parallel microbiome census data. We have made available all of the materials necessary to completely reproduce the analysis and figures included in this article, an example of best practices for reproducible research.Conclusions:The phyloseq project for R is a new open-source software package, freely available on the web from both GitHub and Bioconductor. © 2013 McMurdie, Holmes., Holmes S.), (10.1137/07070111X, Kolda, Tamara G. (6602726792); Bader, Brett W. (14024152800), SIAM Review, Review, 6344, This survey provides an overview of higher-order tensor decompositions, their applications, and available software. A tensor is a multidimensional or N-way array. Decompositions of higher-order tensors (i.e., N-way arrays with N ≥ 3) have applications in psychometrics, chemometrics, signal processing, numerical linear algebra, computer vision, numerical analysis, data mining, neuroscience, graph analysis, and elsewhere. Two particular tensor decompositions can be considered to be higher-order extensions of the matrix singular value decomposition: CANDECOMP/PARAFAC (CP) decomposes a tensor as a sum of rank-one tensors, and the Tucker decomposition is a higher-order form of principal component analysis. There are many other tensor decompositions, including INDSCAL, PARAFAC2, CANDELINC, DEDICOM, and PARATUCK2 as well as nonnegative variants of all of the above. The N-way Toolbox, Tensor Toolbox, and Multilinear Engine are examples of software packages for working with tensors. © 2009 Society for Industrial and Applied Mathematics., Kolda T.G.), (10.1137/07070111X, Kolda, Tamara G. (6602726792); Bader, Brett W. (14024152800), SIAM Review, Review, 6344, This survey provides an overview of higher-order tensor decompositions, their applications, and available software. A tensor is a multidimensional or N-way array. Decompositions of higher-order tensors (i.e., N-way arrays with N ≥ 3) have applications in psychometrics, chemometrics, signal processing, numerical linear algebra, computer vision, numerical analysis, data mining, neuroscience, graph analysis, and elsewhere. Two particular tensor decompositions can be considered to be higher-order extensions of the matrix singular value decomposition: CANDECOMP/PARAFAC (CP) decomposes a tensor as a sum of rank-one tensors, and the Tucker decomposition is a higher-order form of principal component analysis. There are many other tensor decompositions, including INDSCAL, PARAFAC2, CANDELINC, DEDICOM, and PARATUCK2 as well as nonnegative variants of all of the above. The N-way Toolbox, Tensor Toolbox, and Multilinear Engine are examples of software packages for working with tensors. © 2009 Society for Industrial and Applied Mathematics., Bader B.W.), (10.1002/0471787779, Bazaraa, Mokhtar S. (6603978524); Sherali, Hanif D. (7006900646); Shetty, C. M. (6701714902), Nonlinear Programming: Theory and Algorithms, Book, 6116, COMPREHENSIVE COVERAGE OF NONLINEAR PROGRAMMING THEORY AND ALGORITHMS, THOROUGHLY REVISED AND EXPANDED Nonlinear Programming: Theory and Algorithms-now in an extensively updated Third Edition-addresses the problem of optimizing an objective function in the presence of equality and inequality constraints. Many realistic problems cannot be adequately represented as a linear program owing to the nature of the nonlinearity of the objective function and/or the nonlinearity of any constraints. The Third Edition begins with a general introduction to nonlinear programming with illustrative examples and guidelines for model construction. Concentration on the three major parts of nonlinear programming is provided: Convex analysis with discussion of topological properties of convex sets, separation and support of convex sets, polyhedral sets, extreme points and extreme directions of polyhedral sets, and linear programming Optimality conditions and duality with coverage of the nature, interpretation, and value of the classical Fritz John (FJ) and the Karush-Kuhn-Tucker (KKT) optimality conditions; the interrelationships between various proposed constraint qualifications; and Lagrangian duality and saddle point optimality conditions Algorithms and their convergence, with a presentation of algorithms for solving both unconstrained and constrained nonlinear programming problems Important features of the Third Edition include: New topics such as second interior point methods, nonconvex optimization, nondifferentiable optimization, and more Updated discussion and new applications in each chapter Detailed numerical examples and graphical illustrations Essential coverage of modeling and formulating nonlinear programs Simple numerical problems Advanced theoretical exercises The book is a solid reference for professionals as well as a useful text for students in the fields of operations research, management science, industrial engineering, applied mathematics, and also in engineering disciplines that deal with analytical optimization techniques. The logical and self-contained format uniquely covers nonlinear programming techniques with a great depth of information and an abundance of valuable examples and illustrations that showcase the most current advances in nonlinear problems. © 2006 John Wiley & Sons, Inc. All rights reserved., Bazaraa M.S.), (10.1002/0471787779, Bazaraa, Mokhtar S. (6603978524); Sherali, Hanif D. (7006900646); Shetty, C. M. (6701714902), Nonlinear Programming: Theory and Algorithms, Book, 6116, COMPREHENSIVE COVERAGE OF NONLINEAR PROGRAMMING THEORY AND ALGORITHMS, THOROUGHLY REVISED AND EXPANDED Nonlinear Programming: Theory and Algorithms-now in an extensively updated Third Edition-addresses the problem of optimizing an objective function in the presence of equality and inequality constraints. Many realistic problems cannot be adequately represented as a linear program owing to the nature of the nonlinearity of the objective function and/or the nonlinearity of any constraints. The Third Edition begins with a general introduction to nonlinear programming with illustrative examples and guidelines for model construction. Concentration on the three major parts of nonlinear programming is provided: Convex analysis with discussion of topological properties of convex sets, separation and support of convex sets, polyhedral sets, extreme points and extreme directions of polyhedral sets, and linear programming Optimality conditions and duality with coverage of the nature, interpretation, and value of the classical Fritz John (FJ) and the Karush-Kuhn-Tucker (KKT) optimality conditions; the interrelationships between various proposed constraint qualifications; and Lagrangian duality and saddle point optimality conditions Algorithms and their convergence, with a presentation of algorithms for solving both unconstrained and constrained nonlinear programming problems Important features of the Third Edition include: New topics such as second interior point methods, nonconvex optimization, nondifferentiable optimization, and more Updated discussion and new applications in each chapter Detailed numerical examples and graphical illustrations Essential coverage of modeling and formulating nonlinear programs Simple numerical problems Advanced theoretical exercises The book is a solid reference for professionals as well as a useful text for students in the fields of operations research, management science, industrial engineering, applied mathematics, and also in engineering disciplines that deal with analytical optimization techniques. The logical and self-contained format uniquely covers nonlinear programming techniques with a great depth of information and an abundance of valuable examples and illustrations that showcase the most current advances in nonlinear problems. © 2006 John Wiley & Sons, Inc. All rights reserved., Sherali H.D.), (10.1002/0471787779, Bazaraa, Mokhtar S. (6603978524); Sherali, Hanif D. (7006900646); Shetty, C. M. (6701714902), Nonlinear Programming: Theory and Algorithms, Book, 6116, COMPREHENSIVE COVERAGE OF NONLINEAR PROGRAMMING THEORY AND ALGORITHMS, THOROUGHLY REVISED AND EXPANDED Nonlinear Programming: Theory and Algorithms-now in an extensively updated Third Edition-addresses the problem of optimizing an objective function in the presence of equality and inequality constraints. Many realistic problems cannot be adequately represented as a linear program owing to the nature of the nonlinearity of the objective function and/or the nonlinearity of any constraints. The Third Edition begins with a general introduction to nonlinear programming with illustrative examples and guidelines for model construction. Concentration on the three major parts of nonlinear programming is provided: Convex analysis with discussion of topological properties of convex sets, separation and support of convex sets, polyhedral sets, extreme points and extreme directions of polyhedral sets, and linear programming Optimality conditions and duality with coverage of the nature, interpretation, and value of the classical Fritz John (FJ) and the Karush-Kuhn-Tucker (KKT) optimality conditions; the interrelationships between various proposed constraint qualifications; and Lagrangian duality and saddle point optimality conditions Algorithms and their convergence, with a presentation of algorithms for solving both unconstrained and constrained nonlinear programming problems Important features of the Third Edition include: New topics such as second interior point methods, nonconvex optimization, nondifferentiable optimization, and more Updated discussion and new applications in each chapter Detailed numerical examples and graphical illustrations Essential coverage of modeling and formulating nonlinear programs Simple numerical problems Advanced theoretical exercises The book is a solid reference for professionals as well as a useful text for students in the fields of operations research, management science, industrial engineering, applied mathematics, and also in engineering disciplines that deal with analytical optimization techniques. The logical and self-contained format uniquely covers nonlinear programming techniques with a great depth of information and an abundance of valuable examples and illustrations that showcase the most current advances in nonlinear problems. © 2006 John Wiley & Sons, Inc. All rights reserved., Shetty C.M.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Bustin S.A.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Benes V.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Garson J.A.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Hellemans J.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Huggett J.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Kubista M.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Mueller R.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Nolan T.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Pfaffl M.W.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Shipley G.L.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Vandesompele J.), (10.1373/clinchem.2008.112797, Bustin, Stephen A. (7004591466); Benes, Vladimir (35197826600); Garson, Jeremy A. (7006765284); Hellemans, Jan (9639683700); Huggett, Jim (36979844800); Kubista, Mikael (55238107800); Mueller, Reinhold (57196583546); Nolan, Tania (7102073919); Pfaffl, Michael W. (7004517305); Shipley, Gregory L. (7005550861); Vandesompele, Jo (6701639561); Wittwer, Carl T. (7006481035), Clinical Chemistry, Article, 10640, BACKGROUND: Currently, a lack of consensus exists on how best to perform and interpret quantitative real- time PCR (qPCR) experiments. The problem is exac- erbated by a lack of sufficient experimental detail in many publications, which impedes a reader's ability to evaluate critically the quality of the results presented or to repeat the experiments. CONTENT: The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines target the reliability of results to help ensure the integrity of the scientific literature, promote con- sistency between laboratories, and increase experimen- tal transparency. MIQE is a set of guidelines that de- scribe the minimum information necessary for evaluating qPCR experiments. Included is a checklist to accompany the initial submission of a manuscript to the publisher. By providing all relevant experimental conditions and assay characteristics, reviewers can as- sess the validity of the protocols used. Full disclosure of all reagents, sequences, and analysis methods is neces- sary to enable other investigators to reproduce results. MIQE details should be published either in abbreviated form or as an online supplement. SUMMARY: Following these guidelines will encourage better experimental practice, allowing more reliable and unequivocal interpretation of qPCR results. © 2009 American Association for Clinical Chemistry., Wittwer C.T.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., James Kent W.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Sugnet C.W.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Furey T.S.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Roskin K.M.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Pringle T.H.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Zahler A.M.), (10.1101/gr.229102. Article published online before print in May 2002, James Kent, W. (14052305000); Sugnet, Charles W. (6508252449); Furey, Terrence S. (6602302257); Roskin, Krishna M. (6603650729); Pringle, Tom H. (7006022668); Zahler, Alan M. (7003556937); Haussler, David (7006637893), Genome Research, Article, 6877, As vertebrate genome sequences near completion and research refocuses to their analysis, the issue of effective genome annotation display becomes critical. A mature web tool for rapid and reliable display of any requested portion of the genome at any scale, together with several dozen aligned annotation tracks, is provided at http://genome.ucsc.edu. This browser displays assembly contigs and gaps, mRNA and expressed sequence tag alignments, multiple gene predictions, cross-species homologies, single nucleotide polymorphisms, sequence-tagged sites, radiation hybrid data, transposon repeats, and more as a stack of coregistered tracks. Text and sequence-based searches provide quick and precise access to any region of specific interest. Secondary links from individual features lead to sequence details and supplementary off-site databases. One-half of the annotation tracks are computed at the University of California, Santa Cruz from publicly available sequence data; collaborators worldwide provide the rest. Users can stably add their own custom tracks to the browser for educational or research purposes. The conceptual and technical framework of the browser, its underlying MYSQL database, and overall use are described. The web site currently serves over 50,000 pages per day to over 3000 different users., Haussler D.), (10.1038/nature14539, Lecun, Yann (55666793600); Bengio, Yoshua (7003958245); Hinton, Geoffrey (7006699573), Nature, Review, 44908, Deep learning allows computational models that are composed of multiple processing layers to learn representations of data with multiple levels of abstraction. These methods have dramatically improved the state-of-the-art in speech recognition, visual object recognition, object detection and many other domains such as drug discovery and genomics. Deep learning discovers intricate structure in large data sets by using the backpropagation algorithm to indicate how a machine should change its internal parameters that are used to compute the representation in each layer from the representation in the previous layer. Deep convolutional nets have brought about breakthroughs in processing images, video, speech and audio, whereas recurrent nets have shone light on sequential data such as text and speech. © 2015 Macmillan Publishers Limited. All rights reserved., Lecun Y.), (10.1038/nature14539, Lecun, Yann (55666793600); Bengio, Yoshua (7003958245); Hinton, Geoffrey (7006699573), Nature, Review, 44908, Deep learning allows computational models that are composed of multiple processing layers to learn representations of data with multiple levels of abstraction. These methods have dramatically improved the state-of-the-art in speech recognition, visual object recognition, object detection and many other domains such as drug discovery and genomics. Deep learning discovers intricate structure in large data sets by using the backpropagation algorithm to indicate how a machine should change its internal parameters that are used to compute the representation in each layer from the representation in the previous layer. Deep convolutional nets have brought about breakthroughs in processing images, video, speech and audio, whereas recurrent nets have shone light on sequential data such as text and speech. © 2015 Macmillan Publishers Limited. All rights reserved., Bengio Y.), (10.1038/nature14539, Lecun, Yann (55666793600); Bengio, Yoshua (7003958245); Hinton, Geoffrey (7006699573), Nature, Review, 44908, Deep learning allows computational models that are composed of multiple processing layers to learn representations of data with multiple levels of abstraction. These methods have dramatically improved the state-of-the-art in speech recognition, visual object recognition, object detection and many other domains such as drug discovery and genomics. Deep learning discovers intricate structure in large data sets by using the backpropagation algorithm to indicate how a machine should change its internal parameters that are used to compute the representation in each layer from the representation in the previous layer. Deep convolutional nets have brought about breakthroughs in processing images, video, speech and audio, whereas recurrent nets have shone light on sequential data such as text and speech. © 2015 Macmillan Publishers Limited. All rights reserved., Hinton G.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Tamura K.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Dudley J.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Nei M.), (10.1093/molbev/msm092, Tamura, Koichiro (55509065700); Dudley, Joel (18633685600); Nei, Masatoshi (7102222781); Kumar, Sudhir (35199760000), Molecular Biology and Evolution, Article, 25633, We announce the release of the fourth version of MEGA software, which expands on the existing facilities for editing DNA sequence data from autosequencers, mining Web-databases, performing automatic and manual sequence alignment, analyzing sequence alignments to estimate evolutionary distances, inferring phylogenetic trees, and testing evolutionary hypotheses. Version 4 includes a unique facility to generate captions, written in figure legend format, in order to provide natural language descriptions of the models and methods used in the analyses. This facility aims to promote a better understanding of the underlying assumptions used in analyses, and of the results generated. Another new feature is the Maximum Composite Likelihood (MCL) method for estimating evolutionary distances between all pairs of sequences simultaneously, with and without incorporating rate variation among sites and substitution pattern heterogeneities among lineages. This MCL method also can be used to estimate transition/transversion bias and nucleotide substitution pattern without knowledge of the phylogenetic tree. This new version is a native 32-bit Windows application with multi-threading and multi-user supports, and it is also available to run in a Linux desktop environment (via the Wine compatibility layer) and on Intel-based Macintosh computers under the Parallels program. The current version of MEGA is available free of charge at http://www.megasoftware.net. © The Author 2007. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved., Kumar S.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Murshudov G.N.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Skubák P.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Lebedev A.A.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Pannu N.S.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Steiner R.A.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Nicholls R.A.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Winn M.D.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Long F.), (10.1107/S0907444911001314, Murshudov, Garib N. (6701788980); Skubák, Pavol (8272725200); Lebedev, Andrey A. (55415462800); Pannu, Navraj S. (6603790127); Steiner, Roberto A. (7402618778); Nicholls, Robert A. (47961370000); Winn, Martyn D. (57203031960); Long, Fei (47961241200); Vagin, Alexei A. (6701816307), Acta Crystallographica Section D: Biological Crystallography, Article, 6105, This paper describes various components of the macromolecular crystallographic refinement program REFMAC5, which is distributed as part of the CCP4 suite. REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS experimental diffraction data. To ensure chemical and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization. Reliable models at resolutions at least as low as 4 Å can be achieved thanks to low-resolution refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on atomic displacement parameters (ADPs) based on the Kullback-Leibler divergence. REFMAC5 additionally offers TLS parameterization and, when high-resolution data are available, fast refinement of anisotropic ADPs. Refinement in the presence of twinning is performed in a fully automated fashion. REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resolution spectrum encountered in macromolecular crystallography., Vagin A.A.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Bates D.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Mächler M.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Bolker B.M.), (10.18637/jss.v067.i01, Bates, Douglas (7202955759); Mächler, Martin (56466481900); Bolker, Benjamin M. (7003958481); Walker, Steven C. (57203474453), Journal of Statistical Software, Article, 41300, Maximum likelihood or restricted maximum likelihood (REML) estimates of the parameters in linear mixed-effects models can be determined using the lmer function in the lme4 package for R. As for most model-fitting functions in R, the model is described in an lmer call by a formula, in this case including both fixed- and random-effects terms. The formula and data together determine a numerical representation of the model from which the profiled deviance or the profiled REML criterion can be evaluated as a function of some of the model parameters. The appropriate criterion is optimized, using one of the constrained optimization functions in R, to provide the parameter estimates. We describe the structure of the model, the steps in evaluating the profiled deviance or REML criterion, and the structure of classes or types that represents such a model. Sufficient detail is included to allow specialization of these structures by users who wish to write functions to fit specialized linear mixed models, such as models incorporating pedigrees or smoothing splines, that are not easily expressible in the formula language used by lmer. © 2015, American Statistical Association. All rights reserved., Walker S.C.), (10.1109/TPAMI.2013.50, Bengio, Yoshua (7003958245); Courville, Aaron (6507291186); Vincent, Pascal (57203214842), IEEE Transactions on Pattern Analysis and Machine Intelligence, Article, 7471, The success of machine learning algorithms generally depends on data representation, and we hypothesize that this is because different representations can entangle and hide more or less the different explanatory factors of variation behind the data. Although specific domain knowledge can be used to help design representations, learning with generic priors can also be used, and the quest for AI is motivating the design of more powerful representation-learning algorithms implementing such priors. This paper reviews recent work in the area of unsupervised feature learning and deep learning, covering advances in probabilistic models, autoencoders, manifold learning, and deep networks. This motivates longer term unanswered questions about the appropriate objectives for learning good representations, for computing representations (i.e., inference), and the geometrical connections between representation learning, density estimation, and manifold learning. © 1979-2012 IEEE., Bengio Y.), (10.1109/TPAMI.2013.50, Bengio, Yoshua (7003958245); Courville, Aaron (6507291186); Vincent, Pascal (57203214842), IEEE Transactions on Pattern Analysis and Machine Intelligence, Article, 7471, The success of machine learning algorithms generally depends on data representation, and we hypothesize that this is because different representations can entangle and hide more or less the different explanatory factors of variation behind the data. Although specific domain knowledge can be used to help design representations, learning with generic priors can also be used, and the quest for AI is motivating the design of more powerful representation-learning algorithms implementing such priors. This paper reviews recent work in the area of unsupervised feature learning and deep learning, covering advances in probabilistic models, autoencoders, manifold learning, and deep networks. This motivates longer term unanswered questions about the appropriate objectives for learning good representations, for computing representations (i.e., inference), and the geometrical connections between representation learning, density estimation, and manifold learning. © 1979-2012 IEEE., Courville A.), ...]\n",
       "\n",
       "[12874 rows x 0 columns]"
      ]
     },
     "execution_count": 98,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index([], dtype='object')\n",
      "(12874, 0)\n",
      "MultiIndex([(        '10.1007/BF00197809', ...),\n",
      "            (        '10.1007/BF00197809', ...),\n",
      "            (        '10.1007/BF00197809', ...),\n",
      "            (        '10.1007/BF00197809', ...),\n",
      "            (        '10.1007/BF00197809', ...),\n",
      "            (        '10.1007/BF00197809', ...),\n",
      "            ('10.1038/s41586-021-03819-2', ...),\n",
      "            ('10.1038/s41586-021-03819-2', ...),\n",
      "            ('10.1038/s41586-021-03819-2', ...),\n",
      "            ('10.1038/s41586-021-03819-2', ...),\n",
      "            ...\n",
      "            (   '10.1145/3272127.3275035', ...),\n",
      "            (   '10.1145/3272127.3275035', ...),\n",
      "            (   '10.1145/3272127.3275035', ...),\n",
      "            (   '10.1145/3272127.3275035', ...),\n",
      "            (     '10.1145/780822.781133', ...),\n",
      "            (     '10.1145/780822.781133', ...),\n",
      "            (     '10.1145/780822.781133', ...),\n",
      "            (     '10.1145/780822.781133', ...),\n",
      "            (     '10.1145/780822.781133', ...),\n",
      "            (     '10.1145/780822.781133', ...)],\n",
      "           names=['DOI', 'Author Full Names', 'Source Title', 'Document Type', 'Times Cited, All Databases', 'Abstract', 'Authors'], length=12874)\n",
      "['DOI', 'Author Full Names', 'Source Title', 'Document Type', 'Times Cited, All Databases', 'Abstract', 'Authors']\n"
     ]
    }
   ],
   "source": [
    "print(dataframe.columns)\n",
    "print(dataframe.shape)\n",
    "print(dataframe.index)\n",
    "print(dataframe.index.names)\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
